// @clerk/ui/dist/no-rhc/components/ConfigureSSO/ConfigureSSO.js
import { withCoreUserGuard } from "../../contexts/CoreUserContext.js";
import { localizationKeys } from "../../localization/localizationKeys.js";
import { Route } from "../../router/Route.js";
import { Switch } from "../../router/Switch.js";
import SvgExclamationTriangle from "../../icons/exclamation-triangle.js";
import { useProtect } from "../../common/Gate.js";
import { withCardStateProvider } from "../../elements/contexts/index.js";
import { Flow } from "../../customizables/Flow.js";
import { Col, Flex, Heading, Icon, Text } from "../../customizables/index.js";
import { ProfileCard } from "../../elements/ProfileCard/index.js";
import { ConfigureSSONavbar } from "./ConfigureSSONavbar.js";
import { ProfileCardFooter, ProfileCardHeader } from "./elements/ProfileCard.js";
import { Step } from "./elements/Step.js";
import { ConfigureSSOSkeleton } from "./ConfigureSSOSkeleton.js";
import { ConfigureSSOWizard } from "./ConfigureSSOWizard.js";
import { useOrganizationEnterpriseConnection } from "./hooks/useOrganizationEnterpriseConnection.js";
import React from "react";
import { useSession } from "@clerk/shared/react";
import { Fragment as Fragment$1, jsx, jsxs } from "@emotion/react/jsx-runtime";

//#region src/components/ConfigureSSO/ConfigureSSO.tsx
const ConfigureSSOInternal = () => {
	return /* @__PURE__ */ jsx(Flow.Root, {
		flow: "configureSSO",
		children: /* @__PURE__ */ jsx(Switch, { children: /* @__PURE__ */ jsx(Route, { children: /* @__PURE__ */ jsx(AuthenticatedContent, {}) }) })
	});
};
const AuthenticatedContent = withCoreUserGuard(() => {
	const contentRef = React.useRef(null);
	return /* @__PURE__ */ jsx(ProfileCard.Root, {
		sx: (t) => ({
			display: "grid",
			gridTemplateColumns: "1fr 3fr",
			height: t.sizes.$176,
			overflow: "hidden"
		}),
		children: /* @__PURE__ */ jsx(ConfigureSSONavbar, {
			contentRef,
			children: /* @__PURE__ */ jsx(ConfigureSSOContent, { contentRef })
		})
	});
});
const ConfigureSSOContent = ({ contentRef }) => {
	const { isLoading, enterpriseConnection, organizationEnterpriseConnection, testRuns, enterpriseConnectionMutations, organizationDomains, organizationDomainMutations } = useOrganizationEnterpriseConnection();
	if (isLoading) return /* @__PURE__ */ jsx(ConfigureSSOSkeleton, {});
	return /* @__PURE__ */ jsx(ConfigureSSOProtect, { children: /* @__PURE__ */ jsx(ConfigureSSOWizard, {
		organizationEnterpriseConnection,
		testRuns,
		enterpriseConnection,
		contentRef,
		enterpriseConnectionMutations,
		organizationDomainMutations,
		organizationDomains
	}) });
};
/** Permission gate shared by the wizard's hosts — personal workspaces pass, since there is no membership to check. */
const ConfigureSSOProtect = ({ children }) => {
	const { session } = useSession();
	const isPersonalWorkspace = !session?.lastActiveOrganizationId;
	if (!useProtect((has) => isPersonalWorkspace || has({ permission: "org:sys_entconns:manage" }))) return /* @__PURE__ */ jsx(MissingManageEnterpriseConnectionsPermission, {});
	return children;
};
const MissingManageEnterpriseConnectionsPermission = () => /* @__PURE__ */ jsxs(Fragment$1, { children: [
	/* @__PURE__ */ jsx(ProfileCardHeader, {}),
	/* @__PURE__ */ jsx(Step.Body, { children: /* @__PURE__ */ jsx(Step.Section, {
		sx: { flex: 1 },
		align: "center",
		justify: "center",
		children: /* @__PURE__ */ jsx(Flex, {
			align: "center",
			justify: "center",
			sx: (t) => ({
				flex: 1,
				padding: t.space.$8
			}),
			children: /* @__PURE__ */ jsxs(Col, {
				align: "center",
				sx: (t) => ({
					gap: t.space.$2,
					textAlign: "center",
					maxWidth: t.sizes.$94
				}),
				children: [
					/* @__PURE__ */ jsx(Icon, {
						icon: SvgExclamationTriangle,
						sx: (t) => ({
							width: t.sizes.$8,
							height: t.sizes.$8,
							color: t.colors.$neutralAlpha600
						})
					}),
					/* @__PURE__ */ jsx(Heading, {
						localizationKey: localizationKeys("configureSSO.missingManageEnterpriseConnectionsPermission.title"),
						textVariant: "h1",
						sx: (t) => ({
							fontSize: t.fontSizes.$lg,
							textWrap: "balance"
						})
					}),
					/* @__PURE__ */ jsx(Text, {
						as: "p",
						variant: "body",
						colorScheme: "secondary",
						localizationKey: localizationKeys("configureSSO.missingManageEnterpriseConnectionsPermission.subtitle"),
						sx: { textWrap: "balance" }
					})
				]
			})
		})
	}) }),
	/* @__PURE__ */ jsx(ProfileCardFooter, {})
] });
const ConfigureSSO = withCardStateProvider(ConfigureSSOInternal);

//#endregion
export { ConfigureSSO, ConfigureSSOProtect };
