// @clerk/ui/dist/no-rhc/components/OrganizationProfile/InviteMembersForm.js
import { useEnvironment } from "../../contexts/EnvironmentContext.js";
import { localizationKeys } from "../../localization/localizationKeys.js";
import { createListFormat } from "../../utils/passwordUtils.js";
import { getPaidSeatsUnitTier, getSeatUnitPrice, organizationAndInvitationsExceedsPurchasedSeats } from "../../utils/billingPlanSeats.js";
import { getClosestProfileScrollBoxFromElement } from "../../utils/getClosestProfileScrollBox.js";
import { usePlansContext, useSubscription } from "../../contexts/components/Plans.js";
import { useLocalizations } from "../../localization/makeLocalizable.js";
import { mqu } from "../../styledSystem/breakpoints.js";
import { useFormControl } from "../../utils/useFormControl.js";
import { useCardState } from "../../elements/contexts/index.js";
import { Flex } from "../../customizables/index.js";
import { handleError } from "../../utils/errorHandler.js";
import { Form } from "../../elements/Form.js";
import { FormButtonContainer } from "../../elements/FormButtons.js";
import { TagInput } from "../../elements/TagInput.js";
import { useFetchRoles } from "../../hooks/useFetchRoles.js";
import { RoleSelect } from "./MemberListTable.js";
import { isClerkAPIResponseError } from "@clerk/shared/error";
import { useEffect, useState } from "react";
import { useClerk, useOrganization } from "@clerk/shared/react";
import { jsx, jsxs } from "@emotion/react/jsx-runtime";

//#region src/components/OrganizationProfile/InviteMembersForm.tsx
const isEmail = (str) => /^\S+@\S+\.\S+$/.test(str);
const InviteMembersForm = (props) => {
	const { onSuccess, onReset, resetButtonLabel } = props;
	const clerk = useClerk();
	const { organization, invitations } = useOrganization({ invitations: {
		pageSize: 10,
		keepPreviousData: true
	} });
	const { data: subscription, subscriptionItems } = useSubscription();
	const activeSubscriptionItem = subscription?.subscriptionItems.find((si) => si.status === "active");
	const { handleSelectPlan } = usePlansContext();
	const card = useCardState();
	const { t, locale } = useLocalizations();
	const [isValidUnsubmittedEmail, setIsValidUnsubmittedEmail] = useState(false);
	const validateUnsubmittedEmail = (value) => setIsValidUnsubmittedEmail(isEmail(value));
	const emailAddressField = useFormControl("emailAddress", "", {
		type: "text",
		label: localizationKeys("formFieldLabel__emailAddresses")
	});
	const roleField = useFormControl("role", "", { label: localizationKeys("formFieldLabel__role") });
	if (!organization) return null;
	const { props: { setError, setWarning, setSuccess, setInfo, isFocused, validatePassword, setHasPassedComplexity, hasPassedComplexity, feedback, feedbackType, clearFeedback, ...restEmailAddressProps } } = emailAddressField;
	const canSubmit = (!!emailAddressField.value.length || isValidUnsubmittedEmail) && !!roleField.value;
	const emailAddresses = emailAddressField.value.split(",");
	const seatUnitPrice = activeSubscriptionItem ? getSeatUnitPrice(activeSubscriptionItem.plan) : null;
	const isPerSeatCostPlan = !!(seatUnitPrice ? getPaidSeatsUnitTier(seatUnitPrice) : null);
	const mustPurchaseSeats = isPerSeatCostPlan && organizationAndInvitationsExceedsPurchasedSeats(activeSubscriptionItem, organization, emailAddresses.length);
	const onSubmit = async (e) => {
		e.preventDefault();
		if (!canSubmit) return;
		const submittedData = new FormData(e.currentTarget);
		const portalRoot = getClosestProfileScrollBoxFromElement(e.currentTarget);
		const inviteMembersParams = {
			emailAddresses: emailAddressField.value.split(","),
			role: submittedData.get("role")
		};
		try {
			await inviteMembers(inviteMembersParams);
		} catch (err) {
			await handleInviteMembersError(err, inviteMembersParams, { portalRoot });
		}
	};
	/**
	* Attempt to invite members to an organization. Throws errors if unsuccessful.
	*/
	const inviteMembers = async (params) => {
		await organization.inviteMembers(params);
		await invitations?.revalidate?.();
		onSuccess?.();
	};
	/**
	* Attempt to invite members after a checkout has been performed. If inviting members throws an error, handle the
	* error without triggering the checkout flow again.
	*/
	const inviteMembersAfterCheckout = (params) => {
		(async () => {
			card.setLoading();
			card.setError(void 0);
			try {
				await inviteMembers(params);
			} catch (err) {
				await handleInviteMembersError(err, params, { openCheckoutOnInsufficientSeats: false });
			} finally {
				card.setIdle();
			}
		})();
	};
	/**
	* Handle errors that are thrown after attempting to invite members to an organization.
	*/
	const handleInviteMembersError = async (err, inviteMembersParams, { openCheckoutOnInsufficientSeats = true, portalRoot } = {}) => {
		if (!isClerkAPIResponseError(err)) {
			if (err instanceof Error) {
				handleError(err, [], card.setError);
				return;
			}
			throw err;
		}
		const apiError = err.errors[0];
		removeInvalidEmails(apiError, inviteMembersParams.emailAddresses);
		switch (apiError?.code) {
			case "duplicate_record": {
				const unlocalizedEmailsList = apiError.meta?.emailAddresses || [];
				card.setError(t(localizationKeys("organizationProfile.invitePage.detailsTitle__inviteFailed", { email_addresses: createListFormat(unlocalizedEmailsList, locale) })));
				break;
			}
			case "already_a_member_in_organization": {
				const email = (apiError.longMessage ?? "").match(/\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b/)?.[0];
				handleError(err, [], (err) => email ? card.setError(t(localizationKeys("unstable__errors.already_a_member_in_organization", { email }))) : card.setError(err));
				break;
			}
			case "insufficient_seats":
				if (!openCheckoutOnInsufficientSeats) {
					handleError(err, [], () => card.setError(t(localizationKeys("unstable__errors.insufficient_seats_change_plan"))));
					break;
				}
				try {
					const { data: plans } = await clerk.billing.getPlans({
						for: "organization",
						orgId: organization.id,
						minSeats: apiError.meta?.seatsQuantity,
						pageSize: 500
					});
					if (plans.length === 0) {
						handleError(err, [], () => card.setError(t(localizationKeys("unstable__errors.insufficient_seats_contact_support"))));
						break;
					}
					const activeSubscriptionItem = subscriptionItems.find((si) => si.status === "active" || si.status === "past_due");
					if (activeSubscriptionItem) {
						const currentPlan = activeSubscriptionItem.plan;
						if (plans.some((p) => p.id === currentPlan.id && p.availablePrices?.some((price) => price.id === activeSubscriptionItem.priceId))) {
							handleSelectPlan({
								mode: "modal",
								plan: currentPlan,
								planPeriod: activeSubscriptionItem.planPeriod,
								seatsQuantity: apiError.meta?.seatsQuantity,
								priceId: activeSubscriptionItem.priceId,
								portalRoot,
								onSubscriptionComplete: () => inviteMembersAfterCheckout(inviteMembersParams)
							});
							break;
						}
					}
					handleError(err, [], () => card.setError(t(localizationKeys("unstable__errors.insufficient_seats_change_plan"))));
					break;
				} catch (err) {
					if (err instanceof Error) handleError(err, [], () => card.setError(t(localizationKeys("unstable__errors.insufficient_seats_contact_support"))));
					break;
				}
			default: handleError(err, [], card.setError);
		}
	};
	const removeInvalidEmails = (err, emails = emailAddressField.value.split(",")) => {
		const invalidEmails = new Set([...err.meta?.emailAddresses ?? [], ...err.meta?.identifiers ?? []]);
		emailAddressField.setValue(emails.filter((e) => !invalidEmails.has(e)).join(","));
	};
	return /* @__PURE__ */ jsxs(Form.Root, {
		onSubmit,
		children: [/* @__PURE__ */ jsx(Form.ControlRow, {
			elementId: emailAddressField.id,
			children: /* @__PURE__ */ jsx(TagInput, {
				...restEmailAddressProps,
				autoFocus: true,
				validate: isEmail,
				sx: { width: "100%" },
				validateUnsubmittedEmail,
				placeholder: localizationKeys("formFieldInputPlaceholder__emailAddresses")
			})
		}), /* @__PURE__ */ jsxs(Flex, {
			align: "center",
			justify: "between",
			sx: (t) => ({
				marginTop: t.space.$2,
				gap: t.space.$4,
				flexWrap: "wrap",
				[mqu.sm]: { justifyContent: "center" }
			}),
			children: [/* @__PURE__ */ jsx(AsyncRoleSelect, { ...roleField }), /* @__PURE__ */ jsxs(FormButtonContainer, {
				sx: (t) => ({
					margin: t.space.$none,
					flexWrap: "wrap",
					justifyContent: "center"
				}),
				children: [/* @__PURE__ */ jsx(Form.SubmitButton, {
					block: false,
					isDisabled: !canSubmit,
					localizationKey: isPerSeatCostPlan && mustPurchaseSeats ? localizationKeys("organizationProfile.invitePage.formButtonPrimary__purchaseSeats") : localizationKeys("organizationProfile.invitePage.formButtonPrimary__continue")
				}), /* @__PURE__ */ jsx(Form.ResetButton, {
					localizationKey: resetButtonLabel || localizationKeys("userProfile.formButtonReset"),
					block: false,
					onClick: onReset
				})]
			})]
		})]
	});
};
const AsyncRoleSelect = (field) => {
	const { options, isLoading, hasRoleSetMigration } = useFetchRoles();
	const { t } = useLocalizations();
	const defaultRole = useDefaultRole();
	useEffect(() => {
		if (field.value || !defaultRole) return;
		if (!options?.some((option) => option.value === defaultRole)) return;
		field.setValue(defaultRole);
	}, [
		defaultRole,
		options,
		field
	]);
	return /* @__PURE__ */ jsx(Form.ControlRow, {
		elementId: field.id,
		children: /* @__PURE__ */ jsx(Flex, {
			direction: "col",
			gap: 2,
			children: /* @__PURE__ */ jsx(RoleSelect, {
				...field.props,
				roles: options,
				isDisabled: isLoading || hasRoleSetMigration,
				onChange: (value) => field.setValue(value),
				triggerSx: (t) => ({
					minWidth: t.sizes.$40,
					justifyContent: "space-between",
					display: "flex"
				}),
				optionListSx: (t) => ({ minWidth: t.sizes.$48 }),
				prefixLocalizationKey: `${t(localizationKeys("formFieldLabel__role"))}:`
			})
		})
	});
};
/**
* Determines default Role from the Organization settings or fallback to
* the only available Role.
*/
const useDefaultRole = () => {
	const { options } = useFetchRoles();
	const { organizationSettings } = useEnvironment();
	let defaultRole = organizationSettings.domains.defaultRole;
	if (!defaultRole && options?.length === 1) defaultRole = options[0].value;
	return defaultRole;
};

//#endregion
export { InviteMembersForm };
