// @clerk/ui/dist/no-rhc/components/OrganizationProfile/OrganizationSecurityPage.js
import { localizationKeys } from "../../localization/localizationKeys.js";
import SvgChevronLeft from "../../icons/chevron-left.js";
import { descriptors } from "../../customizables/elementDescriptors.js";
import { Col, Flex, Icon, SimpleButton, Spinner, Text } from "../../customizables/index.js";
import { Header } from "../../elements/Header.js";
import { ProfileCard } from "../../elements/ProfileCard/index.js";
import { ConfigureSSOWizard } from "../ConfigureSSO/ConfigureSSOWizard.js";
import { useOrganizationEnterpriseConnection } from "../ConfigureSSO/hooks/useOrganizationEnterpriseConnection.js";
import { SecuritySsoSection } from "./SecuritySsoSection.js";
import React, { useState } from "react";
import { useOrganization } from "@clerk/shared/react";
import { jsx, jsxs } from "@emotion/react/jsx-runtime";

//#region src/components/OrganizationProfile/OrganizationSecurityPage.tsx
const OrganizationSecurityPage = ({ contentRef }) => {
	const { organization } = useOrganization();
	if (!organization) return null;
	return /* @__PURE__ */ jsx(OrganizationSecurityPageContent, { contentRef });
};
const OrganizationSecurityPageContent = ({ contentRef }) => {
	const { organization, isLoading, enterpriseConnection, organizationEnterpriseConnection, testRuns, enterpriseConnectionMutations, organizationDomains, organizationDomainMutations } = useOrganizationEnterpriseConnection();
	const [view, setView] = useState("overview");
	const [forceFirstStep, setForceFirstStep] = useState(false);
	const exitWizard = () => setView("overview");
	const openWizard = (forceInitialStep = false) => {
		setForceFirstStep(forceInitialStep);
		setView("wizard");
	};
	if (isLoading && view === "overview") return /* @__PURE__ */ jsx(SecurityPageOverview, {
		fillHeight: true,
		children: /* @__PURE__ */ jsx(Flex, {
			align: "center",
			justify: "center",
			sx: (t) => ({
				flex: 1,
				paddingBlock: t.space.$5
			}),
			children: /* @__PURE__ */ jsx(Spinner, {
				size: "xs",
				colorScheme: "neutral",
				elementDescriptor: descriptors.spinner
			})
		})
	});
	const backControl = /* @__PURE__ */ jsxs(SimpleButton, {
		elementDescriptor: descriptors.configureSSOHeaderBackButton,
		variant: "unstyled",
		onClick: exitWizard,
		sx: (t) => ({
			gap: t.space.$1,
			padding: 0,
			color: t.colors.$colorMutedForeground,
			"&:hover": { color: t.colors.$colorForeground }
		}),
		children: [/* @__PURE__ */ jsx(Icon, { icon: SvgChevronLeft }), /* @__PURE__ */ jsx(Text, {
			as: "span",
			variant: "body",
			localizationKey: localizationKeys("organizationProfile.navbar.security")
		})]
	});
	return view === "overview" ? /* @__PURE__ */ jsx(SecurityPageOverview, { children: /* @__PURE__ */ jsx(SecuritySsoSection, {
		connection: organizationEnterpriseConnection,
		enterpriseConnection,
		setConnectionActive: enterpriseConnectionMutations.setConnectionActive,
		deleteConnection: enterpriseConnectionMutations.deleteConnection,
		organizationName: organization?.name ?? "",
		contentRef,
		onConfigure: openWizard
	}) }) : /* @__PURE__ */ jsx(ConfigureSSOWizard, {
		organizationEnterpriseConnection,
		testRuns,
		enterpriseConnection,
		contentRef,
		enterpriseConnectionMutations,
		organizationDomainMutations,
		organizationDomains,
		forceInitialStep: forceFirstStep,
		title: backControl,
		onExit: exitWizard
	});
};
/**
* The overview's stable page chrome — the security `ProfileCard.Page` and its
* "Security" header. Both the settled overview and the on-mount loading state
* render through this, so the section body is the only thing that swaps in.
*
* `fillHeight` grows the page to the scroll box so the loading state's spinner
* can center in the remaining height beneath the header.
*/
const SecurityPageOverview = ({ children, fillHeight = false }) => /* @__PURE__ */ jsx(ProfileCard.Page, {
	sx: fillHeight ? { flex: 1 } : void 0,
	children: /* @__PURE__ */ jsx(Col, {
		elementDescriptor: descriptors.page,
		sx: (t) => ({
			gap: t.space.$8,
			...fillHeight && { flex: 1 }
		}),
		children: /* @__PURE__ */ jsxs(Col, {
			elementDescriptor: descriptors.profilePage,
			elementId: descriptors.profilePage.setId("organizationSecurity"),
			sx: fillHeight ? { flex: 1 } : void 0,
			children: [/* @__PURE__ */ jsx(Header.Root, { children: /* @__PURE__ */ jsx(Header.Title, {
				localizationKey: localizationKeys("organizationProfile.securityPage.title"),
				sx: (t) => ({ marginBottom: t.space.$4 }),
				textVariant: "h2"
			}) }), children]
		})
	})
});

//#endregion
export { OrganizationSecurityPage };
