// @clerk/ui/dist/no-rhc/components/ConfigureSSO/hooks/useOrganizationEnterpriseConnection.js
import { isEnterpriseConnectionConfigured, organizationEnterpriseConnection } from "../domain/organizationEnterpriseConnection.js";
import { useEnterpriseConnectionTestRuns } from "./useEnterpriseConnectionTestRuns.js";
import { useCallback, useMemo, useRef } from "react";
import { __internal_useOrganizationDomains, __internal_useOrganizationEnterpriseConnections, useOrganization, useSession, useUser } from "@clerk/shared/react";

//#region src/components/ConfigureSSO/hooks/useOrganizationEnterpriseConnection.ts
/**
* Umbrella hook for the active organization's enterprise connection. Composes
* the source query, the domain aggregate, the mutations, and the test-run state
* into one surface, exposing a single `isLoading` flag so the caller can gate
* the skeleton above the provider.
*
* `__internal_useOrganizationEnterpriseConnections` is the single swappable
* seam: a future non-org context only swaps this source and everything below
* stays put.
*/
const useOrganizationEnterpriseConnection = () => {
	const { data: enterpriseConnections, isLoading: isLoadingEnterpriseConnections, createEnterpriseConnection, updateEnterpriseConnection, deleteEnterpriseConnection } = __internal_useOrganizationEnterpriseConnections({ enabled: true });
	const enterpriseConnection = enterpriseConnections?.[0];
	const hadInitialConnectionRef = useRef(void 0);
	if (hadInitialConnectionRef.current === void 0 && !isLoadingEnterpriseConnections) hadInitialConnectionRef.current = Boolean(enterpriseConnection);
	const hadInitialConnection = hadInitialConnectionRef.current === true;
	const { hasSuccessfulTestRun, isLoading: isLoadingTestRuns, isFetching: isFetchingTestRuns, rows: testRunRows, totalCount: testRunTotalCount, isPolling: isPollingTestRuns, page: testRunPage, setPage: setTestRunPage, refresh: refreshTestRuns, revalidateHasSuccessfulTestRun } = useEnterpriseConnectionTestRuns(enterpriseConnection, isEnterpriseConnectionConfigured(enterpriseConnection) || Boolean(enterpriseConnection?.active));
	const { user } = useUser();
	const { session } = useSession();
	const { organization } = useOrganization();
	const { isLoading: isLoadingOrganizationDomains, data: organizationDomains, createDomain, prepareOwnershipVerification, attemptOwnershipVerification, revalidate: revalidateDomains } = __internal_useOrganizationDomains({
		enrollmentMode: "enterprise_sso",
		onOwnershipVerified: useCallback(async (verifiedDomains) => {
			if (!enterpriseConnection) return;
			const verifiedDomainNames = verifiedDomains.map((domain) => domain.name);
			const domains = Array.from(new Set([...enterpriseConnection.domains ?? [], ...verifiedDomainNames]));
			if (!(domains.length !== (enterpriseConnection.domains?.length ?? 0))) return;
			await updateEnterpriseConnection(enterpriseConnection.id, { domains });
		}, [enterpriseConnection, updateEnterpriseConnection])
	});
	const organizationDomainMutations = useMemo(() => ({
		createDomain,
		prepareOwnershipVerification,
		attemptOwnershipVerification,
		revalidate: revalidateDomains
	}), [
		createDomain,
		prepareOwnershipVerification,
		attemptOwnershipVerification,
		revalidateDomains
	]);
	const enterpriseConnectionMutations = useMemo(() => {
		const createConnection = (provider) => {
			return createEnterpriseConnection({
				provider,
				domains: organizationDomains?.map((domain) => domain.name)
			});
		};
		const changeProvider = async (provider) => {
			if (enterpriseConnection) await deleteEnterpriseConnection(enterpriseConnection.id);
			return createEnterpriseConnection({
				provider,
				domains: enterpriseConnection?.domains ?? organizationDomains?.map((domain) => domain.name)
			});
		};
		const updateConnection = (id, params) => updateEnterpriseConnection(id, params);
		const setConnectionActive = (id, active) => updateEnterpriseConnection(id, { active });
		const deleteConnection = (id) => deleteEnterpriseConnection(id);
		const createTestRun = (id) => {
			if (!organization) throw new Error("useOrganizationEnterpriseConnection.createTestRun called before the organization resource was loaded.");
			return organization.createEnterpriseConnectionTestRun(id);
		};
		return {
			createConnection,
			changeProvider,
			updateConnection,
			setConnectionActive,
			deleteConnection,
			createTestRun
		};
	}, [
		organization,
		organizationDomains,
		enterpriseConnection,
		createEnterpriseConnection,
		updateEnterpriseConnection,
		deleteEnterpriseConnection
	]);
	const testRuns = useMemo(() => ({
		rows: testRunRows,
		totalCount: testRunTotalCount,
		isLoading: isLoadingTestRuns,
		isFetching: isFetchingTestRuns,
		isPolling: isPollingTestRuns,
		page: testRunPage,
		setPage: setTestRunPage,
		refresh: refreshTestRuns,
		revalidateHasSuccessfulTestRun
	}), [
		testRunRows,
		testRunTotalCount,
		isLoadingTestRuns,
		isFetchingTestRuns,
		isPollingTestRuns,
		testRunPage,
		setTestRunPage,
		refreshTestRuns,
		revalidateHasSuccessfulTestRun
	]);
	const organizationEnterpriseConnection$1 = useMemo(() => organizationEnterpriseConnection({
		connection: enterpriseConnection,
		hasSuccessfulTestRun
	}), [enterpriseConnection, hasSuccessfulTestRun]);
	return {
		user,
		session,
		organization,
		isLoading: isLoadingEnterpriseConnections || isLoadingOrganizationDomains || hadInitialConnection && isLoadingTestRuns,
		enterpriseConnection,
		organizationEnterpriseConnection: organizationEnterpriseConnection$1,
		enterpriseConnectionMutations,
		testRuns,
		organizationDomains,
		organizationDomainMutations
	};
};

//#endregion
export { useOrganizationEnterpriseConnection };
