// @clerk/ui/dist/components/SignUp/SignUpProtectCheck.js
import { useRouter } from "../../router/RouteContext.js";
import { useNavigateToFlowStart } from "../../hooks/useNavigateToFlowStart.js";
import { localizationKeys } from "../../localization/localizationKeys.js";
import { useSpinDelay } from "../../hooks/useSpinDelay.js";
import { useSignUpContext } from "../../contexts/components/SignUp.js";
import { useCoreSignUp } from "../../contexts/CoreClientContext.js";
import { useLocalizations } from "../../localization/makeLocalizable.js";
import { descriptors } from "../../customizables/elementDescriptors.js";
import { useCardState, withCardStateProvider } from "../../elements/contexts/index.js";
import { Flow } from "../../customizables/Flow.js";
import { Box, Button, Col, Flex, Spinner } from "../../customizables/index.js";
import { Card } from "../../elements/Card/index.js";
import { Header } from "../../elements/Header.js";
import { withRedirectToAfterSignUp } from "../../common/withRedirect.js";
import { useProtectCheckRunner } from "../../hooks/useProtectCheckRunner.js";
import { completeSignUpFlow } from "./util.js";
import { useEffect, useRef, useState } from "react";
import { useClerk } from "@clerk/shared/react";
import { jsx, jsxs } from "@emotion/react/jsx-runtime";

//#region src/components/SignUp/SignUpProtectCheck.tsx
function SignUpProtectCheckInternal({ verifyEmailPath = "../verify-email-address", verifyPhonePath = "../verify-phone-number", continuePath = "../continue", protectCheckPath = "." } = {}) {
	const card = useCardState();
	const { t } = useLocalizations();
	const signUp = useCoreSignUp();
	const { navigate } = useRouter();
	const { navigateToFlowStart } = useNavigateToFlowStart();
	const { setActive } = useClerk();
	const { afterSignUpUrl, navigateOnSetActive } = useSignUpContext();
	const [everSawProtectCheck, setEverSawProtectCheck] = useState(!!signUp.protectCheck);
	const didStartNoCheckFallbackRef = useRef(false);
	if (signUp.protectCheck && !everSawProtectCheck) setEverSawProtectCheck(true);
	useEffect(() => {
		if (!signUp.protectCheck && !everSawProtectCheck && !didStartNoCheckFallbackRef.current) {
			didStartNoCheckFallbackRef.current = true;
			navigateToFlowStart();
		}
	}, [
		everSawProtectCheck,
		navigateToFlowStart,
		signUp.protectCheck
	]);
	const { containerRef, isRunning, isWidgetVisible, hasError, retry } = useProtectCheckRunner({
		getProtectCheck: () => signUp.protectCheck,
		getResource: () => signUp,
		reload: () => signUp.reload(),
		submitProtectCheck: (params) => signUp.submitProtectCheck(params),
		onResolved: async (updatedSignUp, isCancelled) => {
			if (isCancelled()) return;
			await completeSignUpFlow({
				signUp: updatedSignUp,
				verifyEmailPath,
				verifyPhonePath,
				protectCheckPath,
				continuePath,
				handleComplete: () => setActive({
					session: updatedSignUp.createdSessionId,
					navigate: async ({ session, decorateUrl }) => {
						await navigateOnSetActive({
							session,
							redirectUrl: afterSignUpUrl,
							decorateUrl
						});
					}
				}),
				navigate
			});
		}
	});
	const showSpinner = useSpinDelay(isRunning, { delay: 300 });
	if (!signUp.protectCheck && !everSawProtectCheck) return null;
	return /* @__PURE__ */ jsx(Flow.Part, {
		part: "protectCheck",
		children: /* @__PURE__ */ jsxs(Card.Root, { children: [/* @__PURE__ */ jsxs(Card.Content, { children: [
			/* @__PURE__ */ jsxs(Header.Root, {
				showLogo: true,
				children: [/* @__PURE__ */ jsx(Header.Title, { localizationKey: localizationKeys("signUp.protectCheck.title") }), /* @__PURE__ */ jsx(Header.Subtitle, { localizationKey: localizationKeys("signUp.protectCheck.subtitle") })]
			}),
			/* @__PURE__ */ jsx(Card.Alert, { children: card.error }),
			/* @__PURE__ */ jsxs(Col, {
				elementDescriptor: descriptors.main,
				gap: 6,
				children: [
					/* @__PURE__ */ jsx(Box, {
						ref: containerRef,
						id: "clerk-protect-check",
						"aria-busy": isRunning,
						style: {
							display: "block",
							alignSelf: "center",
							position: isWidgetVisible ? "static" : "absolute"
						}
					}),
					showSpinner && !hasError && !isWidgetVisible ? /* @__PURE__ */ jsx(Flex, {
						center: true,
						children: /* @__PURE__ */ jsx(Spinner, {
							size: "lg",
							colorScheme: "primary",
							elementDescriptor: descriptors.spinner,
							"aria-label": t(localizationKeys("signUp.protectCheck.loading"))
						})
					}) : null,
					hasError ? /* @__PURE__ */ jsx(Button, {
						onClick: retry,
						localizationKey: localizationKeys("signUp.protectCheck.retryButton")
					}) : null
				]
			})
		] }), /* @__PURE__ */ jsx(Card.Footer, {})] })
	});
}
const SignUpProtectCheck = withRedirectToAfterSignUp(withCardStateProvider(SignUpProtectCheckInternal));

//#endregion
export { SignUpProtectCheck };
