// @clerk/ui/dist/no-rhc/components/SignIn/buildOAuthCallbackParams.js
//#region src/components/SignIn/buildOAuthCallbackParams.ts
function buildSignInOAuthCallbackParams(ctx) {
	return {
		signUpUrl: ctx.signUpUrl,
		signInUrl: ctx.signInUrl,
		signInForceRedirectUrl: ctx.afterSignInUrl,
		signUpForceRedirectUrl: ctx.afterSignUpUrl,
		continueSignUpUrl: ctx.signUpContinueUrl,
		transferable: ctx.transferable,
		firstFactorUrl: "../factor-one",
		secondFactorUrl: "../factor-two",
		resetPasswordUrl: "../reset-password",
		signInProtectCheckUrl: "../protect-check",
		signUpProtectCheckUrl: ctx.signUpProtectCheckUrl,
		unsafeMetadata: ctx.unsafeMetadata
	};
}
function buildSignInOAuthTransportCallbackParams(ctx) {
	return {
		...buildSignInOAuthCallbackParams(ctx),
		firstFactorUrl: "factor-one",
		secondFactorUrl: "factor-two",
		resetPasswordUrl: "reset-password",
		signInProtectCheckUrl: "protect-check"
	};
}
function buildSignUpOAuthCallbackParams(ctx) {
	return {
		signUpUrl: ctx.signUpUrl,
		signInUrl: ctx.signInUrl,
		signUpForceRedirectUrl: ctx.afterSignUpUrl,
		signInForceRedirectUrl: ctx.afterSignInUrl,
		secondFactorUrl: ctx.secondFactorUrl,
		continueSignUpUrl: "../continue",
		verifyEmailAddressUrl: "../verify-email-address",
		verifyPhoneNumberUrl: "../verify-phone-number",
		signUpProtectCheckUrl: "../protect-check",
		unsafeMetadata: ctx.unsafeMetadata
	};
}
function buildSignUpOAuthTransportCallbackParams(ctx) {
	return {
		...buildSignUpOAuthCallbackParams(ctx),
		continueSignUpUrl: "continue",
		verifyEmailAddressUrl: "verify-email-address",
		verifyPhoneNumberUrl: "verify-phone-number",
		signUpProtectCheckUrl: "protect-check"
	};
}

//#endregion
export { buildSignInOAuthCallbackParams, buildSignInOAuthTransportCallbackParams, buildSignUpOAuthCallbackParams, buildSignUpOAuthTransportCallbackParams };
