// @clerk/ui/dist/no-rhc/components/SignIn/SignInProtectCheck.js
import { useRouter } from "../../router/RouteContext.js";
import { localizationKeys } from "../../localization/localizationKeys.js";
import { useSpinDelay } from "../../hooks/useSpinDelay.js";
import { useSignInContext } from "../../contexts/components/SignIn.js";
import { useCoreSignIn } from "../../contexts/CoreClientContext.js";
import { useLocalizations } from "../../localization/makeLocalizable.js";
import { descriptors } from "../../customizables/elementDescriptors.js";
import { useCardState, withCardStateProvider } from "../../elements/contexts/index.js";
import { Flow } from "../../customizables/Flow.js";
import { Box, Button, Col, Flex, Spinner } from "../../customizables/index.js";
import { Card } from "../../elements/Card/index.js";
import { Header } from "../../elements/Header.js";
import { withRedirectToAfterSignIn } from "../../common/withRedirect.js";
import { useProtectCheckRunner } from "../../hooks/useProtectCheckRunner.js";
import { useClerk } from "@clerk/shared/react";
import { jsx, jsxs } from "@emotion/react/jsx-runtime";

//#region src/components/SignIn/SignInProtectCheck.tsx
/**
* Routes the user to the next step after a protect check has been resolved (or short-circuits
* to the same route to handle a chained challenge).
*
* After the gate clears, the client should retry the operation that was gated.
* For most steps (factor-one/factor-two cards), the underlying card uses `useFetch` to call
* `prepareFirstFactor`/`prepareSecondFactor` on mount, so navigating back is sufficient to
* re-trigger the gated work.
*/
function navigateNext(signIn, navigate) {
	if (signIn.protectCheck || signIn.status === "needs_protect_check") return navigate(".");
	switch (signIn.status) {
		case "needs_first_factor": return navigate("../factor-one");
		case "needs_second_factor": return navigate("../factor-two");
		case "needs_client_trust": return navigate("../client-trust");
		case "needs_new_password": return navigate("../reset-password");
		case "complete": return navigate("..");
		default: return navigate("..");
	}
}
function SignInProtectCheckInternal() {
	const card = useCardState();
	const { t } = useLocalizations();
	const signIn = useCoreSignIn();
	const { navigate } = useRouter();
	const { setActive } = useClerk();
	const { afterSignInUrl, navigateOnSetActive } = useSignInContext();
	const { containerRef, isRunning, isWidgetVisible, hasError, retry } = useProtectCheckRunner({
		getProtectCheck: () => signIn.protectCheck,
		getResource: () => signIn,
		reload: () => signIn.reload(),
		submitProtectCheck: (params) => signIn.submitProtectCheck(params),
		onResolved: async (updatedSignIn, isCancelled) => {
			if (isCancelled()) return;
			if (updatedSignIn.status === "complete" && updatedSignIn.createdSessionId) {
				await setActive({
					session: updatedSignIn.createdSessionId,
					navigate: async ({ session, decorateUrl }) => {
						await navigateOnSetActive({
							session,
							redirectUrl: afterSignInUrl,
							decorateUrl
						});
					}
				});
				return;
			}
			await navigateNext(updatedSignIn, navigate);
		}
	});
	const showSpinner = useSpinDelay(isRunning, { delay: 300 });
	return /* @__PURE__ */ jsx(Flow.Part, {
		part: "protectCheck",
		children: /* @__PURE__ */ jsxs(Card.Root, { children: [/* @__PURE__ */ jsxs(Card.Content, { children: [
			/* @__PURE__ */ jsxs(Header.Root, {
				showLogo: true,
				children: [/* @__PURE__ */ jsx(Header.Title, { localizationKey: localizationKeys("signIn.protectCheck.title") }), /* @__PURE__ */ jsx(Header.Subtitle, { localizationKey: localizationKeys("signIn.protectCheck.subtitle") })]
			}),
			/* @__PURE__ */ jsx(Card.Alert, { children: card.error }),
			/* @__PURE__ */ jsxs(Col, {
				elementDescriptor: descriptors.main,
				gap: 6,
				children: [
					/* @__PURE__ */ jsx(Box, {
						ref: containerRef,
						id: "clerk-protect-check",
						"aria-busy": isRunning,
						style: {
							display: "block",
							alignSelf: "center",
							position: isWidgetVisible ? "static" : "absolute"
						}
					}),
					showSpinner && !hasError && !isWidgetVisible ? /* @__PURE__ */ jsx(Flex, {
						center: true,
						children: /* @__PURE__ */ jsx(Spinner, {
							size: "lg",
							colorScheme: "primary",
							elementDescriptor: descriptors.spinner,
							"aria-label": t(localizationKeys("signIn.protectCheck.loading"))
						})
					}) : null,
					hasError ? /* @__PURE__ */ jsx(Button, {
						onClick: retry,
						localizationKey: localizationKeys("signIn.protectCheck.retryButton")
					}) : null
				]
			})
		] }), /* @__PURE__ */ jsx(Card.Footer, {})] })
	});
}
const SignInProtectCheck = withRedirectToAfterSignIn(withCardStateProvider(SignInProtectCheckInternal));

//#endregion
export { SignInProtectCheck };
