// @clerk/ui/dist/no-rhc/components/ConfigureSSO/steps/ConfigureStep/saml/shared/IdentityProviderConfigurationForm.js
import SvgArrowUpTray from "../../../../../../icons/arrow-up-tray.js";
import SvgClose from "../../../../../../icons/close.js";
import { useLocalizations } from "../../../../../../localization/makeLocalizable.js";
import { descriptors } from "../../../../../../customizables/elementDescriptors.js";
import { Badge, Box, Button, Col, Flex, Icon, Text } from "../../../../../../customizables/index.js";
import { handleError } from "../../../../../../utils/errorHandler.js";
import { Field } from "../../../../../../elements/FieldControl.js";
import { Form } from "../../../../../../elements/Form.js";
import { ActiveConnectionAlert } from "./ActiveConnectionAlert.js";
import { isClerkAPIResponseError } from "@clerk/shared/error";
import React from "react";
import { Fragment as Fragment$1, jsx, jsxs } from "@emotion/react/jsx-runtime";

//#region src/components/ConfigureSSO/steps/ConfigureStep/saml/shared/IdentityProviderConfigurationForm.tsx
const ConfigurationPanel = (config) => {
	switch (config.mode) {
		case "metadataUrl": return /* @__PURE__ */ jsx(MetadataUrlPanel, {
			form: config.form,
			labels: config.labels
		});
		case "metadataFile": return /* @__PURE__ */ jsx(MetadataFilePanel, {
			form: config.form,
			labels: config.labels
		});
		case "manual": return /* @__PURE__ */ jsx(ManualPanel, {
			form: config.form,
			labels: config.labels
		});
	}
};
const IdentityProviderConfigurationForm = (config) => {
	return /* @__PURE__ */ jsxs(Fragment$1, { children: [/* @__PURE__ */ jsx(ConfigurationPanel, { ...config }), /* @__PURE__ */ jsx(ActiveConnectionAlert, {})] });
};
const MetadataUrlPanel = ({ form, labels }) => /* @__PURE__ */ jsxs(Fragment$1, { children: [/* @__PURE__ */ jsx(Text, {
	as: "p",
	colorScheme: "secondary",
	localizationKey: labels.description
}), /* @__PURE__ */ jsx(Form.ControlRow, {
	elementId: form.field.id,
	children: /* @__PURE__ */ jsx(Form.PlainInput, { ...form.field.props })
})] });
const MetadataFilePanel = ({ form, labels }) => /* @__PURE__ */ jsxs(Fragment$1, { children: [/* @__PURE__ */ jsx(Text, {
	as: "p",
	colorScheme: "secondary",
	localizationKey: labels.description
}), /* @__PURE__ */ jsx(FileUploadField, {
	field: form.field,
	file: form.file,
	onFileChange: form.onFileChange,
	existingFilePresent: Boolean(form.existingFilePresent),
	labels,
	accept: ".xml"
})] });
const ManualPanel = ({ form, labels }) => /* @__PURE__ */ jsxs(Fragment$1, { children: [
	/* @__PURE__ */ jsx(Text, {
		as: "p",
		colorScheme: "secondary",
		localizationKey: labels.description
	}),
	/* @__PURE__ */ jsx(Form.ControlRow, {
		elementId: form.signOnUrlField.id,
		children: /* @__PURE__ */ jsx(Form.PlainInput, { ...form.signOnUrlField.props })
	}),
	/* @__PURE__ */ jsx(Form.ControlRow, {
		elementId: form.issuerField.id,
		children: /* @__PURE__ */ jsx(Form.PlainInput, { ...form.issuerField.props })
	}),
	/* @__PURE__ */ jsx(FileUploadField, {
		field: form.certificateField,
		file: form.certFile,
		onFileChange: form.onCertFileChange,
		existingFilePresent: Boolean(form.existingCertPresent),
		labels,
		accept: ".pem,.key,.crt,.cer,.cert"
	})
] });
const buildSamlConfigurationPayload = async ({ mode, metadataUrl, metadataFile, manual }) => {
	if (mode === "metadataUrl") {
		if (!metadataUrl) throw new Error("metadataUrl values missing for mode \"metadataUrl\"");
		return { idpMetadataUrl: metadataUrl.value.trim() };
	}
	if (mode === "metadataFile") {
		if (!metadataFile?.file) throw new Error("metadataFile is missing for mode \"metadataFile\"");
		return { idpMetadata: await metadataFile.file.text() };
	}
	if (!manual) throw new Error("manual values missing for mode \"manual\"");
	const payload = {
		idpSsoUrl: manual.signOnUrl.trim(),
		idpEntityId: manual.issuer.trim()
	};
	if (manual.certFile !== null) payload.idpCertificate = await manual.certFile.text();
	return payload;
};
const applySamlSubmitError = (err, card, primaryField, additionalFields = []) => {
	handleError(err, [primaryField, ...additionalFields], card.setError);
	if (isClerkAPIResponseError(err)) {
		const unscopedSamlError = err.errors.find((e) => e.code?.startsWith("saml_") && !e.meta?.paramName);
		if (unscopedSamlError) {
			primaryField.setError(unscopedSamlError);
			card.setError(void 0);
		}
	}
};
const FileUploadField = ({ field, file, onFileChange, existingFilePresent, labels, accept }) => {
	const { t } = useLocalizations();
	const inputRef = React.useRef(null);
	return /* @__PURE__ */ jsx(Box, { children: /* @__PURE__ */ jsxs(Field.Root, {
		...field.props,
		children: [/* @__PURE__ */ jsxs(Col, {
			gap: 2,
			children: [
				/* @__PURE__ */ jsx(Field.LabelRow, { children: /* @__PURE__ */ jsx(Field.Label, {}) }),
				/* @__PURE__ */ jsx("input", {
					ref: inputRef,
					type: "file",
					accept,
					multiple: false,
					style: { display: "none" },
					onChange: (e) => {
						onFileChange(e.target.files?.[0] ?? null);
						field.clearFeedback();
					}
				}),
				file === null ? /* @__PURE__ */ jsxs(Flex, {
					align: "center",
					gap: 2,
					sx: {
						alignSelf: "flex-start",
						flexWrap: "wrap"
					},
					children: [existingFilePresent && /* @__PURE__ */ jsx(Badge, {
						elementDescriptor: descriptors.configureSSOCertificateFileBadge,
						localizationKey: labels.fileUploaded
					}), /* @__PURE__ */ jsxs(Button, {
						elementDescriptor: descriptors.configureSSOCertificateUploadButton,
						size: "xs",
						variant: "outline",
						onClick: () => inputRef.current?.click(),
						children: [/* @__PURE__ */ jsx(Icon, {
							icon: SvgArrowUpTray,
							size: "sm",
							colorScheme: "neutral",
							sx: (theme) => ({ marginInlineEnd: theme.space.$1 })
						}), /* @__PURE__ */ jsx(Text, {
							as: "span",
							localizationKey: existingFilePresent ? labels.replaceFile : labels.uploadFile
						})]
					})]
				}) : /* @__PURE__ */ jsxs(Flex, {
					align: "center",
					gap: 2,
					sx: (theme) => ({
						paddingTop: theme.space.$1,
						paddingBottom: theme.space.$1
					}),
					children: [/* @__PURE__ */ jsx(Text, {
						elementDescriptor: descriptors.configureSSOCertificateFileName,
						as: "span",
						colorScheme: "secondary",
						variant: "buttonSmall",
						children: file.name
					}), /* @__PURE__ */ jsx(Button, {
						elementDescriptor: descriptors.configureSSOCertificateRemoveButton,
						variant: "ghost",
						colorScheme: "neutral",
						"aria-label": t(labels.removeFile),
						onClick: () => {
							onFileChange(null);
							field.clearFeedback();
							if (inputRef.current) inputRef.current.value = "";
						},
						sx: (theme) => ({ padding: theme.space.$1 }),
						children: /* @__PURE__ */ jsx(Icon, {
							icon: SvgClose,
							size: "xs"
						})
					})]
				})
			]
		}), /* @__PURE__ */ jsx(Field.Feedback, {})]
	}) });
};

//#endregion
export { IdentityProviderConfigurationForm, applySamlSubmitError, buildSamlConfigurationPayload };
