// @clerk/ui/dist/components/ConfigureSSO/ChangeProviderDialog.js
import { localizationKeys } from "../../localization/localizationKeys.js";
import { useLocalizations } from "../../localization/makeLocalizable.js";
import { descriptors } from "../../customizables/elementDescriptors.js";
import { withCardStateProvider } from "../../elements/contexts/index.js";
import { Button, Col, Flex, Heading, Text } from "../../customizables/index.js";
import { Modal } from "../../elements/Modal.js";
import { Card } from "../../elements/Card/index.js";
import { jsx, jsxs } from "@emotion/react/jsx-runtime";

//#region src/components/ConfigureSSO/ChangeProviderDialog.tsx
const ChangeProviderDialog = (props) => {
	if (!props.isOpen) return null;
	return /* @__PURE__ */ jsx(Modal, {
		handleClose: props.onClose,
		canCloseModal: false,
		portalRoot: props.contentRef,
		containerSx: (t) => ({
			alignItems: "center",
			position: "absolute",
			inset: 0,
			width: "auto",
			height: "auto",
			backgroundColor: "inherit",
			backdropFilter: `blur(${t.sizes.$2})`
		}),
		children: /* @__PURE__ */ jsx(ChangeProviderDialogContent, { ...props })
	});
};
const ChangeProviderDialogContent = withCardStateProvider((props) => {
	const { onClose, onConfirm, isSubmitting, nextProviderLabel, currentProviderLabel } = props;
	const { t } = useLocalizations();
	const nextProvider = t(nextProviderLabel);
	const currentProvider = t(currentProviderLabel);
	return /* @__PURE__ */ jsx(Card.Root, {
		elementDescriptor: descriptors.configureSSOChangeProviderDialog,
		sx: (t) => ({ borderRadius: t.radii.$md }),
		children: /* @__PURE__ */ jsx(Card.Content, {
			sx: (t) => ({
				textAlign: "start",
				padding: t.sizes.$5
			}),
			children: /* @__PURE__ */ jsxs(Col, {
				sx: (t) => ({ gap: t.space.$4 }),
				children: [/* @__PURE__ */ jsxs(Col, {
					sx: (t) => ({ gap: t.space.$2 }),
					children: [/* @__PURE__ */ jsx(Heading, {
						textVariant: "h2",
						localizationKey: localizationKeys("configureSSO.changeProviderDialog.title", { provider: nextProvider }),
						sx: (t) => ({ fontSize: t.fontSizes.$md })
					}), /* @__PURE__ */ jsx(Text, {
						as: "p",
						colorScheme: "secondary",
						localizationKey: localizationKeys("configureSSO.changeProviderDialog.subtitle", {
							provider: nextProvider,
							currentProvider
						})
					})]
				}), /* @__PURE__ */ jsxs(Flex, {
					justify: "end",
					sx: (t) => ({ gap: t.space.$3 }),
					children: [/* @__PURE__ */ jsx(Button, {
						elementDescriptor: descriptors.configureSSOChangeProviderDialogCancelButton,
						variant: "ghost",
						isDisabled: isSubmitting,
						onClick: onClose,
						localizationKey: localizationKeys("configureSSO.changeProviderDialog.cancelButton")
					}), /* @__PURE__ */ jsx(Button, {
						elementDescriptor: descriptors.configureSSOChangeProviderDialogConfirmButton,
						variant: "solid",
						isLoading: isSubmitting,
						onClick: onConfirm,
						localizationKey: localizationKeys("configureSSO.changeProviderDialog.confirmButton")
					})]
				})]
			})
		})
	});
});

//#endregion
export { ChangeProviderDialog };
