// @clerk/ui/dist/components/ConfigureSSO/ConfigureSSONavbar.js
import { useEnvironment } from "../../contexts/EnvironmentContext.js";
import { localizationKeys } from "../../localization/localizationKeys.js";
import SvgBox from "../../icons/box.js";
import { useAppearance } from "../../customizables/AppearanceContext.js";
import { mqu } from "../../styledSystem/breakpoints.js";
import { descriptors } from "../../customizables/elementDescriptors.js";
import { Box, Col, Flex, Heading, Icon, Text } from "../../customizables/index.js";
import { ApplicationLogo } from "../../elements/ApplicationLogo.js";
import { NavBar, NavbarContextProvider } from "../../elements/Navbar.js";
import React from "react";
import { jsx, jsxs } from "@emotion/react/jsx-runtime";
import { __internal_useOrganizationBase } from "@clerk/shared/react/index";

//#region src/components/ConfigureSSO/ConfigureSSONavbar.tsx
const ConfigureSSONavbar = ({ children, contentRef }) => {
	const { parsedOptions } = useAppearance();
	const { organizationSettings, displayConfig: { applicationName, logoImageUrl } } = useEnvironment();
	const hasLogo = Boolean(parsedOptions.logoImageUrl || logoImageUrl);
	return /* @__PURE__ */ jsxs(NavbarContextProvider, {
		contentRef,
		children: [/* @__PURE__ */ jsx(NavBar, {
			contentRef,
			title: localizationKeys("configureSSO.navbar.title"),
			titleSx: (t) => ({ fontSize: t.fontSizes.$lg }),
			containerSx: {
				flexDirection: "column-reverse",
				flex: 0
			},
			routes: [],
			header: /* @__PURE__ */ jsxs(Flex, {
				align: "center",
				sx: (t) => ({
					gap: t.space.$2,
					padding: `${t.space.$none} ${t.space.$3}`,
					maxWidth: "100%"
				}),
				children: [hasLogo ? /* @__PURE__ */ jsx(ApplicationLogo, { sx: (t) => ({
					width: t.space.$9,
					height: t.space.$9,
					borderRadius: t.radii.$md,
					overflow: "hidden"
				}) }) : /* @__PURE__ */ jsx(Box, {
					sx: (t) => ({
						width: t.space.$9,
						height: t.space.$9,
						flexShrink: 0,
						borderRadius: t.radii.$md,
						backgroundColor: t.colors.$primary500,
						color: t.colors.$colorPrimaryForeground,
						display: "flex",
						alignItems: "center",
						justifyContent: "center"
					}),
					"aria-hidden": true,
					children: /* @__PURE__ */ jsx(Icon, {
						icon: SvgBox,
						sx: (t) => ({
							width: t.sizes.$4,
							height: t.sizes.$4
						})
					})
				}), /* @__PURE__ */ jsxs(Col, {
					sx: { minWidth: 0 },
					children: [/* @__PURE__ */ jsx(Text, {
						as: "p",
						truncate: true,
						children: applicationName
					}), organizationSettings.enabled && /* @__PURE__ */ jsx(OrganizationSubtitle, {})]
				})]
			})
		}), /* @__PURE__ */ jsxs(Col, {
			ref: contentRef,
			elementDescriptor: descriptors.scrollBox,
			sx: (t) => ({
				backgroundColor: t.colors.$colorBackground,
				position: "relative",
				borderRadius: t.radii.$lg,
				width: "100%",
				overflow: "hidden",
				borderWidth: t.borderWidths.$normal,
				borderStyle: t.borderStyles.$solid,
				borderColor: t.colors.$borderAlpha150,
				flex: 1
			}),
			children: [/* @__PURE__ */ jsx(ConfigureSSOMobileNavbar, {}), children]
		})]
	});
};
const ConfigureSSOMobileNavbar = () => {
	const { parsedOptions } = useAppearance();
	const { organizationSettings, displayConfig: { applicationName, logoImageUrl } } = useEnvironment();
	const hasLogo = Boolean(parsedOptions.logoImageUrl || logoImageUrl);
	return /* @__PURE__ */ jsxs(Col, {
		as: "header",
		elementDescriptor: descriptors.configureSSOMobileNavbar,
		sx: (t) => ({
			display: "none",
			[mqu.md]: { display: "flex" },
			gap: t.space.$4,
			padding: t.space.$5,
			borderBottomWidth: t.borderWidths.$normal,
			borderBottomStyle: t.borderStyles.$solid,
			borderBottomColor: t.colors.$borderAlpha100
		}),
		children: [/* @__PURE__ */ jsxs(Flex, {
			align: "center",
			sx: (t) => ({
				gap: t.space.$2,
				maxWidth: "100%"
			}),
			children: [hasLogo ? /* @__PURE__ */ jsx(ApplicationLogo, { sx: (t) => ({
				width: t.space.$9,
				height: t.space.$9,
				borderRadius: t.radii.$md,
				overflow: "hidden"
			}) }) : /* @__PURE__ */ jsx(Box, {
				sx: (t) => ({
					width: t.space.$9,
					height: t.space.$9,
					flexShrink: 0,
					borderRadius: t.radii.$md,
					backgroundColor: t.colors.$primary500,
					color: t.colors.$colorPrimaryForeground,
					display: "flex",
					alignItems: "center",
					justifyContent: "center"
				}),
				"aria-hidden": true,
				children: /* @__PURE__ */ jsx(Icon, {
					icon: SvgBox,
					sx: (t) => ({
						width: t.sizes.$4,
						height: t.sizes.$4
					})
				})
			}), /* @__PURE__ */ jsxs(Col, {
				sx: { minWidth: 0 },
				children: [/* @__PURE__ */ jsx(Text, {
					as: "p",
					truncate: true,
					children: applicationName
				}), organizationSettings.enabled && /* @__PURE__ */ jsx(OrganizationSubtitle, {})]
			})]
		}), /* @__PURE__ */ jsx(Heading, {
			as: "h3",
			localizationKey: localizationKeys("configureSSO.navbar.title"),
			sx: (t) => ({ fontSize: t.fontSizes.$lg })
		})]
	});
};
const OrganizationSubtitle = () => {
	const organization = __internal_useOrganizationBase();
	if (!organization) return null;
	return /* @__PURE__ */ jsx(Text, {
		as: "span",
		truncate: true,
		sx: (t) => ({ color: t.colors.$colorMutedForeground }),
		children: organization?.name
	});
};

//#endregion
export { ConfigureSSONavbar };
