// @clerk/ui/dist/components/ConfigureSSO/ResetConnectionDialog.js
import { localizationKeys } from "../../localization/localizationKeys.js";
import { useFormControl } from "../../utils/useFormControl.js";
import { descriptors } from "../../customizables/elementDescriptors.js";
import { useCardState, withCardStateProvider } from "../../elements/contexts/index.js";
import { Col } from "../../customizables/index.js";
import { Modal } from "../../elements/Modal.js";
import { Card } from "../../elements/Card/index.js";
import { handleError } from "../../utils/errorHandler.js";
import { Form } from "../../elements/Form.js";
import { FormButtonContainer } from "../../elements/FormButtons.js";
import { FormContainer } from "../../elements/FormContainer.js";
import { jsx, jsxs } from "@emotion/react/jsx-runtime";

//#region src/components/ConfigureSSO/ResetConnectionDialog.tsx
const ResetConnectionDialog = (props) => {
	if (!props.isOpen) return null;
	return /* @__PURE__ */ jsx(Modal, {
		handleClose: props.onClose,
		canCloseModal: false,
		portalRoot: props.contentRef,
		containerSx: (t) => ({
			alignItems: "center",
			position: "absolute",
			inset: 0,
			width: "auto",
			height: "auto",
			backgroundColor: "inherit",
			backdropFilter: `blur(${t.sizes.$2})`
		}),
		children: /* @__PURE__ */ jsx(ResetConnectionDialogContent, { ...props })
	});
};
const ResetConnectionDialogContent = withCardStateProvider((props) => {
	const { onClose, onDelete, confirmationValue } = props;
	const title = props.title ?? localizationKeys("configureSSO.resetConnectionDialog.title");
	const subtitle = props.subtitle ?? localizationKeys("configureSSO.resetConnectionDialog.subtitle");
	const confirmButtonLabel = props.confirmButtonLabel ?? localizationKeys("configureSSO.resetConnectionDialog.resetButton");
	const card = useCardState();
	const confirmationField = useFormControl("deleteConfirmation", "", {
		type: "text",
		label: localizationKeys("configureSSO.resetConnectionDialog.confirmationFieldLabel", { name: confirmationValue }),
		isRequired: true,
		placeholder: confirmationValue
	});
	const canSubmit = Boolean(confirmationValue && confirmationField.value === confirmationValue);
	const onSubmit = async () => {
		if (!canSubmit) return;
		try {
			await onDelete();
			onClose();
		} catch (err) {
			handleError(err, [confirmationField], card.setError);
		}
	};
	return /* @__PURE__ */ jsx(Card.Root, {
		elementDescriptor: descriptors.configureSSOResetConnectionDialog,
		sx: (t) => ({ borderRadius: t.radii.$md }),
		children: /* @__PURE__ */ jsx(Card.Content, {
			sx: (t) => ({
				textAlign: "start",
				padding: t.sizes.$5
			}),
			children: /* @__PURE__ */ jsx(FormContainer, {
				headerTitle: title,
				headerSubtitle: subtitle,
				sx: (t) => ({ gap: t.space.$4 }),
				children: /* @__PURE__ */ jsx(Form.Root, {
					onSubmit,
					children: /* @__PURE__ */ jsxs(Col, {
						gap: 4,
						children: [/* @__PURE__ */ jsx(Form.ControlRow, {
							elementId: confirmationField.id,
							children: /* @__PURE__ */ jsx(Form.PlainInput, {
								...confirmationField.props,
								elementDescriptor: descriptors.configureSSOResetConnectionDialogConfirmationInput,
								ignorePasswordManager: true
							})
						}), /* @__PURE__ */ jsxs(FormButtonContainer, { children: [/* @__PURE__ */ jsx(Form.SubmitButton, {
							elementDescriptor: descriptors.configureSSOResetConnectionDialogSubmitButton,
							block: false,
							colorScheme: "danger",
							isDisabled: !canSubmit,
							localizationKey: confirmButtonLabel
						}), /* @__PURE__ */ jsx(Form.ResetButton, {
							elementDescriptor: descriptors.configureSSOResetConnectionDialogCancelButton,
							block: false,
							localizationKey: localizationKeys("configureSSO.resetConnectionDialog.cancelButton"),
							onClick: onClose
						})] })]
					})
				})
			})
		})
	});
});

//#endregion
export { ResetConnectionDialog };
