// @clerk/ui/dist/components/ConfigureSSO/steps/ConfigureStep/saml/SamlCustomConfigureSteps.js
import { localizationKeys } from "../../../../../localization/localizationKeys.js";
import SvgCheckmark from "../../../../../icons/checkmark.js";
import SvgClipboard from "../../../../../icons/clipboard.js";
import { useFormControl } from "../../../../../utils/useFormControl.js";
import { descriptors } from "../../../../../customizables/elementDescriptors.js";
import { useCardState } from "../../../../../elements/contexts/index.js";
import { Badge, Col, Flex, Table, Tbody, Td, Text, Th, Thead, Tr } from "../../../../../customizables/index.js";
import { Form } from "../../../../../elements/Form.js";
import { ClipboardInput } from "../../../../../elements/ClipboardInput.js";
import { useConfigureSSO } from "../../../ConfigureSSOContext.js";
import { Step } from "../../../elements/Step.js";
import { useWizard } from "../../../elements/Wizard/WizardContext.js";
import { Wizard } from "../../../elements/Wizard/Wizard.js";
import { InnerStepCounter } from "../../../elements/Wizard/InnerStepCounter.js";
import { IdentityProviderConfigurationForm, applySamlSubmitError, buildSamlConfigurationPayload } from "./shared/IdentityProviderConfigurationForm.js";
import { IdentityProviderConfigurationModes } from "./shared/IdentityProviderConfigurationModes.js";
import React from "react";
import { Fragment as Fragment$1, jsx, jsxs } from "@emotion/react/jsx-runtime";

//#region src/components/ConfigureSSO/steps/ConfigureStep/saml/SamlCustomConfigureSteps.tsx
const CUSTOM_STEPS = [
	{ id: "create-app" },
	{ id: "attribute-mapping" },
	{ id: "assign-users" },
	{ id: "identity-provider-metadata" }
];
const SamlCustomConfigureSteps = () => {
	return /* @__PURE__ */ jsxs(Wizard, {
		steps: CUSTOM_STEPS,
		initialStepId: CUSTOM_STEPS[0].id,
		children: [
			/* @__PURE__ */ jsx(Wizard.Match, {
				id: "create-app",
				children: /* @__PURE__ */ jsx(SamlCustomCreateAppStep, {})
			}),
			/* @__PURE__ */ jsx(Wizard.Match, {
				id: "attribute-mapping",
				children: /* @__PURE__ */ jsx(SamlCustomAttributeMappingStep, {})
			}),
			/* @__PURE__ */ jsx(Wizard.Match, {
				id: "assign-users",
				children: /* @__PURE__ */ jsx(SamlCustomAssignUsersStep, {})
			}),
			/* @__PURE__ */ jsx(Wizard.Match, {
				id: "identity-provider-metadata",
				children: /* @__PURE__ */ jsx(SamlCustomIdentityProviderMetadataStep, {})
			})
		]
	});
};
const SamlCustomCreateAppStep = () => {
	const { goNext, goPrev, isFirstStep, isLastStep } = useWizard();
	const { enterpriseConnection } = useConfigureSSO();
	const acsUrl = enterpriseConnection?.samlConnection?.acsUrl ?? "";
	const spEntityId = enterpriseConnection?.samlConnection?.spEntityId ?? "";
	const acsUrlField = useFormControl("acsUrl", acsUrl, {
		type: "text",
		label: localizationKeys("configureSSO.configureStep.samlCustom.createAppStep.serviceProviderFields.acsUrl.label"),
		isRequired: false
	});
	const spEntityIdField = useFormControl("spEntityId", spEntityId, {
		type: "text",
		label: localizationKeys("configureSSO.configureStep.samlCustom.createAppStep.serviceProviderFields.spEntityId.label"),
		isRequired: false
	});
	return /* @__PURE__ */ jsxs(Fragment$1, { children: [
		/* @__PURE__ */ jsx(Step.Header, {
			title: localizationKeys("configureSSO.configureStep.samlCustom.mainHeaderTitle"),
			description: localizationKeys("configureSSO.configureStep.samlCustom.createAppStep.headerSubtitle"),
			children: /* @__PURE__ */ jsx(InnerStepCounter, {})
		}),
		/* @__PURE__ */ jsx(Step.Body, { children: /* @__PURE__ */ jsxs(Step.Section, {
			sx: (theme) => ({ gap: theme.space.$5 }),
			children: [
				/* @__PURE__ */ jsx(Col, {
					sx: (theme) => ({ gap: theme.space.$1x5 }),
					children: /* @__PURE__ */ jsx(Text, {
						as: "p",
						colorScheme: "secondary",
						localizationKey: localizationKeys("configureSSO.configureStep.samlCustom.createAppStep.createAppInstructions.paragraph")
					})
				}),
				/* @__PURE__ */ jsx(Form.ControlRow, {
					elementId: acsUrlField.id,
					children: /* @__PURE__ */ jsx(Form.CommonInputWrapper, {
						...acsUrlField.props,
						children: /* @__PURE__ */ jsx(ClipboardInput, {
							value: acsUrl,
							readOnly: true,
							copyIcon: SvgClipboard,
							copiedIcon: SvgCheckmark
						})
					})
				}),
				/* @__PURE__ */ jsx(Form.ControlRow, {
					elementId: spEntityIdField.id,
					children: /* @__PURE__ */ jsx(Form.CommonInputWrapper, {
						...spEntityIdField.props,
						children: /* @__PURE__ */ jsx(ClipboardInput, {
							value: spEntityId,
							readOnly: true,
							copyIcon: SvgClipboard,
							copiedIcon: SvgCheckmark
						})
					})
				})
			]
		}) }),
		/* @__PURE__ */ jsxs(Step.Footer, { children: [
			/* @__PURE__ */ jsx(Step.Footer.Reset, {}),
			/* @__PURE__ */ jsx(Step.Footer.Previous, {
				onClick: () => goPrev(),
				isDisabled: isFirstStep
			}),
			/* @__PURE__ */ jsx(Step.Footer.Continue, {
				onClick: () => goNext(),
				isDisabled: isLastStep
			})
		] })
	] });
};
const SamlCustomAttributeMappingStep = () => {
	const { goNext, goPrev, isFirstStep, isLastStep } = useWizard();
	return /* @__PURE__ */ jsxs(Fragment$1, { children: [
		/* @__PURE__ */ jsx(Step.Header, {
			title: localizationKeys("configureSSO.configureStep.samlCustom.mainHeaderTitle"),
			description: localizationKeys("configureSSO.configureStep.samlCustom.attributeMappingStep.headerSubtitle"),
			children: /* @__PURE__ */ jsx(InnerStepCounter, {})
		}),
		/* @__PURE__ */ jsx(Step.Body, { children: /* @__PURE__ */ jsxs(Step.Section, {
			sx: (theme) => ({ gap: theme.space.$3 }),
			children: [/* @__PURE__ */ jsx(Text, {
				as: "p",
				colorScheme: "secondary",
				localizationKey: localizationKeys("configureSSO.configureStep.samlCustom.attributeMappingStep.paragraph")
			}), /* @__PURE__ */ jsx(CustomAttributeMappingTable, {})]
		}) }),
		/* @__PURE__ */ jsxs(Step.Footer, { children: [
			/* @__PURE__ */ jsx(Step.Footer.Reset, {}),
			/* @__PURE__ */ jsx(Step.Footer.Previous, {
				onClick: () => goPrev(),
				isDisabled: isFirstStep
			}),
			/* @__PURE__ */ jsx(Step.Footer.Continue, {
				onClick: () => goNext(),
				isDisabled: isLastStep
			})
		] })
	] });
};
const CUSTOM_ATTRIBUTE_ROWS = [
	{
		id: "email",
		isRequired: true
	},
	{
		id: "firstName",
		isRequired: false
	},
	{
		id: "lastName",
		isRequired: false
	}
];
const CustomAttributeMappingTable = () => /* @__PURE__ */ jsxs(Table, {
	elementDescriptor: descriptors.configureSSOAttributeMappingTable,
	sx: (theme) => ({ "tr > th:first-of-type": { paddingInlineStart: theme.space.$4 } }),
	children: [/* @__PURE__ */ jsx(Thead, { children: /* @__PURE__ */ jsxs(Tr, { children: [/* @__PURE__ */ jsx(Th, { children: /* @__PURE__ */ jsx(Text, {
		sx: (theme) => ({ fontSize: theme.fontSizes.$xs }),
		localizationKey: localizationKeys("configureSSO.configureStep.samlCustom.attributeMappingStep.attributeMappingTable.columns.attributeName")
	}) }), /* @__PURE__ */ jsx(Th, { children: /* @__PURE__ */ jsx(Text, {
		sx: (theme) => ({ fontSize: theme.fontSizes.$xs }),
		localizationKey: localizationKeys("configureSSO.configureStep.samlCustom.attributeMappingStep.attributeMappingTable.columns.userAttribute")
	}) })] }) }), /* @__PURE__ */ jsx(Tbody, { children: CUSTOM_ATTRIBUTE_ROWS.map((row) => /* @__PURE__ */ jsxs(Tr, { children: [/* @__PURE__ */ jsx(Td, { children: /* @__PURE__ */ jsxs(Flex, {
		as: "span",
		align: "center",
		sx: (theme) => ({ gap: theme.space.$2 }),
		children: [/* @__PURE__ */ jsx(Text, {
			as: "span",
			colorScheme: "secondary",
			sx: { fontFamily: "monospace" },
			localizationKey: localizationKeys(`configureSSO.configureStep.samlCustom.attributeMappingStep.attributeMappingTable.rows.${row.id}.userAttribute`)
		}), /* @__PURE__ */ jsx(Badge, {
			elementDescriptor: descriptors.configureSSOAttributeMappingBadge,
			elementId: descriptors.configureSSOAttributeMappingBadge.setId(row.isRequired ? "required" : "optional"),
			colorScheme: row.isRequired ? "warning" : "primary",
			localizationKey: localizationKeys(row.isRequired ? "configureSSO.configureStep.attributeMappingTable.badges.required" : "configureSSO.configureStep.attributeMappingTable.badges.optional")
		})]
	}) }), /* @__PURE__ */ jsx(Td, { children: /* @__PURE__ */ jsx(Text, {
		as: "span",
		localizationKey: localizationKeys(`configureSSO.configureStep.samlCustom.attributeMappingStep.attributeMappingTable.rows.${row.id}.attributeName`)
	}) })] }, row.id)) })]
});
const SamlCustomAssignUsersStep = () => {
	const { goNext, goPrev, isFirstStep, isLastStep } = useWizard();
	return /* @__PURE__ */ jsxs(Fragment$1, { children: [
		/* @__PURE__ */ jsx(Step.Header, {
			title: localizationKeys("configureSSO.configureStep.samlCustom.mainHeaderTitle"),
			description: localizationKeys("configureSSO.configureStep.samlCustom.assignUsersStep.headerSubtitle"),
			children: /* @__PURE__ */ jsx(InnerStepCounter, {})
		}),
		/* @__PURE__ */ jsx(Step.Body, { children: /* @__PURE__ */ jsx(Step.Section, {
			sx: (theme) => ({ gap: theme.space.$3 }),
			children: /* @__PURE__ */ jsx(Text, {
				as: "p",
				colorScheme: "secondary",
				localizationKey: localizationKeys("configureSSO.configureStep.samlCustom.assignUsersStep.paragraph")
			})
		}) }),
		/* @__PURE__ */ jsxs(Step.Footer, { children: [
			/* @__PURE__ */ jsx(Step.Footer.Reset, {}),
			/* @__PURE__ */ jsx(Step.Footer.Previous, {
				onClick: () => goPrev(),
				isDisabled: isFirstStep
			}),
			/* @__PURE__ */ jsx(Step.Footer.Continue, {
				onClick: () => goNext(),
				isDisabled: isLastStep
			})
		] })
	] });
};
const CUSTOM_SAML_IDP_MODES = ["metadataUrl", "manual"];
const SamlCustomIdentityProviderMetadataStep = () => {
	const card = useCardState();
	const { goNext, goPrev, isFirstStep } = useWizard();
	const { enterpriseConnection, enterpriseConnectionMutations: { updateConnection } } = useConfigureSSO();
	const samlConnection = enterpriseConnection?.samlConnection;
	const hasExistingConfig = Boolean(samlConnection?.idpSsoUrl || samlConnection?.idpEntityId || samlConnection?.idpCertificate || samlConnection?.idpMetadataUrl);
	const existingCertPresent = Boolean(samlConnection?.idpCertificate);
	const [mode, setMode] = React.useState(hasExistingConfig ? "manual" : "metadataUrl");
	const [certFile, setCertFile] = React.useState(null);
	const [isSubmitting, setIsSubmitting] = React.useState(false);
	const metadataUrlField = useFormControl("idpMetadataUrl", samlConnection?.idpMetadataUrl ?? "", {
		type: "text",
		label: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.metadataUrl.label"),
		placeholder: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.metadataUrl.placeholder"),
		isRequired: true
	});
	const signOnUrlField = useFormControl("idpSsoUrl", samlConnection?.idpSsoUrl ?? "", {
		type: "text",
		label: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.manual.signOnUrl.label"),
		placeholder: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.manual.signOnUrl.placeholder"),
		isRequired: true
	});
	const issuerField = useFormControl("idpEntityId", samlConnection?.idpEntityId ?? "", {
		type: "text",
		label: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.manual.issuer.label"),
		placeholder: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.manual.issuer.placeholder"),
		isRequired: true
	});
	const certificateField = useFormControl("idpCertificate", "", {
		type: "text",
		label: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.manual.signingCertificate.label"),
		isRequired: true
	});
	const trimmedMetadataUrl = metadataUrlField.value.trim();
	const trimmedSignOnUrl = signOnUrlField.value.trim();
	const trimmedIssuer = issuerField.value.trim();
	const hasCert = certFile !== null || existingCertPresent;
	const canSubmit = (mode === "metadataUrl" ? trimmedMetadataUrl.length > 0 : trimmedSignOnUrl.length > 0 && trimmedIssuer.length > 0 && hasCert) && !isSubmitting;
	const formProps = mode === "metadataUrl" ? {
		mode: "metadataUrl",
		form: { field: metadataUrlField },
		labels: { description: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.metadataUrl.description") }
	} : {
		mode: "manual",
		form: {
			signOnUrlField,
			issuerField,
			certificateField,
			certFile,
			onCertFileChange: setCertFile,
			existingCertPresent
		},
		labels: {
			description: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.manual.description"),
			uploadFile: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.manual.signingCertificate.uploadFile"),
			replaceFile: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.manual.signingCertificate.replaceFile"),
			removeFile: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.manual.signingCertificate.removeFile"),
			fileUploaded: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.manual.signingCertificate.fileUploaded")
		}
	};
	const handleContinue = async () => {
		if (!enterpriseConnection || !canSubmit) return;
		card.setError(void 0);
		setIsSubmitting(true);
		try {
			const saml = await buildSamlConfigurationPayload({
				mode,
				metadataUrl: { value: metadataUrlField.value },
				manual: {
					signOnUrl: signOnUrlField.value,
					issuer: issuerField.value,
					certFile
				}
			});
			await updateConnection(enterpriseConnection.id, { saml });
			goNext();
		} catch (err) {
			if (mode === "metadataUrl") applySamlSubmitError(err, card, metadataUrlField);
			else applySamlSubmitError(err, card, signOnUrlField, [issuerField, certificateField]);
			setIsSubmitting(false);
		}
	};
	return /* @__PURE__ */ jsxs(Fragment$1, { children: [
		/* @__PURE__ */ jsx(Step.Header, {
			title: localizationKeys("configureSSO.configureStep.samlCustom.mainHeaderTitle"),
			description: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.headerSubtitle"),
			children: /* @__PURE__ */ jsx(InnerStepCounter, {})
		}),
		/* @__PURE__ */ jsx(Step.Body, { children: /* @__PURE__ */ jsxs(Step.Section, {
			fill: true,
			gap: 5,
			children: [/* @__PURE__ */ jsx(IdentityProviderConfigurationModes, {
				modes: CUSTOM_SAML_IDP_MODES,
				value: mode,
				onChange: (next) => {
					card.setError(void 0);
					setMode(next);
				},
				labels: {
					ariaLabel: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.modes.ariaLabel"),
					metadataUrl: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.modes.metadataUrl"),
					manual: localizationKeys("configureSSO.configureStep.samlCustom.identityProviderMetadataStep.modes.manual")
				}
			}), /* @__PURE__ */ jsx(IdentityProviderConfigurationForm, { ...formProps })]
		}) }),
		/* @__PURE__ */ jsxs(Step.Footer, { children: [
			/* @__PURE__ */ jsx(Step.Footer.Reset, {}),
			/* @__PURE__ */ jsx(Step.Footer.Previous, {
				onClick: () => goPrev(),
				isDisabled: isFirstStep || isSubmitting
			}),
			/* @__PURE__ */ jsx(Step.Footer.Continue, {
				onClick: handleContinue,
				isLoading: isSubmitting,
				isDisabled: !canSubmit
			})
		] })
	] });
};

//#endregion
export { SamlCustomConfigureSteps };
