// @clerk/ui/dist/elements/ProfileCard/ProfileCardPage.js
import { mqu } from "../../styledSystem/breakpoints.js";
import { Col } from "../../customizables/index.js";
import { Fragment, jsx } from "@emotion/react/jsx-runtime";

//#region src/elements/ProfileCard/ProfileCardPage.tsx
/**
* Per-page padding wrapper rendered inside `ProfileCardContent`
*
* Each routed page inside `UserProfile` / `OrganizationProfile` should wrap its content
* in this component
*/
const ProfileCardPage = ({ children, padding = true, bleeding = false, sx }) => {
	if (!padding && !bleeding) return /* @__PURE__ */ jsx(Fragment, { children });
	return /* @__PURE__ */ jsx(Col, {
		sx: [(theme) => ({
			...padding && {
				paddingTop: theme.space.$7,
				paddingBottom: theme.space.$7,
				paddingInlineStart: theme.space.$8,
				paddingInlineEnd: theme.space.$6,
				[mqu.sm]: { padding: `${theme.space.$8} ${theme.space.$5}` }
			},
			...bleeding && {
				marginInlineStart: `calc(${theme.space.$8} * -1)`,
				marginInlineEnd: `calc(${theme.space.$6} * -1)`,
				[mqu.sm]: { marginInline: `calc(${theme.space.$5} * -1)` }
			}
		}), sx],
		children
	});
};

//#endregion
export { ProfileCardPage };
