// @clerk/ui/dist/components/ConfigureSSO/steps/OrganizationDomainsStep.js
import { localizationKeys } from "../../../localization/localizationKeys.js";
import SvgCheckmark from "../../../icons/checkmark.js";
import SvgClipboard from "../../../icons/clipboard.js";
import SvgClose from "../../../icons/close.js";
import SvgRotateLeftRight from "../../../icons/rotate-left-right.js";
import { useLocalizations } from "../../../localization/makeLocalizable.js";
import { common } from "../../../styledSystem/common.js";
import { useFormControl } from "../../../utils/useFormControl.js";
import { descriptors } from "../../../customizables/elementDescriptors.js";
import { useCardState } from "../../../elements/contexts/index.js";
import { Flow } from "../../../customizables/Flow.js";
import { Badge, Box, Button, Col, Flex, Icon, Spinner, Text } from "../../../customizables/index.js";
import { Alert } from "../../../elements/Alert.js";
import { getFieldError, getGlobalError } from "../../../utils/errorHandler.js";
import { Field } from "../../../elements/FieldControl.js";
import { Form } from "../../../elements/Form.js";
import { Animated } from "../../../elements/Animated.js";
import { ClipboardInput } from "../../../elements/ClipboardInput.js";
import { Tooltip } from "../../../elements/Tooltip.js";
import { useConfigureSSO } from "../ConfigureSSOContext.js";
import { Step } from "../elements/Step.js";
import { useWizard } from "../elements/Wizard/WizardContext.js";
import { areAllOrganizationDomainsVerified } from "../domain/organizationEnterpriseConnection.js";
import { RemoveDomainDialog } from "../RemoveDomainDialog.js";
import { useEffect, useRef, useState } from "react";
import { useClerk, useOrganization, useUser } from "@clerk/shared/react";
import { jsx, jsxs } from "@emotion/react/jsx-runtime";
import { eventFlowStepMounted } from "@clerk/shared/telemetry";

//#region src/components/ConfigureSSO/steps/OrganizationDomainsStep.tsx
const OrganizationDomainsStep = () => {
	const { t } = useLocalizations();
	const { enterpriseConnection, organizationDomains, organizationEnterpriseConnection, contentRef, organizationDomainMutations: { createDomain, revalidate, prepareOwnershipVerification }, enterpriseConnectionMutations: { updateConnection } } = useConfigureSSO();
	const { goPrev, goNext, isFirstStep, isLastStep } = useWizard();
	const card = useCardState();
	const clerk = useClerk();
	const { organization } = useOrganization();
	const [domainToRemove, setDomainToRemove] = useState(null);
	const hasRecordedTelemetryEvent = useRef(false);
	useEffect(() => {
		if (hasRecordedTelemetryEvent.current) return;
		hasRecordedTelemetryEvent.current = true;
		clerk.telemetry?.record(eventFlowStepMounted("configureSSO", "verify-domain", {
			timestamp: (/* @__PURE__ */ new Date()).toISOString(),
			connectionStatus: organizationEnterpriseConnection.status,
			connectionId: enterpriseConnection?.id ?? null,
			organizationId: organization?.id ?? null
		}));
	}, []);
	const handleCreateDomain = async (domain) => {
		card.setError(void 0);
		try {
			await createDomain(domain);
		} catch (err) {
			const apiError = getFieldError(err) ?? getGlobalError(err);
			card.setError(apiError);
		}
	};
	const handlePrepareOwnershipVerification = async (domain) => {
		card.setError(void 0);
		try {
			await prepareOwnershipVerification([domain]);
		} catch (err) {
			const apiError = getFieldError(err) ?? getGlobalError(err);
			card.setError(apiError);
		}
	};
	const handleRemoveDomain = async (domain) => {
		if (enterpriseConnection) {
			const domains = enterpriseConnection.domains.filter((name) => name !== domain.name);
			await updateConnection(enterpriseConnection.id, { domains });
		}
		await domain.delete();
		await revalidate();
	};
	const hasAllDomainsVerified = areAllOrganizationDomainsVerified(organizationDomains);
	const verifiedDomainCount = organizationDomains?.filter((domain) => domain.ownershipVerification?.status === "verified").length ?? 0;
	const lockLastVerifiedDomain = Boolean(enterpriseConnection);
	const lastVerifiedDomainTooltip = enterpriseConnection?.active ? localizationKeys("configureSSO.organizationDomainsStep.domainCard.removeButtonTooltip__lastVerifiedDomainActive") : localizationKeys("configureSSO.organizationDomainsStep.domainCard.removeButtonTooltip__lastVerifiedDomain");
	return /* @__PURE__ */ jsxs(Flow.Part, {
		part: "organizationDomains",
		children: [/* @__PURE__ */ jsxs(Step, {
			elementDescriptor: descriptors.configureSSOStep,
			elementId: descriptors.configureSSOStep.setId("verify-domain"),
			children: [
				/* @__PURE__ */ jsx(Step.Header, {
					title: t(localizationKeys("configureSSO.organizationDomainsStep.title")),
					description: t(localizationKeys("configureSSO.organizationDomainsStep.subtitle"))
				}),
				/* @__PURE__ */ jsx(Step.Body, { children: /* @__PURE__ */ jsxs(Step.Section, {
					fill: true,
					sx: (t) => ({
						gap: t.space.$5,
						minHeight: 0
					}),
					children: [/* @__PURE__ */ jsxs(Col, {
						sx: (t) => ({ gap: t.space.$4 }),
						children: [
							/* @__PURE__ */ jsx(DomainsField, {
								onSubmit: handleCreateDomain,
								organizationDomains
							}),
							!organizationDomains?.length && /* @__PURE__ */ jsx(DomainSuggestion, { onSubmit: handleCreateDomain }),
							card.error && /* @__PURE__ */ jsx(Alert, {
								variant: "danger",
								title: card.error
							})
						]
					}), !!organizationDomains?.length && /* @__PURE__ */ jsx(Col, {
						elementDescriptor: descriptors.configureSSOVerifyDomainList,
						sx: (t) => ({
							gap: t.space.$3,
							flex: "0 1 auto",
							minHeight: 0,
							overflowY: "auto",
							marginInline: `calc(${t.space.$1} * -1)`,
							paddingInline: t.space.$1,
							...common.unstyledScrollbar(t)
						}),
						children: organizationDomains.map((domain) => {
							const isLastVerifiedDomain = domain.ownershipVerification?.status === "verified" && verifiedDomainCount === 1;
							return /* @__PURE__ */ jsx(DomainCard, {
								domain,
								onRemove: () => setDomainToRemove(domain),
								onPrepareOwnershipVerification: () => handlePrepareOwnershipVerification(domain),
								isRemoveDisabled: lockLastVerifiedDomain && isLastVerifiedDomain,
								removeDisabledTooltip: lastVerifiedDomainTooltip
							}, domain.id);
						})
					})]
				}) }),
				/* @__PURE__ */ jsxs(Step.Footer, { children: [/* @__PURE__ */ jsx(Step.Footer.Previous, {
					onClick: () => goPrev(),
					isDisabled: isFirstStep
				}), /* @__PURE__ */ jsx(Step.Footer.Continue, {
					onClick: () => goNext(),
					isDisabled: isLastStep || !hasAllDomainsVerified
				})] })
			]
		}), domainToRemove && /* @__PURE__ */ jsx(RemoveDomainDialog, {
			isOpen: !!domainToRemove,
			onClose: () => setDomainToRemove(null),
			domain: domainToRemove.name,
			isConnectionActive: Boolean(enterpriseConnection?.active),
			onRemove: () => handleRemoveDomain(domainToRemove),
			contentRef
		})]
	});
};
/**
* Matches a bare domain such as `example.com` or `sub.example.co.uk`.
* Each label must start and end with an alphanumeric character and a valid
* TLD of at least two letters is required. Protocols, paths, ports, spaces
* and single-label hostnames are rejected.
*/
const DOMAIN_REGEX = /^(?:[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\.)+[a-z]{2,}$/i;
const isValidDomain = (value) => DOMAIN_REGEX.test(value);
const DomainsField = ({ onSubmit, organizationDomains }) => {
	const [isSubmitting, setIsSubmitting] = useState(false);
	const domainField = useFormControl("domain", "", {
		type: "text",
		label: localizationKeys("configureSSO.organizationDomainsStep.formFieldLabel__domain"),
		placeholder: localizationKeys("configureSSO.organizationDomainsStep.formFieldInputPlaceholder__domain")
	});
	const domain = domainField.value.trim().toLowerCase();
	const canSubmit = isValidDomain(domain) && !organizationDomains?.some((d) => d.name === domain) && !isSubmitting;
	const handleSubmit = (e) => {
		e.preventDefault();
		if (!canSubmit) return;
		setIsSubmitting(true);
		onSubmit(domain).then(() => domainField.setValue("")).finally(() => setIsSubmitting(false));
	};
	return /* @__PURE__ */ jsx(Form.Root, {
		onSubmit: handleSubmit,
		children: /* @__PURE__ */ jsx(Field.Root, {
			...domainField.props,
			children: /* @__PURE__ */ jsxs(Col, {
				elementDescriptor: descriptors.formField,
				elementId: descriptors.formField.setId(domainField.id),
				sx: (t) => ({ gap: t.space.$2 }),
				children: [/* @__PURE__ */ jsx(Field.LabelRow, { children: /* @__PURE__ */ jsx(Field.Label, {}) }), /* @__PURE__ */ jsxs(Flex, {
					align: "start",
					sx: (t) => ({ gap: t.space.$2 }),
					children: [/* @__PURE__ */ jsx(Box, {
						sx: { flex: 1 },
						children: /* @__PURE__ */ jsx(Field.Input, {})
					}), /* @__PURE__ */ jsx(Button, {
						type: "submit",
						variant: "bordered",
						colorScheme: "secondary",
						isDisabled: !canSubmit,
						isLoading: isSubmitting,
						localizationKey: localizationKeys("configureSSO.organizationDomainsStep.formButtonPrimary__add"),
						sx: { flexShrink: 0 }
					})]
				})]
			})
		})
	});
};
const getEmailDomain = (email) => email.split("@")[1]?.trim().toLowerCase() || null;
const DomainSuggestion = ({ onSubmit }) => {
	const { user } = useUser();
	const [isDismissed, setIsDismissed] = useState(false);
	const [isSubmitting, setIsSubmitting] = useState(false);
	const primaryEmail = user?.primaryEmailAddress?.emailAddress;
	const domain = primaryEmail ? getEmailDomain(primaryEmail) : null;
	if (!domain || isDismissed) return null;
	const handleAdd = () => {
		setIsSubmitting(true);
		onSubmit(domain).then(() => setIsDismissed(true)).finally(() => setIsSubmitting(false));
	};
	return /* @__PURE__ */ jsxs(Flex, {
		elementDescriptor: descriptors.configureSSOVerifyDomainSuggestion,
		align: "center",
		justify: "between",
		sx: (t) => ({
			gap: t.space.$2,
			paddingInline: t.space.$3,
			paddingBlock: t.space.$1x5,
			borderWidth: t.borderWidths.$normal,
			borderStyle: t.borderStyles.$solid,
			borderColor: t.colors.$borderAlpha150,
			borderRadius: t.radii.$lg,
			background: t.colors.$neutralAlpha50
		}),
		children: [/* @__PURE__ */ jsxs(Flex, {
			align: "center",
			sx: (t) => ({
				gap: t.space.$3,
				minWidth: 0
			}),
			children: [/* @__PURE__ */ jsx(Text, {
				as: "span",
				colorScheme: "secondary",
				localizationKey: localizationKeys("configureSSO.organizationDomainsStep.domainSuggestion.messageLabel", { domain }),
				sx: (t) => ({ fontSize: t.fontSizes.$sm })
			}), /* @__PURE__ */ jsx(Button, {
				variant: "bordered",
				colorScheme: "secondary",
				size: "xs",
				isLoading: isSubmitting,
				onClick: handleAdd,
				localizationKey: localizationKeys("configureSSO.organizationDomainsStep.domainSuggestion.formButtonPrimary__add", { domain }),
				sx: { flexShrink: 0 }
			})]
		}), /* @__PURE__ */ jsx(Button, {
			variant: "ghost",
			colorScheme: "neutral",
			"aria-label": "Dismiss domain suggestion",
			isDisabled: isSubmitting,
			onClick: () => setIsDismissed(true),
			sx: (t) => ({
				flexShrink: 0,
				padding: t.space.$1
			}),
			children: /* @__PURE__ */ jsx(Icon, {
				icon: SvgClose,
				sx: (t) => ({
					width: t.sizes.$4,
					height: t.sizes.$4,
					color: t.colors.$colorMutedForeground
				})
			})
		})]
	});
};
const DomainCard = ({ domain, onRemove, onPrepareOwnershipVerification, isRemoveDisabled = false, removeDisabledTooltip }) => {
	if (!domain.name) return null;
	const ownershipVerification = domain.ownershipVerification;
	const isVerified = ownershipVerification?.status === "verified";
	const isExpired = ownershipVerification?.status === "expired";
	const cardId = ownershipVerification?.status ?? "unverified";
	const removeButton = /* @__PURE__ */ jsx(Button, {
		elementDescriptor: descriptors.configureSSOVerifyDomainCardRemoveButton,
		variant: "ghost",
		colorScheme: "neutral",
		"aria-label": "Remove domain",
		onClick: onRemove,
		isDisabled: isRemoveDisabled,
		sx: (t) => ({
			flexShrink: 0,
			padding: t.space.$1
		}),
		children: /* @__PURE__ */ jsx(Icon, {
			icon: SvgClose,
			sx: (t) => ({
				width: t.sizes.$4,
				height: t.sizes.$4,
				color: t.colors.$colorMutedForeground
			})
		})
	});
	return /* @__PURE__ */ jsxs(Col, {
		elementDescriptor: descriptors.configureSSOVerifyDomainCard,
		elementId: descriptors.configureSSOVerifyDomainCard.setId(cardId),
		sx: (t) => ({
			borderWidth: t.borderWidths.$normal,
			borderStyle: t.borderStyles.$solid,
			borderColor: t.colors.$borderAlpha150,
			borderRadius: t.radii.$lg,
			background: t.colors.$colorBackground
		}),
		children: [/* @__PURE__ */ jsxs(Flex, {
			align: "center",
			justify: "between",
			sx: (t) => ({
				gap: t.space.$2,
				padding: t.space.$4,
				paddingBottom: t.space.$2
			}),
			children: [/* @__PURE__ */ jsxs(Flex, {
				align: "center",
				sx: (t) => ({
					gap: t.space.$2,
					minWidth: 0
				}),
				children: [
					/* @__PURE__ */ jsx(Text, {
						as: "span",
						sx: (t) => ({
							fontWeight: t.fontWeights.$medium,
							whiteSpace: "nowrap",
							overflow: "hidden",
							textOverflow: "ellipsis"
						}),
						children: domain.name
					}),
					/* @__PURE__ */ jsx(Badge, {
						elementDescriptor: descriptors.configureSSOVerifyDomainCardBadge,
						elementId: descriptors.configureSSOVerifyDomainCardBadge.setId(cardId),
						colorScheme: isVerified ? "success" : "danger",
						localizationKey: isVerified ? localizationKeys("configureSSO.organizationDomainsStep.domainCard.badge__verified") : isExpired ? localizationKeys("configureSSO.organizationDomainsStep.domainCard.badge__expired") : localizationKeys("configureSSO.organizationDomainsStep.domainCard.badge__unverified")
					}),
					!isVerified && !isExpired && /* @__PURE__ */ jsx(Spinner, {
						size: "xs",
						colorScheme: "neutral",
						sx: (t) => ({
							flexShrink: 0,
							marginInlineStart: t.space.$1
						})
					})
				]
			}), isRemoveDisabled && removeDisabledTooltip ? /* @__PURE__ */ jsxs(Tooltip.Root, { children: [/* @__PURE__ */ jsx(Tooltip.Trigger, { children: removeButton }), /* @__PURE__ */ jsx(Tooltip.Content, { text: removeDisabledTooltip })] }) : removeButton]
		}), /* @__PURE__ */ jsx(Box, {
			sx: { overflow: "hidden" },
			children: /* @__PURE__ */ jsx(Animated, { children: isExpired ? /* @__PURE__ */ jsx(ExpiredNotice, {
				expiresAt: ownershipVerification?.expiresAt ?? null,
				onPrepareOwnershipVerification
			}, "expired") : ownershipVerification?.verifiedAt ? /* @__PURE__ */ jsx(Text, {
				as: "p",
				colorScheme: "secondary",
				localizationKey: localizationKeys("configureSSO.organizationDomainsStep.domainCard.verifiedAtLabel", { date: ownershipVerification.verifiedAt }),
				sx: (t) => ({
					padding: t.space.$4,
					paddingTop: 0
				})
			}, "verified") : /* @__PURE__ */ jsx(TxtRecord, { ownershipVerification }, "unverified") })
		})]
	});
};
const ExpiredNotice = ({ expiresAt, onPrepareOwnershipVerification }) => {
	const [isVerifying, setIsVerifying] = useState(false);
	const handleVerifyAgain = () => {
		setIsVerifying(true);
		onPrepareOwnershipVerification().finally(() => setIsVerifying(false));
	};
	return /* @__PURE__ */ jsxs(Col, {
		elementDescriptor: descriptors.configureSSOVerifyDomainCardExpired,
		sx: (t) => ({
			gap: t.space.$3,
			paddingInline: t.space.$4,
			paddingBottom: t.space.$4
		}),
		children: [/* @__PURE__ */ jsx(Text, {
			as: "p",
			colorScheme: "secondary",
			localizationKey: expiresAt ? localizationKeys("configureSSO.organizationDomainsStep.domainCard.expiredAtLabel", { date: expiresAt }) : localizationKeys("configureSSO.organizationDomainsStep.domainCard.expiredLabel")
		}), /* @__PURE__ */ jsxs(Button, {
			variant: "bordered",
			colorScheme: "secondary",
			size: "xs",
			isLoading: isVerifying,
			onClick: handleVerifyAgain,
			sx: (t) => ({
				alignSelf: "flex-start",
				gap: t.space.$1x5
			}),
			children: [/* @__PURE__ */ jsx(Icon, {
				icon: SvgRotateLeftRight,
				size: "sm",
				colorScheme: "neutral"
			}), /* @__PURE__ */ jsx(Text, {
				as: "span",
				localizationKey: localizationKeys("configureSSO.organizationDomainsStep.domainCard.verifyAgainButton")
			})]
		})]
	});
};
const TxtRecord = ({ ownershipVerification }) => {
	return /* @__PURE__ */ jsxs(Col, {
		elementDescriptor: descriptors.configureSSOVerifyDomainCardTxtRecord,
		sx: (t) => ({
			gap: t.space.$3,
			paddingInline: t.space.$4,
			paddingBottom: t.space.$4
		}),
		children: [
			/* @__PURE__ */ jsx(Text, {
				as: "p",
				colorScheme: "secondary",
				localizationKey: localizationKeys("configureSSO.organizationDomainsStep.domainCard.txtRecord.instructions"),
				sx: (t) => ({ fontSize: t.fontSizes.$sm })
			}),
			/* @__PURE__ */ jsx(Box, { sx: (t) => ({
				borderTopWidth: t.borderWidths.$normal,
				borderTopStyle: t.borderStyles.$solid,
				borderTopColor: t.colors.$borderAlpha100,
				marginInline: `calc(${t.space.$4} * -1)`
			}) }),
			/* @__PURE__ */ jsxs(Flex, {
				wrap: "wrap",
				sx: (t) => ({ gap: t.space.$6 }),
				children: [/* @__PURE__ */ jsx(RecordEntry, {
					label: localizationKeys("configureSSO.organizationDomainsStep.domainCard.txtRecord.typeLabel"),
					value: "TXT"
				}), /* @__PURE__ */ jsx(RecordEntry, {
					label: localizationKeys("configureSSO.organizationDomainsStep.domainCard.txtRecord.hostLabel"),
					value: ownershipVerification?.txtRecordName ?? "@"
				})]
			}),
			/* @__PURE__ */ jsxs(Flex, {
				align: "center",
				sx: (t) => ({
					gap: t.space.$2,
					minWidth: 0
				}),
				children: [/* @__PURE__ */ jsx(Text, {
					as: "span",
					colorScheme: "secondary",
					localizationKey: localizationKeys("configureSSO.organizationDomainsStep.domainCard.txtRecord.valueLabel"),
					sx: (t) => ({
						fontSize: t.fontSizes.$sm,
						flexShrink: 0
					})
				}), /* @__PURE__ */ jsx(ClipboardInput, {
					elementDescriptor: descriptors.configureSSOVerifyDomainCardTxtRecordValue,
					value: ownershipVerification?.txtRecordValue ?? "—",
					copyIcon: SvgClipboard,
					copiedIcon: SvgCheckmark,
					sx: {
						flex: 1,
						minWidth: 0
					}
				})]
			})
		]
	});
};
const RecordEntry = ({ label, value }) => {
	return /* @__PURE__ */ jsxs(Flex, {
		align: "center",
		sx: (t) => ({
			gap: t.space.$2,
			minWidth: 0
		}),
		children: [/* @__PURE__ */ jsx(Text, {
			as: "span",
			colorScheme: "secondary",
			localizationKey: label,
			sx: (t) => ({
				fontSize: t.fontSizes.$sm,
				flexShrink: 0
			})
		}), /* @__PURE__ */ jsx(Badge, {
			colorScheme: "primary",
			sx: (t) => ({
				fontFamily: t.fonts.$buttons,
				display: "flex",
				alignItems: "center",
				justifyContent: "center",
				paddingBlock: t.space.$1,
				paddingInline: t.space.$1x5
			}),
			children: /* @__PURE__ */ jsx(Box, {
				as: "span",
				sx: {
					display: "flex",
					alignItems: "center",
					justifyContent: "center",
					lineHeight: 1,
					textBoxTrim: "trim-both",
					textBoxEdge: "cap alphabetic"
				},
				children: value
			})
		})]
	});
};

//#endregion
export { OrganizationDomainsStep };
