// @clerk/ui/dist/components/OrganizationProfile/SecuritySsoSection.js
import { useEnvironment } from "../../contexts/EnvironmentContext.js";
import { localizationKeys } from "../../localization/localizationKeys.js";
import SvgInformationCircle from "../../icons/information-circle.js";
import { useLocalizations } from "../../localization/makeLocalizable.js";
import { descriptors } from "../../customizables/elementDescriptors.js";
import { CardStateProvider, useCardState } from "../../elements/contexts/index.js";
import { Badge, Button, Col, Flex, Icon, Text } from "../../customizables/index.js";
import { Card } from "../../elements/Card/index.js";
import { handleError } from "../../utils/errorHandler.js";
import { ThreeDotsMenu } from "../../elements/ThreeDotsMenu.js";
import { ProfileSection } from "../../elements/Section.js";
import { Tooltip } from "../../elements/Tooltip.js";
import { ResetConnectionDialog } from "../ConfigureSSO/ResetConnectionDialog.js";
import { useFetchRoles, useLocalizeCustomRoles } from "../../hooks/useFetchRoles.js";
import { useState } from "react";
import { jsx, jsxs } from "@emotion/react/jsx-runtime";

//#region src/components/OrganizationProfile/SecuritySsoSection.tsx
const STATUS_BADGES = {
	unconfigured: {
		id: "unconfigured",
		colorScheme: "primary",
		label: localizationKeys("organizationProfile.securityPage.ssoSection.badge__unconfigured")
	},
	in_progress: {
		id: "inProgress",
		colorScheme: "warning",
		label: localizationKeys("organizationProfile.securityPage.ssoSection.badge__inProgress")
	},
	active: {
		id: "active",
		colorScheme: "success",
		label: localizationKeys("organizationProfile.securityPage.ssoSection.badge__active")
	},
	inactive: {
		id: "inactive",
		colorScheme: "danger",
		label: localizationKeys("organizationProfile.securityPage.ssoSection.badge__inactive")
	}
};
const SecuritySsoSection = (props) => {
	const { connection, onConfigure } = props;
	const isConfigured = connection.status === "active" || connection.status === "inactive";
	const status = connection.status;
	const badge = STATUS_BADGES[status];
	return /* @__PURE__ */ jsxs(ProfileSection.Root, {
		title: localizationKeys("organizationProfile.securityPage.ssoSection.title"),
		id: "sso",
		centered: false,
		badge: /* @__PURE__ */ jsx(Badge, {
			elementDescriptor: descriptors.organizationProfileSecuritySsoBadge,
			elementId: descriptors.organizationProfileSecuritySsoBadge.setId(badge.id),
			colorScheme: badge.colorScheme,
			localizationKey: badge.label
		}),
		children: [
			status === "unconfigured" && /* @__PURE__ */ jsx(NotConfiguredContent, {
				primaryButtonKey: localizationKeys("organizationProfile.securityPage.ssoSection.primaryButton__startConfiguration"),
				primaryButtonId: "start",
				onConfigure: () => onConfigure(true)
			}),
			status === "in_progress" && /* @__PURE__ */ jsx(NotConfiguredContent, {
				primaryButtonKey: localizationKeys("organizationProfile.securityPage.ssoSection.primaryButton__continueConfiguration"),
				primaryButtonId: "continue",
				onConfigure: () => onConfigure()
			}),
			isConfigured && /* @__PURE__ */ jsx(CardStateProvider, { children: /* @__PURE__ */ jsx(ConfiguredContent, {
				...props,
				isActive: status === "active"
			}) })
		]
	});
};
const NotConfiguredContent = ({ primaryButtonKey, primaryButtonId, onConfigure }) => /* @__PURE__ */ jsxs(Col, {
	align: "start",
	gap: 4,
	children: [/* @__PURE__ */ jsx(SsoDescription, {}), /* @__PURE__ */ jsx(Button, {
		elementDescriptor: descriptors.organizationProfileSecuritySsoConfigureButton,
		elementId: descriptors.organizationProfileSecuritySsoConfigureButton.setId(primaryButtonId),
		variant: "bordered",
		colorScheme: "secondary",
		size: "sm",
		onClick: onConfigure,
		localizationKey: primaryButtonKey
	})]
});
const ConfiguredContent = (props) => {
	const { enterpriseConnection, setConnectionActive, deleteConnection, organizationName, contentRef, onConfigure, isActive } = props;
	const card = useCardState();
	const [isResetDialogOpen, setIsResetDialogOpen] = useState(false);
	const domains = enterpriseConnection?.domains ?? [];
	const onSetActive = async (active) => {
		if (card.isLoading) return;
		card.setError(void 0);
		card.setLoading();
		try {
			await setConnectionActive(enterpriseConnection.id, active);
		} catch (err) {
			handleError(err, [], card.setError);
		} finally {
			card.setIdle();
		}
	};
	return /* @__PURE__ */ jsxs(Col, {
		gap: 4,
		children: [
			/* @__PURE__ */ jsxs(Flex, {
				align: "start",
				justify: "between",
				gap: 3,
				children: [/* @__PURE__ */ jsx(SsoDescription, {}), /* @__PURE__ */ jsx(ThreeDotsMenu, {
					elementId: "sso",
					actions: [
						{
							label: localizationKeys("organizationProfile.securityPage.ssoSection.menuAction__edit"),
							onClick: () => onConfigure(true)
						},
						isActive ? {
							label: localizationKeys("organizationProfile.securityPage.ssoSection.menuAction__deactivate"),
							isDisabled: card.isLoading,
							onClick: () => void onSetActive(false)
						} : {
							label: localizationKeys("organizationProfile.securityPage.ssoSection.menuAction__activate"),
							isDisabled: card.isLoading,
							onClick: () => void onSetActive(true)
						},
						{
							label: localizationKeys("organizationProfile.securityPage.ssoSection.menuAction__remove"),
							isDestructive: true,
							onClick: () => setIsResetDialogOpen(true)
						}
					]
				})]
			}),
			/* @__PURE__ */ jsx(Card.Alert, { children: card.error }),
			domains.length > 0 && /* @__PURE__ */ jsxs(Flex, {
				sx: (t) => ({ gap: t.space.$1x5 }),
				children: [/* @__PURE__ */ jsx(Text, {
					elementDescriptor: descriptors.organizationProfileSecuritySsoDetailRowLabel,
					elementId: descriptors.organizationProfileSecuritySsoDetailRowLabel.setId("domain"),
					colorScheme: "secondary",
					localizationKey: localizationKeys("organizationProfile.securityPage.ssoSection.domainLabel")
				}), /* @__PURE__ */ jsx(Flex, {
					align: "center",
					wrap: "wrap",
					sx: (t) => ({
						minWidth: 0,
						gap: t.space.$1x5
					}),
					children: domains.map((domain) => /* @__PURE__ */ jsx(Badge, {
						elementDescriptor: descriptors.organizationProfileSecuritySsoDetailRowChip,
						children: domain
					}, domain))
				})]
			}),
			/* @__PURE__ */ jsx(ResetConnectionDialog, {
				isOpen: isResetDialogOpen,
				onClose: () => setIsResetDialogOpen(false),
				confirmationValue: organizationName,
				title: localizationKeys("organizationProfile.securityPage.removeDialog.title"),
				subtitle: localizationKeys("organizationProfile.securityPage.removeDialog.subtitle"),
				confirmButtonLabel: localizationKeys("organizationProfile.securityPage.removeDialog.confirmButton"),
				onDelete: () => deleteConnection(enterpriseConnection.id),
				contentRef
			})
		]
	});
};
const SsoDescription = () => {
	const roleName = useEnrollmentRoleName();
	const { t } = useLocalizations();
	return /* @__PURE__ */ jsxs(Text, {
		as: "p",
		children: [/* @__PURE__ */ jsx(Text, {
			as: "span",
			elementDescriptor: descriptors.organizationProfileSecuritySsoDescription,
			colorScheme: "secondary",
			localizationKey: localizationKeys("organizationProfile.securityPage.ssoSection.descriptionLine1")
		}), /* @__PURE__ */ jsxs(Tooltip.Root, { children: [/* @__PURE__ */ jsx(Tooltip.Trigger, { children: /* @__PURE__ */ jsx(Button, {
			variant: "unstyled",
			"aria-label": t(localizationKeys("organizationProfile.securityPage.ssoSection.tooltipLabel")),
			sx: (t) => ({
				display: "inline-flex",
				alignItems: "center",
				verticalAlign: "middle",
				padding: 0,
				height: "fit-content",
				borderRadius: t.radii.$sm,
				color: t.colors.$colorMutedForeground,
				marginInlineStart: t.space.$1
			}),
			children: /* @__PURE__ */ jsx(Icon, {
				icon: SvgInformationCircle,
				"aria-hidden": true,
				sx: (t) => ({
					width: t.sizes.$4,
					height: t.sizes.$4
				})
			})
		}) }), /* @__PURE__ */ jsx(Tooltip.Content, { text: roleName ? localizationKeys("organizationProfile.securityPage.ssoSection.tooltip", { role: roleName }) : localizationKeys("organizationProfile.securityPage.ssoSection.tooltip__noRole") })] })]
	});
};
/**
* The display name of the role SSO-enrolled members are assigned — the environment's
* default member role, name-mapped when the roles list is readable.
*/
const useEnrollmentRoleName = () => {
	const { organizationSettings } = useEnvironment();
	const { options } = useFetchRoles();
	const { localizeCustomRole } = useLocalizeCustomRoles();
	let roleKey = organizationSettings.domains.defaultRole ?? void 0;
	if (!roleKey && options?.length === 1) roleKey = options[0].value;
	if (!roleKey) return;
	return localizeCustomRole(roleKey) || options?.find((option) => option.value === roleKey)?.label || humanizeRoleKey(roleKey);
};
/** `org:billing_admin` → `billing admin`. */
const humanizeRoleKey = (roleKey) => {
	return (roleKey.split(":").pop() ?? roleKey).replace(/[_-]+/g, " ").trim() || roleKey;
};

//#endregion
export { SecuritySsoSection };
