// @clerk/localizations/dist/index.mjs
import { arSA } from "./ar-SA.mjs";
import { beBY } from "./be-BY.mjs";
import { bgBG } from "./bg-BG.mjs";
import { bnIN } from "./bn-IN.mjs";
import { caES } from "./ca-ES.mjs";
import { csCZ } from "./cs-CZ.mjs";
import { daDK } from "./da-DK.mjs";
import { deDE } from "./de-DE.mjs";
import { elGR } from "./el-GR.mjs";
import { enGB } from "./en-GB.mjs";
import { enUS } from "./en-US.mjs";
import { enXA } from "./en-XA.mjs";
import { esCR } from "./es-CR.mjs";
import { esES } from "./es-ES.mjs";
import { esMX } from "./es-MX.mjs";
import { esUY } from "./es-UY.mjs";
import { faIR } from "./fa-IR.mjs";
import { fiFI } from "./fi-FI.mjs";
import { frFR } from "./fr-FR.mjs";
import { heIL } from "./he-IL.mjs";
import { hiIN } from "./hi-IN.mjs";
import { hrHR } from "./hr-HR.mjs";
import { huHU } from "./hu-HU.mjs";
import { idID } from "./id-ID.mjs";
import { isIS } from "./is-IS.mjs";
import { itIT } from "./it-IT.mjs";
import { jaJP } from "./ja-JP.mjs";
import { kkKZ } from "./kk-KZ.mjs";
import { koKR } from "./ko-KR.mjs";
import { mnMN } from "./mn-MN.mjs";
import { msMY } from "./ms-MY.mjs";
import { nbNO } from "./nb-NO.mjs";
import { nlBE } from "./nl-BE.mjs";
import { nlNL } from "./nl-NL.mjs";
import { plPL } from "./pl-PL.mjs";
import { ptBR } from "./pt-BR.mjs";
import { ptPT } from "./pt-PT.mjs";
import { roRO } from "./ro-RO.mjs";
import { ruRU } from "./ru-RU.mjs";
import { skSK } from "./sk-SK.mjs";
import { srRS } from "./sr-RS.mjs";
import { svSE } from "./sv-SE.mjs";
import { taIN } from "./ta-IN.mjs";
import { teIN } from "./te-IN.mjs";
import { thTH } from "./th-TH.mjs";
import { trTR } from "./tr-TR.mjs";
import { ukUA } from "./uk-UA.mjs";
import { viVN } from "./vi-VN.mjs";
import { zhCN } from "./zh-CN.mjs";
import { zhTW } from "./zh-TW.mjs";

export { arSA, beBY, bgBG, bnIN, caES, csCZ, daDK, deDE, elGR, enGB, enUS, enXA, esCR, esES, esMX, esUY, faIR, fiFI, frFR, heIL, hiIN, hrHR, huHU, idID, isIS, itIT, jaJP, kkKZ, koKR, mnMN, msMY, nbNO, nlBE, nlNL, plPL, ptBR, ptPT, roRO, ruRU, skSK, srRS, svSE, taIN, teIN, thTH, trTR, ukUA, viVN, zhCN, zhTW };
