// @clerk/localizations/dist/kk-KZ.js
Object.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });

//#region src/kk-KZ.ts
const kkKZ = {
	locale: "kk-KZ",
	apiKeys: {
		action__add: void 0,
		action__search: void 0,
		copySecret: {
			formButtonPrimary__copyAndClose: "Көшіру және жабу",
			formHint: "Қауіпсіздік себептерімен, біз сізге кейінірек оны қайта көруге рұқсат бере алмаймыз.",
			formTitle: "API кілтіңізді \"{{name}}\" қазір көшіріңіз"
		},
		createdAndExpirationStatus__expiresOn: void 0,
		createdAndExpirationStatus__never: void 0,
		detailsTitle__emptyRow: void 0,
		formButtonPrimary__add: void 0,
		formFieldCaption__expiration__expiresOn: void 0,
		formFieldCaption__expiration__never: void 0,
		formFieldOption__expiration__180d: void 0,
		formFieldOption__expiration__1d: void 0,
		formFieldOption__expiration__1y: void 0,
		formFieldOption__expiration__30d: void 0,
		formFieldOption__expiration__60d: void 0,
		formFieldOption__expiration__7d: void 0,
		formFieldOption__expiration__90d: void 0,
		formFieldOption__expiration__never: void 0,
		formHint: void 0,
		formTitle: void 0,
		lastUsed__days: void 0,
		lastUsed__hours: void 0,
		lastUsed__minutes: void 0,
		lastUsed__months: void 0,
		lastUsed__seconds: void 0,
		lastUsed__years: void 0,
		menuAction__revoke: void 0,
		revokeConfirmation: {
			confirmationText: void 0,
			formButtonPrimary__revoke: void 0,
			formHint: void 0,
			formTitle: void 0,
			inputLabel: void 0
		},
		tableHeader__actions: void 0,
		tableHeader__lastUsed: void 0,
		tableHeader__name: void 0
	},
	backButton: "Артқа",
	badge__activePlan: void 0,
	badge__banned: void 0,
	badge__canceledEndsAt: void 0,
	badge__currentPlan: "Ағымдағы жоспар",
	badge__default: "Әдепкі бойынша",
	badge__endsAt: "{{ date | shortDate('kk-KZ') }} күні аяқталады",
	badge__expired: "Мерзімі біткен",
	badge__freeTrial: void 0,
	badge__otherImpersonatorDevice: "Басқа эмуляциялаушы құрылғы",
	badge__pastDueAt: void 0,
	badge__pastDuePlan: void 0,
	badge__primary: "Негізгі",
	badge__renewsAt: void 0,
	badge__requiresAction: "Әрекет қажет",
	badge__startsAt: "{{ date | shortDate('kk-KZ') }} күні басталады",
	badge__thisDevice: "Осы құрылғы",
	badge__trialEndsAt: void 0,
	badge__unverified: "Расталмаған",
	badge__upcomingPlan: "Алдағы жоспар",
	badge__userDevice: "Пайдаланушы құрылғысы",
	badge__you: "Сіз",
	billing: {
		accountCredit: void 0,
		addPaymentMethod__label: void 0,
		alwaysFree: void 0,
		annually: void 0,
		availableFeatures: void 0,
		billedAnnually: "Жыл сайын ақы төленеді",
		billedAnnuallyOnly: void 0,
		billedMonthly: void 0,
		billedMonthlyOnly: void 0,
		cancelFreeTrial: void 0,
		cancelFreeTrialAccessUntil: void 0,
		cancelFreeTrialTitle: void 0,
		cancelSubscription: "Жазылымды болдырмау",
		cancelSubscriptionAccessUntil: void 0,
		cancelSubscriptionNoCharge: void 0,
		cancelSubscriptionPastDue: void 0,
		cancelSubscriptionTitle: void 0,
		cannotSubscribeMonthly: void 0,
		cannotSubscribeUnrecoverable: void 0,
		checkout: {
			description__paymentSuccessful: "Сіздің жаңа жазылымыңыз дайын.",
			description__subscriptionSuccessful: "Сіздің жаңа жазылымыңыз дайын.",
			downgradeNotice: void 0,
			emailForm: {
				subtitle: void 0,
				title: void 0
			},
			lineItems: {
				title__freeTrialEndsAt: void 0,
				title__paymentMethod: "Төлем әдісі",
				title__statementId: "Шоттың ID нөмірі",
				title__subscriptionBegins: "Жазылым басталады",
				title__totalPaid: "Жалпы төленді"
			},
			pastDueNotice: void 0,
			perMonth: void 0,
			title: void 0,
			title__paymentSuccessful: "Төлем сәтті аяқталды!",
			title__subscriptionSuccessful: "Сәтті!",
			title__trialSuccess: void 0,
			totalDueAfterTrial: void 0,
			totalDuePerPeriod: void 0
		},
		credit: void 0,
		creditRemainder: void 0,
		defaultFreePlanActive: void 0,
		free: "Тегін",
		getStarted: "Бастау",
		highlightedPlanBadge: void 0,
		keepFreeTrial: void 0,
		keepSubscription: "Жазылымды сақтау",
		manage: "Басқару",
		manageSubscription: "Жазылымды басқару",
		month: "Ай",
		monthAbbreviation: void 0,
		monthPerUnit: void 0,
		monthly: void 0,
		pastDue: void 0,
		pay: void 0,
		payerCreditRemainder: void 0,
		paymentMethod: {
			applePayDescription: {
				annual: void 0,
				monthly: void 0
			},
			dev: {
				anyNumbers: void 0,
				cardNumber: void 0,
				cvcZip: void 0,
				developmentMode: void 0,
				expirationDate: void 0,
				testCardInfo: void 0
			}
		},
		paymentMethods__label: void 0,
		pricingTable: {
			billingCycle: void 0,
			included: void 0,
			seatCost: {
				additionalSeats: void 0,
				freeUpToSeats: void 0,
				includedSeats: void 0,
				perSeat: void 0,
				tooltip: {
					additionalSeatsEach: void 0,
					firstSeatsIncludedInPlan: void 0,
					freeForUpToSeats: void 0
				},
				unlimitedSeats: void 0,
				upToSeats: void 0
			}
		},
		proratedDiscount: void 0,
		prorationCredit: void 0,
		reSubscribe: "Қайта жазылу",
		seatBreakdownIncludedPlural: void 0,
		seatBreakdownIncludedSingular: void 0,
		seatBreakdownPlural: void 0,
		seatBreakdownSingular: void 0,
		seats: void 0,
		seatsWithLimit: void 0,
		seeAllFeatures: void 0,
		startFreeTrial: void 0,
		startFreeTrial__days: void 0,
		subscribe: void 0,
		subscriptionDetails: {
			beginsOn: void 0,
			currentBillingCycle: void 0,
			endsOn: void 0,
			firstPaymentAmount: void 0,
			firstPaymentOn: void 0,
			nextPaymentAmount: void 0,
			nextPaymentOn: void 0,
			pastDueAt: void 0,
			renewsAt: void 0,
			subscribedOn: void 0,
			title: void 0,
			trialEndsOn: void 0,
			trialStartedOn: void 0
		},
		subtotal: void 0,
		subtotalRenewal: void 0,
		switchPlan: "Осы жоспарға ауысу",
		switchToAnnual: void 0,
		switchToAnnualWithAnnualPrice: void 0,
		switchToMonthly: void 0,
		switchToMonthlyWithPrice: void 0,
		totalDue: void 0,
		totalDuePerPeriod: void 0,
		totalDueToday: void 0,
		viewFeatures: void 0,
		viewPayment: void 0,
		year: void 0,
		yearAbbreviation: void 0,
		yearPerUnit: void 0
	},
	configureSSO: {
		activate: {
			activateButton: void 0,
			activeSubtitle: void 0,
			activeTitle: void 0,
			doneButton: void 0,
			skipButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		changeProviderDialog: {
			cancelButton: void 0,
			confirmButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		configureStep: {
			activeConnectionWarning: {
				dismiss: void 0,
				title: void 0
			},
			attributeMappingTable: { badges: {
				optional: void 0,
				required: void 0
			} },
			samlCustom: {
				assignUsersStep: {
					headerSubtitle: void 0,
					paragraph: void 0
				},
				attributeMappingStep: {
					attributeMappingTable: {
						columns: {
							attributeName: void 0,
							userAttribute: void 0
						},
						rows: {
							email: {
								attributeName: void 0,
								userAttribute: void 0
							},
							firstName: {
								attributeName: void 0,
								userAttribute: void 0
							},
							lastName: {
								attributeName: void 0,
								userAttribute: void 0
							}
						}
					},
					headerSubtitle: void 0,
					paragraph: void 0
				},
				createAppStep: {
					createAppInstructions: { paragraph: void 0 },
					headerSubtitle: void 0,
					serviceProviderFields: {
						acsUrl: { label: void 0 },
						spEntityId: { label: void 0 }
					}
				},
				identityProviderMetadataStep: {
					headerSubtitle: void 0,
					manual: {
						description: void 0,
						issuer: {
							label: void 0,
							placeholder: void 0
						},
						signOnUrl: {
							label: void 0,
							placeholder: void 0
						},
						signingCertificate: {
							fileUploaded: void 0,
							label: void 0,
							removeFile: void 0,
							replaceFile: void 0,
							uploadFile: void 0
						}
					},
					metadataUrl: {
						description: void 0,
						label: void 0,
						placeholder: void 0
					},
					modes: {
						ariaLabel: void 0,
						manual: void 0,
						metadataUrl: void 0
					}
				},
				mainHeaderTitle: void 0
			},
			samlGoogle: {
				attributeMappingStep: {
					attributeMappingTable: {
						columns: {
							appAttribute: void 0,
							googleAttribute: void 0
						},
						rows: {
							email: {
								appAttribute: void 0,
								googleAttribute: void 0
							},
							firstName: {
								appAttribute: void 0,
								googleAttribute: void 0
							},
							lastName: {
								appAttribute: void 0,
								googleAttribute: void 0
							}
						}
					},
					headerSubtitle: void 0,
					paragraph: void 0,
					step1: void 0,
					step2: void 0
				},
				configureUserAccess: {
					assignUsersInstructions: {
						paragraph1: void 0,
						paragraph2: void 0,
						step1: void 0,
						step2: void 0,
						step3: void 0
					},
					headerSubtitle: void 0
				},
				createAppStep: {
					createAppInstructions: {
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: void 0,
						title: void 0
					},
					headerSubtitle: void 0
				},
				identityProviderMetadataStep: {
					headerSubtitle: void 0,
					manual: {
						description: void 0,
						issuer: {
							label: void 0,
							placeholder: void 0
						},
						signOnUrl: {
							label: void 0,
							placeholder: void 0
						},
						signingCertificate: {
							fileUploaded: void 0,
							label: void 0,
							removeFile: void 0,
							replaceFile: void 0,
							uploadFile: void 0
						}
					},
					metadataFile: {
						description: void 0,
						fileUploaded: void 0,
						label: void 0,
						removeFile: void 0,
						replaceFile: void 0,
						uploadFile: void 0
					},
					modes: {
						ariaLabel: void 0,
						manual: void 0,
						metadataFile: void 0
					}
				},
				mainHeaderTitle: void 0,
				serviceProviderStep: {
					headerSubtitle: void 0,
					nameIdInstructions: {
						step1: void 0,
						step2: void 0
					},
					paragraph: void 0,
					serviceProviderFields: {
						acsUrl: { label: void 0 },
						spEntityId: { label: void 0 }
					},
					title: void 0
				}
			},
			samlMicrosoft: {
				attributeMappingStep: {
					attributeMappingTable: {
						columns: {
							attribute: void 0,
							claimName: void 0,
							value: void 0
						},
						copyClaimName: void 0,
						copyClaimNameCopied: void 0,
						rows: {
							email: {
								attribute: void 0,
								claimName: void 0,
								value: void 0
							},
							firstName: {
								attribute: void 0,
								claimName: void 0,
								value: void 0
							},
							lastName: {
								attribute: void 0,
								claimName: void 0,
								value: void 0
							}
						}
					},
					headerSubtitle: void 0,
					step1: void 0,
					step2: void 0,
					title: void 0
				},
				createAppStep: {
					assignUsersInstructions: {
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: void 0,
						step5: void 0,
						title: void 0
					},
					createAppInstructions: {
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: {
							label: void 0,
							subSteps: {
								appName: void 0,
								create: void 0,
								nonGallery: void 0
							}
						},
						title: void 0
					},
					headerSubtitle: void 0
				},
				identityProviderMetadataStep: {
					headerSubtitle: void 0,
					manual: {
						description: void 0,
						issuer: {
							label: void 0,
							placeholder: void 0
						},
						signOnUrl: {
							label: void 0,
							placeholder: void 0
						},
						signingCertificate: {
							fileUploaded: void 0,
							label: void 0,
							removeFile: void 0,
							replaceFile: void 0,
							uploadFile: void 0
						}
					},
					metadataUrl: {
						description: void 0,
						label: void 0,
						placeholder: void 0
					},
					modes: {
						ariaLabel: void 0,
						manual: void 0,
						metadataUrl: void 0
					}
				},
				mainHeaderTitle: void 0,
				serviceProviderStep: {
					headerSubtitle: void 0,
					serviceProviderFields: {
						acsUrl: { label: void 0 },
						spEntityId: { label: void 0 }
					},
					step1: void 0,
					step2: void 0,
					step3: void 0,
					step4: void 0,
					step5: void 0,
					step6: void 0,
					title: void 0
				}
			},
			samlOkta: {
				assignUsersStep: {
					assignUsersInstructions: {
						paragraph: void 0,
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: void 0,
						step5: void 0
					},
					headerSubtitle: void 0
				},
				attributeMappingStep: {
					attributeMappingTable: {
						columns: {
							expression: void 0,
							name: void 0
						},
						rows: {
							email: {
								expression: void 0,
								name: void 0
							},
							firstName: {
								expression: void 0,
								name: void 0
							},
							lastName: {
								expression: void 0,
								name: void 0
							}
						}
					},
					headerSubtitle: void 0,
					paragraph: void 0,
					step1: void 0,
					step2: void 0
				},
				createAppStep: {
					completeSamlIntegrationInstructions: {
						step1: void 0,
						step2: void 0,
						title: void 0
					},
					createAppInstructions: {
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: void 0,
						title: void 0
					},
					headerSubtitle: void 0,
					serviceProviderInstructions: {
						paragraph1: void 0,
						paragraph2: void 0,
						serviceProviderFields: {
							acsUrl: { label: void 0 },
							spEntityId: { label: void 0 }
						},
						title: void 0
					}
				},
				identityProviderMetadataStep: {
					headerSubtitle: void 0,
					manual: {
						description: void 0,
						issuer: {
							label: void 0,
							placeholder: void 0
						},
						signOnUrl: {
							label: void 0,
							placeholder: void 0
						},
						signingCertificate: {
							fileUploaded: void 0,
							label: void 0,
							removeFile: void 0,
							replaceFile: void 0,
							uploadFile: void 0
						}
					},
					metadataUrl: {
						description: void 0,
						label: void 0,
						placeholder: void 0
					},
					modes: {
						ariaLabel: void 0,
						manual: void 0,
						metadataUrl: void 0
					}
				},
				mainHeaderTitle: void 0
			}
		},
		missingManageEnterpriseConnectionsPermission: {
			subtitle: "Рұқсаттарыңызды жаңарту үшін ұйым әкімшісіне хабарласыңыз.",
			title: "Сізде бірыңғай кіруді (SSO) басқаруға рұқсат жоқ"
		},
		navbar: { title: "Бірыңғай кіруді конфигурациялау (SSO)" },
		organizationDomainsStep: {
			domainCard: {
				badge__expired: void 0,
				badge__unverified: "Расталмаған",
				badge__verified: "Расталған",
				expiredAtLabel: void 0,
				expiredLabel: void 0,
				removeButtonTooltip__lastVerifiedDomain: void 0,
				removeButtonTooltip__lastVerifiedDomainActive: void 0,
				txtRecord: {
					hostLabel: "Хост / Атау",
					instructions: "Бұл TXT жазбасын DNS провайдеріңізге қосыңыз. Жазба белсенді болған бойда біз оны автоматты түрде растаймыз.",
					typeLabel: "Түрі",
					valueLabel: "Мән"
				},
				verifiedAtLabel: "{{ date | shortDate('kk-KZ') }} күні расталды",
				verifyAgainButton: void 0
			},
			domainSuggestion: {
				formButtonPrimary__add: "{{domain}} қосу",
				messageLabel: "Электрондық поштаңыз {{domain}} пайдаланады. Оны қосқыңыз келе ме?"
			},
			formButtonPrimary__add: "Қосу",
			formFieldInputPlaceholder__domain: "Домен қосу",
			formFieldLabel__domain: "Домен",
			removeDomainDialog: {
				cancelButton: void 0,
				removeButton: void 0,
				subtitle__active: void 0,
				subtitle__inactive: void 0,
				title: void 0
			},
			subtitle: "Ұйымыңыз кіру үшін пайдаланатын домендердің меншігін қосып, растаңыз.",
			title: "SSO домендерін қосу"
		},
		resetConnectionDialog: {
			cancelButton: void 0,
			confirmationFieldLabel: void 0,
			confirmationFieldPlaceholder: void 0,
			resetButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		selectProviderStep: {
			saml: {
				customSaml: "Реттелетін SAML провайдері",
				google: void 0,
				groupLabel: "SAML",
				microsoft: void 0,
				okta: "Okta Workforce"
			},
			subtitle: "SSO-ны баптамақшы провайдерді таңдаңыз.",
			title: "Провайдерді таңдау",
			warning: "Провайдер таңдалғаннан кейін, конфигурация аяқталғанша өзгерте алмайсыз"
		},
		testConfigurationStep: {
			error__noSuccessfulTestRun: void 0,
			subtitle: void 0,
			testResults: {
				actionLabel__refresh: void 0,
				empty: {
					subtitle: void 0,
					title: void 0
				},
				polling: void 0,
				status__failed: void 0,
				status__pending: void 0,
				status__success: void 0,
				title: void 0
			},
			testRunDetails: {
				howToFix: {
					actionLabel__viewDocumentation: void 0,
					oauth_access_denied: { description: void 0 },
					oauth_fetch_user_error: {
						intro: void 0,
						step1: void 0,
						step2: void 0
					},
					oauth_token_exchange_error: { description: void 0 },
					saml_email_address_domain_mismatch: { description: void 0 },
					saml_response_relaystate_missing: { description: void 0 },
					saml_user_attribute_missing: {
						intro: void 0,
						step1: void 0,
						step2: void 0,
						step3: void 0
					},
					sectionTitle: void 0
				},
				parsedUserInfo: {
					email: void 0,
					firstName: void 0,
					sectionTitle: void 0
				},
				runDetails: {
					actionLabel__copied: void 0,
					actionLabel__copy: void 0,
					errorCode: void 0,
					fullMessage: void 0,
					sectionTitle: void 0,
					status: void 0,
					timestamp: void 0
				},
				title: void 0
			},
			testUrl: { actionLabel__open: void 0 },
			title: void 0
		}
	},
	createOrganization: {
		formButtonSubmit: "Ұйым құру",
		invitePage: { formButtonReset: "Өткізіп жіберу" },
		title: "Ұйым құру"
	},
	dates: {
		lastDay: "Кеше {{ date | timeString('kk-KZ') }}",
		next6Days: "{{ date | weekday('kk-KZ','long') }} {{ date | timeString('kk-KZ') }}",
		nextDay: "Ертең {{ date | timeString('kk-KZ') }}",
		numeric: "{{ date | numeric('kk-KZ') }}",
		previous6Days: "Өткен {{ date | weekday('kk-KZ','long') }} {{ date | timeString('kk-KZ') }}",
		sameDay: "Бүгін {{ date | timeString('kk-KZ') }}"
	},
	dividerText: "немесе",
	footerActionLink__alternativePhoneCodeProvider: void 0,
	footerActionLink__useAnotherMethod: "Басқа әдісті қолдану",
	footerPageLink__help: "Көмек",
	footerPageLink__privacy: "Құпиялық",
	footerPageLink__terms: "Шарттар",
	formButtonPrimary: "Жалғастыру",
	formButtonPrimary__verify: "Растау",
	formFieldAction__forgotPassword: "Құпия сөзді ұмыттыңыз ба?",
	formFieldError__matchingPasswords: "Құпия сөздер сәйкес келеді.",
	formFieldError__notMatchingPasswords: "Құпия сөздер сәйкес келмейді.",
	formFieldError__verificationLinkExpired: "Растау сілтемесінің мерзімі бітті. Жаңа сілтеме сұраңыз.",
	formFieldHintText__optional: "Қосымша",
	formFieldHintText__slug: "Slug - бұл бірегей ID. URL мекенжайларында жиі қолданылады.",
	formFieldInputPlaceholder__apiKeyDescription: void 0,
	formFieldInputPlaceholder__apiKeyExpirationDate: void 0,
	formFieldInputPlaceholder__apiKeyName: void 0,
	formFieldInputPlaceholder__backupCode: "Сақтық кодын енгізіңіз",
	formFieldInputPlaceholder__confirmDeletionUserAccount: "Есепті жою",
	formFieldInputPlaceholder__emailAddress: "Электрондық поштаны енгізіңіз",
	formFieldInputPlaceholder__emailAddress_username: "Электрондық поштаны немесе пайдаланушы атын енгізіңіз",
	formFieldInputPlaceholder__emailAddresses: void 0,
	formFieldInputPlaceholder__firstName: "Аты",
	formFieldInputPlaceholder__lastName: "Тегі",
	formFieldInputPlaceholder__organizationDomain: "webfilter.com",
	formFieldInputPlaceholder__organizationDomainEmailAddress: "siz@webfilter.com",
	formFieldInputPlaceholder__organizationName: "Ұйым атауы",
	formFieldInputPlaceholder__organizationSlug: "my-org",
	formFieldInputPlaceholder__password: "Құпия сөзді енгізіңіз",
	formFieldInputPlaceholder__phoneNumber: "Телефон нөмірін енгізіңіз",
	formFieldInputPlaceholder__signUpPassword: void 0,
	formFieldInputPlaceholder__username: void 0,
	formFieldInput__emailAddress_format: void 0,
	formFieldLabel__apiKey: "API кілті",
	formFieldLabel__apiKeyDescription: "Сипаттама",
	formFieldLabel__apiKeyExpiration: "Мерзімі",
	formFieldLabel__apiKeyName: "Құпия кілт атауы",
	formFieldLabel__automaticInvitations: "Бұл доменге автошақыруларды қосу",
	formFieldLabel__backupCode: "Сақтық коды",
	formFieldLabel__confirmDeletion: "Растау",
	formFieldLabel__confirmPassword: "Құпия сөзді растау",
	formFieldLabel__currentPassword: "Ағымдағы құпия сөз",
	formFieldLabel__emailAddress: "Электрондық пошта",
	formFieldLabel__emailAddress_username: "Электрондық пошта немесе пайдаланушы аты",
	formFieldLabel__emailAddresses: "Электрондық пошталар",
	formFieldLabel__firstName: "Аты",
	formFieldLabel__lastName: "Тегі",
	formFieldLabel__newPassword: "Жаңа құпия сөз",
	formFieldLabel__organizationDomain: "Домен",
	formFieldLabel__organizationDomainDeletePending: "Күтудегі шақыруларды жою",
	formFieldLabel__organizationDomainEmailAddress: "Растау электрондық поштасы",
	formFieldLabel__organizationDomainEmailAddressDescription: "Растау кодын алу үшін осы домен астындағы электрондық поштаны енгізіңіз.",
	formFieldLabel__organizationName: "Атауы",
	formFieldLabel__organizationSlug: "Slug",
	formFieldLabel__passkeyName: "Құпия кілт атауы",
	formFieldLabel__password: "Құпия сөз",
	formFieldLabel__phoneNumber: "Телефон нөмірі",
	formFieldLabel__role: "Рөлі",
	formFieldLabel__signOutOfOtherSessions: "Барлық басқа құрылғылардан шығу",
	formFieldLabel__username: "Пайдаланушы аты",
	identityPreviewEditButton__emailAddress: void 0,
	identityPreviewEditButton__identifier: void 0,
	identityPreviewEditButton__phoneNumber: void 0,
	impersonationFab: {
		action__signOut: "Шығу",
		title: "{{identifier}} ретінде кірдіңіз"
	},
	lastAuthenticationStrategy: "Соңғы пайдаланылған",
	maintenanceMode: "Қазір техникалық қызмет көрсету жұмыстары жүріп жатыр, бірақ бірнеше минуттан аспайды.",
	membershipRole__admin: "Әкімші",
	membershipRole__basicMember: "Мүше",
	membershipRole__guestMember: "Қонақ",
	oauthConsent: {
		action__allow: void 0,
		action__deny: void 0,
		offlineAccessNotice: void 0,
		redirectNotice: void 0,
		redirectUriModal: {
			subtitle: void 0,
			title: void 0
		},
		scopeList: { title: void 0 },
		subtitle: void 0,
		viewFullUrl: void 0,
		warning: void 0
	},
	organizationList: {
		action__createOrganization: "Ұйым құру",
		action__invitationAccept: "Қосылу",
		action__suggestionsAccept: "Қосылу сұрауы",
		createOrganization: "Ұйым құру",
		invitationAcceptedLabel: "Қосылды",
		subtitle: "{{applicationName}} қолдануды жалғастыру үшін",
		suggestionsAcceptedLabel: "Растауды күтуде",
		title: "Есептік жазбаны таңдаңыз",
		titleWithoutPersonal: "Ұйымды таңдаңыз"
	},
	organizationProfile: {
		apiKeysPage: { title: void 0 },
		badge__automaticInvitation: "Автошақырулар",
		badge__automaticSuggestion: "Автоұсыныстар",
		badge__enterpriseSso: void 0,
		badge__manualInvitation: "Автоқосылу жоқ",
		badge__unverified: "Расталмаған",
		billingPage: {
			accountCreditsSection: {
				title: void 0,
				viewHistory: void 0
			},
			creditHistoryPage: {
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				title: void 0
			},
			paymentHistorySection: {
				empty: void 0,
				notFound: void 0,
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				tableHeader__status: void 0
			},
			paymentMethodsSection: {
				actionLabel__default: "Әдепкі ету",
				actionLabel__remove: "Жою",
				add: "Жаңа төлем әдісін қосу",
				addSubtitle: "Есептік жазбаңызға жаңа төлем әдісін қосыңыз.",
				cancelButton: "Болдырмау",
				formButtonPrimary__add: "Төлем әдісін қосу",
				formButtonPrimary__pay: "{{amount}} төлеу",
				payWithTestCardButton: void 0,
				removeMethod: {
					messageLine1: "{{identifier}} есептік жазбадан жойылады.",
					messageLine2: "Бұл төлем әдісін қолдана алмайсыз және оған тәуелді жазылымдар жұмыс істемейді.",
					successMessage: "{{paymentMethod}} жойылды.",
					title: "Төлем әдісін жою"
				},
				title: "Қолжетімді опциялар"
			},
			start: {
				headerTitle__payments: void 0,
				headerTitle__plans: "Жоспарлар",
				headerTitle__statements: "Шот-фактуралар",
				headerTitle__subscriptions: "Жазылымдар"
			},
			statementsSection: {
				empty: void 0,
				itemCaption__paidForPlan: void 0,
				itemCaption__payerCredit: void 0,
				itemCaption__proratedCredit: void 0,
				itemCaption__subscribedAndPaidForPlan: void 0,
				notFound: void 0,
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				title: void 0,
				totalPaid: void 0
			},
			subscriptionsListSection: {
				actionLabel__manageSubscription: void 0,
				actionLabel__newSubscription: void 0,
				actionLabel__switchPlan: void 0,
				includedSeatsUsage: void 0,
				overview: void 0,
				paidSeatsUsage: void 0,
				seatLimit: void 0,
				seatLimitAndIncludedSeats: void 0,
				tableHeader__edit: void 0,
				tableHeader__plan: void 0,
				tableHeader__startDate: void 0,
				title: void 0
			},
			subscriptionsSection: { actionLabel__default: "Басқару" },
			switchPlansSection: { title: void 0 },
			title: "Төлемдер мен есеп айырысу"
		},
		createDomainPage: {
			subtitle: "Ұйымға автоқосылу үшін доменді қосыңыз. Осы домендегі электрондық поштасы бар пайдаланушылар автоматты түрде қосыла алады.",
			title: "Домен қосу"
		},
		invitePage: {
			detailsTitle__inviteFailed: "Келесі электрондық пошталарға шақыру жіберілмеді: {{email_addresses}}.",
			formButtonPrimary__continue: "Шақыруларды жіберу",
			formButtonPrimary__purchaseSeats: void 0,
			selectDropdown__role: "Рөлді таңдау",
			subtitle: "Бірнеше электрондық поштаны енгізіңіз немесе қойыңыз.",
			successMessage: "Шақырулар сәтті жіберілді",
			title: "Жаңа мүшелерді шақыру"
		},
		membersPage: {
			action__invite: "Шақыру",
			action__search: "Іздеу",
			activeMembersTab: {
				menuAction__remove: "Мүшені жою",
				tableHeader__actions: "Әрекеттер",
				tableHeader__joined: "Қосылды",
				tableHeader__role: "Рөл",
				tableHeader__user: "Пайдаланушы"
			},
			alerts: { roleSetMigrationInProgress: {
				subtitle: "Қолжетімді рөлдерді жаңартып жатырмыз. Бұл аяқталғаннан кейін рөлдерді қайта жаңарта аласыз.",
				title: "Рөлдер уақытша құлыпталған"
			} },
			detailsTitle__emptyRow: "Көрсету үшін мүшелер жоқ",
			invitationsTab: {
				autoInvitations: {
					headerSubtitle: "Электрондық пошта доменін қосып, пайдаланушыларды автошақыру арқылы ұйымға қосыңыз.",
					headerTitle: "Автошақырулар",
					primaryButton: "Расталған домендерді басқару"
				},
				table__emptyRow: "Көрсету үшін шақырулар жоқ"
			},
			invitedMembersTab: {
				menuAction__revoke: "Шақыруды жою",
				tableHeader__invited: "Шақырылды"
			},
			requestsTab: {
				autoSuggestions: {
					headerSubtitle: "Домен бойынша тіркелген пайдаланушылар ұйымыңызға қосылу сұрауын көре алады.",
					headerTitle: "Автоұсыныстар",
					primaryButton: "Расталған домендерді басқару"
				},
				menuAction__approve: "Қабылдау",
				menuAction__reject: "Қабылдамау",
				tableHeader__requested: "Қосылу сұралды",
				table__emptyRow: "Көрсету үшін сұраулар жоқ"
			},
			start: {
				headerTitle__invitations: "Шақырулар",
				headerTitle__members: "Мүшелер",
				headerTitle__requests: "Сұраулар"
			}
		},
		navbar: {
			apiKeys: void 0,
			billing: "Төлемдер",
			description: "Ұйымыңызды басқарыңыз.",
			general: "Жалпы",
			members: "Мүшелер",
			security: void 0,
			title: "Ұйым"
		},
		plansPage: {
			alerts: {
				noPermissionsToManageBilling: void 0,
				planMembershipLimitExceeded: void 0
			},
			title: void 0
		},
		profilePage: {
			dangerSection: {
				deleteOrganization: {
					actionDescription: "Жалғастыру үшін \"{{organizationName}}\" деп енгізіңіз.",
					messageLine1: "Бұл ұйымды жоюға сенімдісіз бе?",
					messageLine2: "Бұл әрекет қайтымсыз және қалпына келтірілмейді.",
					successMessage: "Ұйым жойылды.",
					title: "Ұйымды жою"
				},
				leaveOrganization: {
					actionDescription: "Жалғастыру үшін \"{{organizationName}}\" деп енгізіңіз.",
					messageLine1: "Бұл ұйымнан шығуға сенімдісіз бе? Ұйымға және қосымшаларға қол жеткізе алмайсыз.",
					messageLine2: "Бұл әрекет қайтымсыз.",
					successMessage: "Сіз ұйымнан шықтыңыз.",
					title: "Ұйымнан шығу"
				},
				title: "Қауіпті"
			},
			domainSection: {
				menuAction__manage: "Басқару",
				menuAction__remove: "Жою",
				menuAction__verify: "Растау",
				primaryButton: "Домен қосу",
				subtitle: "Расталған домен арқылы пайдаланушыларды автоқосылуын реттеңіз.",
				title: "Расталған домендер"
			},
			successMessage: "Ұйым жаңартылды.",
			title: "Профильді жаңарту"
		},
		removeDomainPage: {
			messageLine1: "{{domain}} домені жойылады.",
			messageLine2: "Пайдаланушылар автоқосыла алмайды.",
			successMessage: "{{domain}} жойылды.",
			title: "Доменді жою"
		},
		securityPage: {
			removeDialog: {
				confirmButton: void 0,
				subtitle: void 0,
				title: void 0
			},
			ssoSection: {
				badge__active: void 0,
				badge__inProgress: void 0,
				badge__inactive: void 0,
				badge__unconfigured: void 0,
				descriptionLine1: void 0,
				domainLabel: void 0,
				menuAction__activate: void 0,
				menuAction__deactivate: void 0,
				menuAction__edit: void 0,
				menuAction__remove: void 0,
				primaryButton__continueConfiguration: void 0,
				primaryButton__startConfiguration: void 0,
				title: void 0,
				tooltip: void 0,
				tooltipLabel: void 0,
				tooltip__noRole: void 0
			},
			title: void 0
		},
		start: {
			headerTitle__general: "Жалпы",
			headerTitle__members: "Мүшелер",
			membershipSeatUsageLabel: void 0,
			profileSection: {
				primaryButton: "Профильді жаңарту",
				title: "Ұйым профилі",
				uploadAction__title: "Логотип"
			}
		},
		verifiedDomainPage: {
			dangerTab: {
				calloutInfoLabel: "Доменді жою шақырылған пайдаланушыларға әсер етеді.",
				removeDomainActionLabel__remove: "Доменді жою",
				removeDomainSubtitle: "Доменді расталған тізімнен алып тастау",
				removeDomainTitle: "Доменді жою"
			},
			enrollmentTab: {
				automaticInvitationOption__description: "Пайдаланушылар тіркелген кезде автошақырылады.",
				automaticInvitationOption__label: "Автошақырулар",
				automaticSuggestionOption__description: "Пайдаланушыларға ұсыныс жіберіледі, бірақ әкімші растауы керек.",
				automaticSuggestionOption__label: "Автоұсыныстар",
				calloutInfoLabel: "Тіркеу режимі жаңа пайдаланушыларға ғана әсер етеді.",
				calloutInvitationCountLabel: "Күтудегі шақырулар: {{count}}",
				calloutSuggestionCountLabel: "Күтудегі ұсыныстар: {{count}}",
				manualInvitationOption__description: "Пайдаланушылар тек қолмен шақырылады.",
				manualInvitationOption__label: "Автотіркеу жоқ",
				subtitle: "Домен арқылы қосылу әдісін таңдаңыз."
			},
			start: {
				headerTitle__danger: "Қауіпті",
				headerTitle__enrollment: "Тіркеу опциялары"
			},
			subtitle: "{{domain}} домені расталды. Тіркеу режимін таңдаңыз.",
			title: "{{domain}} жаңарту"
		},
		verifyDomainPage: {
			formSubtitle: "Электрондық поштаңызға жіберілген растау кодын енгізіңіз",
			formTitle: "Растау коды",
			resendButton: "Код алмадыңыз ба? Қайта жіберу",
			subtitle: "{{domainName}} доменін электрондық пошта арқылы растау қажет.",
			subtitleVerificationCodeScreen: "{{emailAddress}} жіберілген растау кодын енгізіңіз.",
			title: "Доменді растау"
		}
	},
	organizationSwitcher: {
		action__closeOrganizationSwitcher: void 0,
		action__createOrganization: "Ұйым құру",
		action__invitationAccept: "Қосылу",
		action__manageOrganization: "Басқару",
		action__openOrganizationSwitcher: void 0,
		action__suggestionsAccept: "Қосылу сұрауы",
		notSelected: "Ұйым таңдалмады",
		personalWorkspace: "Жеке есептік жазба",
		suggestionsAcceptedLabel: "Растауды күтуде"
	},
	paginationButton__next: "Келесі",
	paginationButton__previous: "Алдыңғы",
	paginationRowText__displaying: "Көрсетілуде",
	paginationRowText__of: "/",
	reverification: {
		alternativeMethods: {
			actionLink: "Көмек алу",
			actionText: "Бұл опциялар жоқ па?",
			blockButton__backupCode: "Сақтық кодын қолдану",
			blockButton__emailCode: "{{identifier}} электрондық поштасына код жіберу",
			blockButton__passkey: "Passkey қолдану",
			blockButton__password: "Құпия сөзбен жалғастыру",
			blockButton__phoneCode: "{{identifier}} нөміріне SMS жіберу",
			blockButton__totp: "Аутентификатор қолдану",
			getHelp: {
				blockButton__emailSupport: "Қолдау қызметіне хабарласу",
				content: "Есептік жазбаны растауда қиындық туындаса, бізге хабарласыңыз.",
				title: "Көмек алу"
			},
			subtitle: "Растау әдісін өзгерту",
			title: "Басқа әдісті қолдану"
		},
		backupCodeMfa: {
			subtitle: "Екі қадамды растау кезінде алынған сақтық кодын енгізіңіз",
			title: "Сақтық кодын енгізіңіз"
		},
		emailCode: {
			formTitle: "Растау коды",
			resendButton: "Код алмадыңыз ба? Қайта жіберу",
			subtitle: "Жалғастыру үшін электрондық поштаңызға жіберілген кодты енгізіңіз",
			title: "Растау қажет"
		},
		noAvailableMethods: {
			message: "Растау мүмкін емес. Ешбір аутентификация әдісі қосылмаған.",
			subtitle: "Қате орын алды",
			title: "Есептік жазбаны растау мүмкін емес"
		},
		passkey: {
			blockButton__passkey: "Құпия кілт қолдану",
			subtitle: "Құпия кілт арқылы сіздің тұлғаңыз расталады. Құрылғыңыз саусақ ізі, бет бейнесі немесе экран құлыптамасын сұрауы мүмкін.",
			title: "Құпия кілт қолдану"
		},
		password: {
			actionLink: "Басқа әдісті қолдану",
			subtitle: "Жалғастыру үшін құпия сөзді енгізіңіз",
			title: "Растау қажет"
		},
		phoneCode: {
			formTitle: "Растау коды",
			resendButton: "Код алмадыңыз ба? Қайта жіберу",
			subtitle: "Жалғастыру үшін телефонға жіберілген кодты енгізіңіз",
			title: "Растау қажет"
		},
		phoneCodeMfa: {
			formTitle: "Растау коды",
			resendButton: "Код алмадыңыз ба? Қайта жіберу",
			subtitle: "Жалғастыру үшін телефонға жіберілген кодты енгізіңіз",
			title: "Растау қажет"
		},
		totpMfa: {
			formTitle: "Растау коды",
			subtitle: "Аутентификатор қолданбасындағы кодты енгізіңіз",
			title: "Растау қажет"
		}
	},
	searchInput: { action__clear: void 0 },
	signIn: {
		accountSwitcher: {
			action__addAccount: "Есептік жазба қосу",
			action__signOutAll: "Барлық есептік жазбалардан шығу",
			subtitle: "Жалғастыратын есептік жазбаны таңдаңыз.",
			title: "Есептік жазбаны таңдау"
		},
		alternativeMethods: {
			actionLink: "Көмек алу",
			actionText: "Басқа опциялар жоқ па?",
			blockButton__backupCode: "Сақтық кодын қолдану",
			blockButton__emailCode: "{{identifier}} электрондық поштасына код жіберу",
			blockButton__emailLink: "{{identifier}} электрондық поштасына сілтеме жіберу",
			blockButton__passkey: "Құпия кілт арқылы кіру",
			blockButton__password: "Құпия сөзбен кіру",
			blockButton__phoneCode: "{{identifier}} нөміріне SMS жіберу",
			blockButton__totp: "Аутентификатор қолдану",
			getHelp: {
				blockButton__emailSupport: "Қолдау қызметіне хабарласу",
				content: "Кіруде қиындық туындаса, бізге хабарласыңыз.",
				title: "Көмек алу"
			},
			subtitle: "Кіру әдісін өзгерту",
			title: "Басқа әдісті қолдану"
		},
		alternativePhoneCodeProvider: {
			formTitle: void 0,
			resendButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		backupCodeMfa: {
			subtitle: "Екі қадамды растау кезінде алынған сақтық кодын енгізіңіз",
			title: "Сақтық кодын енгізіңіз"
		},
		emailCode: {
			formTitle: "Растау коды",
			resendButton: "Код алмадыңыз ба? Қайта жіберу",
			subtitle: "{{applicationName}} қолдануды жалғастыру үшін",
			title: "Электрондық поштаңызды тексеріңіз"
		},
		emailCodeMfa: {
			formTitle: "Электрондық поштаңызды тексеріңіз",
			resendButton: "Код алмадыңыз ба? Қайта жіберу",
			subtitle: "{{applicationName}} қолдануды жалғастыру үшін",
			title: "Электрондық поштаңызды тексеріңіз"
		},
		emailLink: {
			clientMismatch: {
				subtitle: "Бастапқы құрылғыда растау сілтемесін ашыңыз.",
				title: "Сілтеме осы құрылғыға жарамсыз"
			},
			expired: {
				subtitle: "Жалғастыру үшін бастапқы бетке оралыңыз.",
				title: "Растау сілтемесінің мерзімі бітті"
			},
			failed: {
				subtitle: "Жалғастыру үшін бастапқы бетке оралыңыз.",
				title: "Растау сілтемесі жарамсыз"
			},
			formSubtitle: "Электрондық поштаңызға жіберілген сілтемені қолданыңыз",
			formTitle: "Растау сілтемесі",
			loading: {
				subtitle: "Сіз бағытталасыз...",
				title: "Кіру процесі..."
			},
			resendButton: "Сілтеме алмадыңыз ба? Қайта жіберу",
			subtitle: "{{applicationName}} қолдануды жалғастыру үшін",
			title: "Электрондық поштаңызды тексеріңіз",
			unusedTab: { title: "Бұл бетті жауып қойуға болады" },
			verified: {
				subtitle: "Сіз сәтті кірдіңіз",
				title: "Сәтті кірдіңіз"
			},
			verifiedSwitchTab: {
				subtitle: "Жалғастыру үшін бастапқы бетке оралыңыз",
				subtitleNewTab: "Жалғастыру үшін жаңа бетке оралыңыз",
				titleNewTab: "Басқа бетте кірдіңіз"
			}
		},
		emailLinkMfa: {
			formSubtitle: "Электрондық поштаңызға жіберілген растау сілтемесін пайдаланыңыз",
			resendButton: "Сілтеме алмадыңыз ба? Қайта жіберу",
			subtitle: "{{applicationName}} қосымшасына жалғастыру үшін",
			title: "Электрондық поштаңызды тексеріңіз"
		},
		enterpriseConnections: {
			subtitle: void 0,
			title: void 0
		},
		forgotPassword: {
			formTitle: "Құпия сөзді қалпына келтіру коды",
			resendButton: "Код алмадыңыз ба? Қайта жіберу",
			subtitle: "Құпия сөзді қалпына келтіру үшін",
			subtitle_email: "Алдымен электрондық поштаңызға жіберілген кодты енгізіңіз",
			subtitle_phone: "Алдымен телефонға жіберілген кодты енгізіңіз",
			title: "Құпия сөзді қалпына келтіру"
		},
		forgotPasswordAlternativeMethods: {
			blockButton__resetPassword: "Құпия сөзді қалпына келтіру",
			label__alternativeMethods: "Немесе басқа әдісті қолданыңыз",
			title: "Құпия сөзді ұмыттыңыз ба?"
		},
		newDeviceVerificationNotice: "Сіз жаңа құрылғыдан кірудесіз. Біз сіздің есептік жазбаңызды қауіпсіз сақтау үшін растау сұраймыз.",
		noAvailableMethods: {
			message: "Кіру мүмкін емес. Аутентификация әдісі қолжетімсіз.",
			subtitle: "Қате орын алды",
			title: "Кіру мүмкін емес"
		},
		passkey: {
			subtitle: "Құпия кілт арқылы тұлғаңыз расталады. Құрылғыңыз саусақ ізі, бет бейнесі немесе экран құлыптамасын сұрауы мүмкін.",
			title: "Құпия кілт қолдану"
		},
		password: {
			actionLink: "Басқа әдісті қолдану",
			subtitle: "Есептік жазбаңыздың құпия сөзін енгізіңіз",
			title: "Құпия сөзді енгізіңіз"
		},
		passwordCompromised: { title: void 0 },
		passwordPwned: { title: "Құпия сөз қауіпті" },
		passwordUntrusted: { title: void 0 },
		phoneCode: {
			formTitle: "Растау коды",
			resendButton: "Код алмадыңыз ба? Қайта жіберу",
			subtitle: "{{applicationName}} қолдануды жалғастыру үшін",
			title: "Телефонды тексеріңіз"
		},
		phoneCodeMfa: {
			formTitle: "Растау коды",
			resendButton: "Код алмадыңыз ба? Қайта жіберу",
			subtitle: "Жалғастыру үшін телефонға жіберілген кодты енгізіңіз",
			title: "Телефонды тексеріңіз"
		},
		protectCheck: {
			loading: void 0,
			retryButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		resetPassword: {
			formButtonPrimary: "Құпия сөзді өзгерту",
			requiredMessage: "Қауіпсіздік себептерімен құпия сөзді өзгерту қажет.",
			successMessage: "Құпия сөз сәтті өзгертілді. Кіру үшін күтіңіз.",
			title: "Жаңа құпия сөзді орнату"
		},
		resetPasswordMfa: { detailsLabel: "Құпия сөзді өзгерту үшін тұлғаңызды растау қажет." },
		start: {
			actionLink: "Тіркелу",
			actionLink__join_waitlist: "Күту тізіміне қосылу",
			actionLink__use_email: "Электрондық поштаны қолдану",
			actionLink__use_email_username: "Электрондық пошта немесе пайдаланушы аты",
			actionLink__use_passkey: "Құпия кілт қолдану",
			actionLink__use_phone: "Телефон нөмірін қолдану",
			actionLink__use_username: "Пайдаланушы атын қолдану",
			actionText: "Есептік жазбаңыз жоқ па?",
			actionText__join_waitlist: "Ерте қол жеткізгіңіз келе ме?",
			alternativePhoneCodeProvider: {
				actionLink: void 0,
				label: void 0,
				subtitle: void 0,
				title: void 0
			},
			subtitle: "Қош келдіңіз! Жалғастыру үшін кіріңіз",
			subtitleCombined: void 0,
			title: "{{applicationName}} қолдану үшін кіріңіз",
			titleCombined: "{{applicationName}} қолдануды жалғастыру"
		},
		totpMfa: {
			formTitle: "Растау коды",
			subtitle: "Аутентификатор қолданбасындағы кодты енгізіңіз",
			title: "Екі қадамды растау"
		},
		web3Solana: {
			subtitle: "Кіру үшін төменде әмиянды таңдаңыз",
			title: "Solana арқылы кіру"
		}
	},
	signInEnterPasswordTitle: "Құпия сөзді енгізіңіз",
	signUp: {
		alternativePhoneCodeProvider: {
			resendButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		continue: {
			actionLink: "Кіру",
			actionText: "Есептік жазбаңыз бар ма?",
			subtitle: "Қалған ақпаратты толтырыңыз.",
			title: "Жетіспейтін өрістерді толтыру"
		},
		emailCode: {
			formSubtitle: "Электрондық поштаңызға жіберілген растау кодын енгізіңіз",
			formTitle: "Растау коды",
			resendButton: "Код алмадыңыз ба? Қайта жіберу",
			subtitle: "Электрондық поштаңызды растаңыз",
			title: "Электрондық поштаны растау"
		},
		emailLink: {
			clientMismatch: {
				subtitle: "Растау сілтемесін бастапқы құрылғыда ашыңыз.",
				title: "Сілтеме осы құрылғыға жарамсыз"
			},
			formSubtitle: "Электрондық поштаңызға жіберілген сілтемені қолданыңыз",
			formTitle: "Растау сілтемесі",
			loading: { title: "Тіркелу..." },
			resendButton: "Сілтеме алмадыңыз ба? Қайта жіберу",
			subtitle: "{{applicationName}} қолдануды жалғастыру үшін",
			title: "Электрондық поштаңызды растау",
			verified: { title: "Сәтті тіркелдіңіз" },
			verifiedSwitchTab: {
				subtitle: "Жалғастыру үшін жаңа бетке оралыңыз",
				subtitleNewTab: "Жалғастыру үшін алдыңғы бетке оралыңыз",
				title: "Электрондық пошта расталды"
			}
		},
		enterpriseConnections: {
			subtitle: void 0,
			title: void 0
		},
		legalConsent: {
			checkbox: {
				label__onlyPrivacyPolicy: "{{ privacyPolicyLink || link(\"Құпиялылық саясаты\") }} қабылдаймын",
				label__onlyTermsOfService: "{{ termsOfServiceLink || link(\"Қызмет көрсету шарттары\") }} қабылдаймын",
				label__termsOfServiceAndPrivacyPolicy: "{{ termsOfServiceLink || link(\"Қызмет көрсету шарттары\") }} және {{ privacyPolicyLink || link(\"Құпиялылық саясаты\") }} қабылдаймын"
			},
			continue: {
				subtitle: "Жалғастыру үшін шарттарды қабылдаңыз",
				title: "Заңды келісім"
			}
		},
		phoneCode: {
			formSubtitle: "Телефон нөміріңізге жіберілген растау кодын енгізіңіз",
			formTitle: "Растау коды",
			resendButton: "Код алмадыңыз ба? Қайта жіберу",
			subtitle: "Телефон нөміріңізді растаңыз",
			title: "Телефон нөмірін растау"
		},
		protectCheck: {
			loading: void 0,
			retryButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		restrictedAccess: {
			actionLink: "Кіру",
			actionText: "Есептік жазбаңыз бар ма?",
			blockButton__emailSupport: "Қолдау қызметіне хабарласу",
			blockButton__joinWaitlist: "Күту тізіміне қосылу",
			subtitle: "Тіркеу уақытша өшірілген. Қолдау қызметіне хабарласыңыз.",
			subtitleWaitlist: "Тіркеу уақытша өшірілген. Күту тізіміне қосылыңыз.",
			title: "Қол жеткізу шектеулі"
		},
		start: {
			actionLink: "Кіру",
			actionLink__use_email: "Электрондық поштаны қолдану",
			actionLink__use_phone: "Телефон нөмірін қолдану",
			actionText: "Есептік жазбаңыз бар ма?",
			alternativePhoneCodeProvider: {
				actionLink: void 0,
				label: void 0,
				subtitle: void 0,
				title: void 0
			},
			subtitle: "Қош келдіңіз! Тіркелу үшін ақпаратты толтырыңыз.",
			subtitleCombined: "Қош келдіңіз! Тіркелу үшін ақпаратты толтырыңыз.",
			title: "Есептік жазбаны құру",
			titleCombined: "Есептік жазбаны құру"
		},
		web3Solana: {
			subtitle: "Тіркелу үшін төменде әмиянды таңдаңыз",
			title: "Solana арқылы тіркелу"
		}
	},
	socialButtonsBlockButton: "{{provider|titleize}} арқылы жалғастыру",
	socialButtonsBlockButtonManyInView: "{{provider|titleize}}",
	taskChooseOrganization: {
		alerts: { organizationAlreadyExists: "Анықталған компания атауы ({{organizationName}}) және {{organizationDomain}} үшін ұйым бұрыннан бар. Шақыру арқылы қосылыңыз." },
		chooseOrganization: {
			action__createOrganization: "Жаңа ұйым құру",
			action__invitationAccept: "Қосылу",
			action__suggestionsAccept: "Қосылуды сұрау",
			subtitle: "Мәні бар ұйымға қосылыңыз немесе жаңасын құрыңыз",
			subtitle__createOrganizationDisabled: "Бар ұйымға қосылыңыз",
			suggestionsAcceptedLabel: "Мақұлдау күтілуде",
			title: "Ұйым таңдаңыз"
		},
		createOrganization: {
			formButtonReset: "Болдырмау",
			formButtonSubmit: "Жалғастыру",
			formFieldInputPlaceholder__name: "Менің ұйымым",
			formFieldInputPlaceholder__slug: "menin-uiyym",
			formFieldLabel__name: "Атауы",
			formFieldLabel__slug: "Slug",
			subtitle: "Жалғастыру үшін ұйымыңыздың мәліметтерін енгізіңіз",
			title: "Ұйымыңызды баптаңыз"
		},
		organizationCreationDisabled: {
			subtitle: "Шақыру алу үшін ұйымыңыздың әкімшісіне хабарласыңыз.",
			title: "Сіз ұйымға тиесілі болуыңыз керек"
		},
		signOut: {
			actionLink: "Шығу",
			actionText: "{{identifier}} ретінде кірді"
		}
	},
	taskResetPassword: {
		formButtonPrimary: void 0,
		signOut: {
			actionLink: void 0,
			actionText: void 0
		},
		subtitle: void 0,
		title: void 0
	},
	taskSetupMfa: {
		badge: void 0,
		signOut: {
			actionLink: void 0,
			actionText: void 0
		},
		smsCode: {
			addPhone: {
				formButtonPrimary: void 0,
				infoText: void 0
			},
			addPhoneNumber: void 0,
			cancel: void 0,
			subtitle: void 0,
			success: {
				finishButton: void 0,
				message1: void 0,
				message2: void 0,
				title: void 0
			},
			title: void 0,
			verifyPhone: {
				formButtonPrimary: void 0,
				formTitle: void 0,
				resendButton: void 0,
				subtitle: void 0,
				title: void 0
			}
		},
		start: {
			methodSelection: {
				phoneCode: void 0,
				totp: void 0
			},
			subtitle: void 0,
			title: void 0
		},
		totpCode: {
			addAuthenticatorApp: {
				buttonAbleToScan__nonPrimary: void 0,
				buttonUnableToScan__nonPrimary: void 0,
				formButtonPrimary: void 0,
				formButtonReset: void 0,
				infoText__ableToScan: void 0,
				infoText__unableToScan: void 0,
				inputLabel__unableToScan1: void 0
			},
			success: {
				finishButton: void 0,
				message1: void 0,
				message2: void 0,
				title: void 0
			},
			title: void 0,
			verifyTotp: {
				formButtonPrimary: void 0,
				formButtonReset: void 0,
				formTitle: void 0,
				subtitle: void 0,
				title: void 0
			}
		}
	},
	unstable__errors: {
		action_blocked: void 0,
		already_a_member_in_organization: "{{email}} ұйымға қазірдің өзінде қосылған.",
		api_key_name_already_exists: void 0,
		api_key_usage_exceeded: void 0,
		avatar_file_size_exceeded: "Файл өлшемі 10 МБ шегінен асып кетті. Кішірек файлды таңдаңыз.",
		avatar_file_type_invalid: "Файл түрі қолдау көрсетілмейді. JPG, PNG, GIF немесе WEBP суретін жүктеңіз.",
		captcha_invalid: "Қауіпсіздік тексерілуі сәтсіз аяқталды. Браузерді өзгерту немесе кеңейтулерді өшіруге тырысыңыз.",
		captcha_unavailable: "Бот тексерілуі сәтсіз аяқталды. Бетті жаңартып немесе қолдау қызметіне хабарласыңыз.",
		form_code_incorrect: void 0,
		form_email_address_blocked: "Уақытша электрондық пошта қызметтері қолдау көрсетілмейді. Есептік жазбаны жасау үшін кәдімгі электрондық пошта мекенжайыңызды пайдаланыңыз.",
		form_identifier_exists__email_address: "Бұл электрондық пошта тіркелген. Басқасын қолданыңыз.",
		form_identifier_exists__phone_number: "Бұл телефон нөмірі тіркелген. Басқасын қолданыңыз.",
		form_identifier_exists__username: "Бұл пайдаланушы аты тіркелген. Басқасын қолданыңыз.",
		form_identifier_not_found: "Есептік жазба табылмады. Дұрыстығын тексеріңіз.",
		form_new_password_matches_current: void 0,
		form_param_format_invalid: "Енгізілген мән жарамсыз пішімде.",
		form_param_format_invalid__email_address: "Электрондық пошта жарамсыз.",
		form_param_format_invalid__phone_number: "Телефон нөмірі жарамсыз.",
		form_param_max_length_exceeded__first_name: "Аты 256 таңбадан аспауы керек.",
		form_param_max_length_exceeded__last_name: "Тегі 256 таңбадан аспауы керек.",
		form_param_max_length_exceeded__name: "Атауы 256 таңбадан аспауы керек.",
		form_param_nil: "Бұл өріс міндетті.",
		form_param_type_invalid: void 0,
		form_param_type_invalid__email_address: void 0,
		form_param_type_invalid__phone_number: void 0,
		form_param_value_invalid: "Енгізілген мән жарамсыз.",
		form_password_compromised__sign_in: void 0,
		form_password_incorrect: "Құпия сөз қате.",
		form_password_length_too_short: "Құпия сөз тым қысқа. Кемінде 8 таңба болуы керек.",
		form_password_not_strong_enough: "Құпия сөз әлсіз.",
		form_password_or_identifier_incorrect: "Құпия сөз немесе электрондық пошта мекенжайы дұрыс емес. Қайталап көріңіз немесе басқа әдісті пайдаланыңыз.",
		form_password_pwned: "Бұл құпия сөз қауіпсіздік бұзылуынан табылды. Басқа құпия сөзді қолданыңыз.",
		form_password_pwned__sign_in: "Бұл құпия сөз қауіпсіз емес. Құпия сөзді өзгертуге болады.",
		form_password_size_in_bytes_exceeded: "Құпия сөзде тым көп байт бар. Оны қысқартыңыз немесе арнайы таңбаларды алып тастаңыз.",
		form_password_untrusted__sign_in: void 0,
		form_password_validation_failed: "Құпия сөз қате",
		form_username_invalid_character: "Пайдаланушы атында жарамсыз таңбалар бар. Тек әріптер, сандар және астыңғы сызықшаларды қолданыңыз.",
		form_username_invalid_length: "Пайдаланушы аты {{min_length}} пен {{max_length}} арасында болуы керек.",
		form_username_needs_non_number_char: "Пайдаланушы атыңызда кемінде бір санды емес таңба болуы керек.",
		identification_deletion_failed: "Соңғы идентификацияны жою мүмкін емес.",
		insufficient_seats_change_plan: void 0,
		insufficient_seats_contact_support: void 0,
		not_allowed_access: "Бұл бетке қол жеткізуге рұқсат жоқ. Қате болса, қолдау қызметіне хабарласыңыз.",
		oauth_access_denied: void 0,
		organization_domain_blocked: "Бұл электрондық пошта домені бұғатталған. Басқасын қолданыңыз.",
		organization_domain_common: "Бұл жалпы электрондық пошта домені. Басқасын қолданыңыз.",
		organization_domain_exists_for_enterprise_connection: "Бұл домен ұйымыңыздың SSO үшін қолданылады.",
		organization_membership_quota_exceeded: "Ұйым мүшеліктерінің шектеуіне жеттіңіз.",
		organization_minimum_permissions_needed: "Ұйымда кемінде бір әкімші болуы керек.",
		organization_not_found_or_unauthorized: void 0,
		organization_not_found_or_unauthorized_with_create_organization_disabled: void 0,
		passkey_already_exists: "Бұл құрылғыда passkey тіркелген.",
		passkey_not_supported: "Бұл құрылғыда passkey қолдауы жоқ.",
		passkey_pa_not_supported: "Тіркеу үшін платформа аутентификаторы қажет.",
		passkey_registration_cancelled: "Passkey тіркеуі тоқтатылды немесе мерзімі бітті.",
		passkey_retrieval_cancelled: "Passkey тексеруі тоқтатылды немесе мерзімі бітті.",
		passwordComplexity: {
			maximumLength: "{{length}} таңбадан кем",
			minimumLength: "{{length}} таңбадан көп",
			requireLowercase: "кіші әріп",
			requireNumbers: "сан",
			requireSpecialCharacter: "арнайы таңба",
			requireUppercase: "бас әріп",
			sentencePrefix: "Құпия сөзде болуы керек:"
		},
		phone_number_exists: "Бұл телефон нөмірі тіркелген. Басқасын қолданыңыз.",
		protect_check_aborted: void 0,
		protect_check_already_resolved: void 0,
		protect_check_execution_failed: void 0,
		protect_check_invalid_script: void 0,
		protect_check_invalid_sdk_url: void 0,
		protect_check_script_load_failed: void 0,
		protect_check_timed_out: void 0,
		protect_check_unsupported_environment: void 0,
		session_exists: void 0,
		web3_missing_identifier: "Web3 Wallet кеңейтуі табылмады. Орнатыңыз.",
		web3_signature_request_rejected: "Сіз қолтаңба сұрауын қабылдамадыңыз. Жалғастыру үшін қайтадан көріңіз.",
		web3_solana_signature_generation_failed: "Қолтаңбаны жасау кезінде қате орын алды. Жалғастыру үшін қайтадан көріңіз.",
		zxcvbn: {
			couldBeStronger: "Құпия сөз әлсіз. Таңбалар санын көбейтіңіз.",
			goodPassword: "Құпия сөз талаптарға сай.",
			notEnough: "Құпия сөз әлсіз.",
			suggestions: {
				allUppercase: "Кейбір әріптерді бас әріппен жазыңыз.",
				anotherWord: "Сирек сөздерді қосыңыз.",
				associatedYears: "Жеке жылдардан аулақ болыңыз.",
				capitalization: "Бірінші әріптен басқаларын да бас әріппен жазыңыз.",
				dates: "Жеке күндер мен жылдардан аулақ болыңыз.",
				l33t: "\"@\" сияқты алмастырулардан аулақ болыңыз.",
				longerKeyboardPattern: "Ұзын әріптер тізбегін қолданыңыз.",
				noNeed: "Құпия сөзді арнайы таңбаларсыз жасауға болады.",
				pwned: "Басқа жерде қолдансаңыз, өзгертіңіз.",
				recentYears: "Соңғы жылдардан аулақ болыңыз.",
				repeated: "Қайталанған сөздер мен таңбалардан аулақ болыңыз.",
				reverseWords: "Кері сөздерден аулақ болыңыз.",
				sequences: "Жиі кездесетін тізбектерден аулақ болыңыз.",
				useWords: "Бірнеше сөзді қолданыңыз, бірақ жалпы сөз тіркестерінен аулақ болыңыз."
			},
			warnings: {
				common: "Бұл жиі қолданылатын құпия сөз.",
				commonNames: "Жалпы атаулар оңай болжауға болады.",
				dates: "Күндер оңай болжауға болады.",
				extendedRepeat: "Қайталанған әріптер тізбегі оңай болжауға болады.",
				keyPattern: "Қысқа перне тізбегі оңай болжауға болады.",
				namesByThemselves: "Жеке атаулар оңай болжауға болады.",
				pwned: "Құпия сөз Интернеттегі деректердің бұзылуынан табылды.",
				recentYears: "Соңғы жылдар оңай болжауға болады.",
				sequences: "Жиі кездесетін тізбектер оңай болжауға болады.",
				similarToCommon: "Бұл жиі қолданылатын құпия сөзге ұқсас.",
				simpleRepeat: "Қайталанған әріптер оңай болжауға болады.",
				straightRow: "Тікелей пернелер қатары оңай болжауға болады.",
				topHundred: "Бұл жиі қолданылатын құпия сөз.",
				topTen: "Бұл өте жиі қолданылатын құпия сөз.",
				userInputs: "Жеке немесе бетке қатысты деректер болмауы керек.",
				wordByItself: "Жалғыз сөздер оңай болжауға болады."
			}
		}
	},
	userButton: {
		action__addAccount: "Есептік жазба қосу",
		action__closeUserMenu: void 0,
		action__manageAccount: "Есептік жазбаны басқару",
		action__openUserMenu: void 0,
		action__signOut: "Шығу",
		action__signOutAll: "Барлық есептік жазбалардан шығу",
		label__accountActions: "Есептік жазба әрекеттері",
		label__activeSessions: "Белсенді сессиялар",
		label__userButtonPopover: "Есептік жазба тақтасы"
	},
	userProfile: {
		apiKeysPage: { title: void 0 },
		backupCodePage: {
			actionLabel__copied: "Көшірілді!",
			actionLabel__copy: "Барлығын көшіру",
			actionLabel__download: ".txt жүктеу",
			actionLabel__print: "Басып шығару",
			infoText1: "Бұл есептік жазба үшін сақтық кодтары қосылады.",
			infoText2: "Сақтық кодтарын қауіпсіз жерде сақтаңыз. Құрылғыға қол жеткізе алмасаңыз, оларды қолданыңыз.",
			subtitle__codelist: "Оларды қауіпсіз сақтаңыз.",
			successMessage: "Сақтық кодтары қосылды. Құрылғыға қол жеткізе алмасаңыз, олардың бірін қолданыңыз.",
			successSubtitle: "Құрылғыға қол жеткізе алмасаңыз, сақтық кодтарын қолданыңыз.",
			title: "Сақтық кодтарын қосу",
			title__codelist: "Сақтық кодтары"
		},
		billingPage: {
			accountCreditsSection: {
				title: void 0,
				viewHistory: void 0
			},
			creditHistoryPage: {
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				title: void 0
			},
			paymentHistorySection: {
				empty: void 0,
				notFound: void 0,
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				tableHeader__status: void 0
			},
			paymentMethodsSection: {
				actionLabel__default: "Әдепкі ету",
				actionLabel__remove: "Жою",
				add: "Жаңа төлем әдісін қосу",
				addSubtitle: "Есептік жазбаңызға жаңа төлем әдісін қосыңыз.",
				cancelButton: "Болдырмау",
				formButtonPrimary__add: "Төлем әдісін қосу",
				formButtonPrimary__pay: "{{amount}} төлеу",
				payWithTestCardButton: void 0,
				removeMethod: {
					messageLine1: "{{identifier}} есептік жазбадан жойылады.",
					messageLine2: "Бұл төлем әдісін қолдана алмайсыз және оған тәуелді жазылымдар жұмыс істемейді.",
					successMessage: "{{paymentMethod}} жойылды.",
					title: "Төлем әдісін жою"
				},
				title: "Қолжетімді опциялар"
			},
			start: {
				headerTitle__payments: void 0,
				headerTitle__plans: "Жоспарлар",
				headerTitle__statements: "Шот-фактуралар",
				headerTitle__subscriptions: "Жазылымдар"
			},
			statementsSection: {
				empty: void 0,
				itemCaption__paidForPlan: void 0,
				itemCaption__payerCredit: void 0,
				itemCaption__proratedCredit: void 0,
				itemCaption__subscribedAndPaidForPlan: void 0,
				notFound: void 0,
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				title: void 0,
				totalPaid: void 0
			},
			subscriptionsListSection: {
				actionLabel__manageSubscription: void 0,
				actionLabel__newSubscription: void 0,
				actionLabel__switchPlan: void 0,
				overview: void 0,
				tableHeader__edit: void 0,
				tableHeader__plan: void 0,
				tableHeader__startDate: void 0,
				title: void 0
			},
			subscriptionsSection: { actionLabel__default: "Басқару" },
			switchPlansSection: { title: void 0 },
			title: "Төлемдер мен есеп айырысу"
		},
		connectedAccountPage: {
			formHint: "Қосылатын провайдерді таңдаңыз.",
			formHint__noAccounts: "Қолжетімді провайдерлер жоқ.",
			removeResource: {
				messageLine1: "{{identifier}} есептік жазбадан жойылады.",
				messageLine2: "Бұл есептік жазбаны қолдана алмайсыз және тәуелді мүмкіндіктер жұмыс істемейді.",
				successMessage: "{{connectedAccount}} жойылды.",
				title: "Қосылған есептік жазбаны жою"
			},
			socialButtonsBlockButton: "{{provider|titleize}}",
			successMessage: "Провайдер есептік жазбаңызға қосылды",
			title: "Қосылған есептік жазбаны қосу"
		},
		deletePage: {
			actionDescription: "Жалғастыру үшін \"Есептік жазбаны жою\" деп енгізіңіз.",
			confirm: "Есептік жазбаны жою",
			messageLine1: "Тіркелгіңізді жойғыңыз келетініне сенімдісіз бе? Кейбір байланысты деректер сақталуы мүмкін. Деректерді толық жою үшін қолдау қызметіне хабарласыңыз.",
			messageLine2: "Бұл әрекет тұрақты және қайтымсыз.",
			title: "Есептік жазбаны жою"
		},
		emailAddressPage: {
			emailCode: {
				formHint: "Растау коды осы электрондық пошта мекенжайына жіберіледі.",
				formSubtitle: "{{identifier}} жіберілген растау кодын енгізіңіз",
				formTitle: "Растау коды",
				resendButton: "Код алмадыңыз ба? Қайта жіберу",
				successMessage: "{{identifier}} электрондық поштасы есептік жазбаңызға қосылды."
			},
			emailLink: {
				formHint: "Растау сілтемесі осы электрондық пошта мекенжайына жіберіледі.",
				formSubtitle: "{{identifier}} жіберілген электрондық хаттағы растау сілтемесін басыңыз",
				formTitle: "Растау сілтемесі",
				resendButton: "Сілтеме алмадыңыз ба? Қайта жіберу",
				successMessage: "{{identifier}} электрондық поштасы есептік жазбаңызға қосылды."
			},
			enterpriseSSOLink: {
				formButton: "Кіру үшін басыңыз",
				formSubtitle: "{{identifier}} арқылы кіруді аяқтаңыз"
			},
			formHint: "Электрондық поштаны есептік жазбаға қоспас бұрын растау қажет.",
			removeResource: {
				messageLine1: "{{identifier}} есептік жазбадан жойылады.",
				messageLine2: "Бұл электрондық пошта арқылы кіре алмайсыз.",
				successMessage: "{{emailAddress}} есептік жазбаңыздан жойылды.",
				title: "Электрондық поштаны жою"
			},
			title: "Электрондық поштаны қосу",
			verifyTitle: "Электрондық поштаны растау"
		},
		formButtonPrimary__add: "Қосу",
		formButtonPrimary__continue: "Жалғастыру",
		formButtonPrimary__finish: "Аяқтау",
		formButtonPrimary__remove: "Жою",
		formButtonPrimary__save: "Сақтау",
		formButtonReset: "Болдырмау",
		mfaPage: {
			formHint: "Қосылатын әдісті таңдаңыз.",
			title: "Екі қадамды растауды қосу"
		},
		mfaPhoneCodePage: {
			backButton: "Бар нөмірді қолдану",
			primaryButton__addPhoneNumber: "Телефон нөмірін қосу",
			removeResource: {
				messageLine1: "{{identifier}} нөміріне кіру кезінде растау кодтары жіберілмейді.",
				messageLine2: "Есептік жазбаның қауіпсіздігі төмендеуі мүмкін. Жалғастырасыз ба?",
				successMessage: "{{mfaPhoneCode}} үшін SMS коды арқылы екі қадамды растау өшірілді",
				title: "Екі қадамды растауды өшіру"
			},
			subtitle__availablePhoneNumbers: "SMS коды арқылы екі қадамды растау үшін бар нөмірді таңдаңыз немесе жаңасын қосыңыз.",
			subtitle__unavailablePhoneNumbers: "SMS коды арқылы екі қадамды растау үшін қолжетімді нөмірлер жоқ, жаңа нөмір қосыңыз.",
			successMessage1: "Кіру кезінде осы телефон нөміріне жіберілген растау кодын енгізу қажет болады.",
			successMessage2: "Сақтық кодтарын қауіпсіз жерге сақтаңыз. Аутентификация құрылғысына қол жеткізе алмасаңыз, сақтық кодтары арқылы кіре аласыз.",
			successTitle: "SMS коды арқылы растау қосылды",
			title: "SMS коды арқылы растауды қосу"
		},
		mfaTOTPPage: {
			authenticatorApp: {
				buttonAbleToScan__nonPrimary: "Орнына QR кодын сканерлеу",
				buttonUnableToScan__nonPrimary: "QR кодын сканерлей алмайсыз ба?",
				infoText__ableToScan: "Аутентификатор қолданбасында жаңа кіру әдісін орнатып, төмендегі QR кодын сканерлеңіз.",
				infoText__unableToScan: "Аутентификатор қолданбасында жаңа кіру әдісін орнатып, төмендегі Кілтті енгізіңіз.",
				inputLabel__unableToScan1: "Уақыт не Бір реттік құпия сөздер қосылғанын тексеріп, есептік жазбаны байланыстырыңыз.",
				inputLabel__unableToScan2: "Аутентификаторыңыз TOTP URI қолдаса, толық URI мекенжайын да көшіре аласыз."
			},
			removeResource: {
				messageLine1: "Осы аутентификатордан растау кодтары енді талап етілмейді.",
				messageLine2: "Есептік жазбаның қауіпсіздігі төмендеуі мүмкін. Жалғастырасыз ба?",
				successMessage: "Аутентификатор арқылы екі қадамды растау өшірілді",
				title: "Екі қадамды растауды өшіру"
			},
			successMessage: "Екі қадамды растау қосылды. Кіру кезінде аутентификатордан алынған растау кодын енгізу қажет болады.",
			title: "Аутентификатор қолданбасын қосу",
			verifySubtitle: "Аутентификаторыңыз жасаған растау кодын енгізіңіз",
			verifyTitle: "Растау коды"
		},
		mobileButton__menu: "Мәзір",
		navbar: {
			account: "Профиль",
			apiKeys: void 0,
			billing: "Төлемдер",
			description: "Есептік жазба ақпаратын басқару.",
			security: "Қауіпсіздік",
			title: "Есептік жазба"
		},
		passkeyScreen: {
			removeResource: {
				messageLine1: "{{name}} есептік жазбадан жойылады.",
				title: "Құпия кілтті жою"
			},
			subtitle__rename: "Құпия кілт атауын өзгерту арқылы іздеуді жеңілдетуге болады.",
			title__rename: "Құпия кілт атауын өзгерту"
		},
		passwordPage: {
			checkboxInfoText__signOutOfOtherSessions: "Ескі құпия сөзді қолданған барлық басқа құрылғылардан шығу ұсынылады.",
			readonly: "Кәсіпорын байланысы арқылы кіретін болғандықтан, құпия сөзді өзгерту мүмкін емес.",
			successMessage__set: "Құпия сөз орнатылды.",
			successMessage__signOutOfOtherSessions: "Барлық басқа құрылғылардан шықтыңыз.",
			successMessage__update: "Құпия сөз жаңартылды.",
			title__set: "Құпия сөзді орнату",
			title__update: "Құпия сөзді жаңарту"
		},
		phoneNumberPage: {
			infoText: "Растау коды осы телефон нөміріне SMS арқылы жіберіледі. SMS жіберу ақысы болуы мүмкін.",
			removeResource: {
				messageLine1: "{{identifier}} есептік жазбадан жойылады.",
				messageLine2: "Бұл телефон нөмірі арқылы кіре алмайсыз.",
				successMessage: "{{phoneNumber}} жойылды.",
				title: "Телефон нөмірін жою"
			},
			successMessage: "{{identifier}} есептік жазбаға қосылды.",
			title: "Телефон нөмірін қосу",
			verifySubtitle: "{{identifier}} нөміріне жіберілген кодты енгізіңіз",
			verifyTitle: "Телефон нөмірін растау"
		},
		plansPage: { title: void 0 },
		profilePage: {
			fileDropAreaHint: "Ұсынылатын өлшемі 1:1, максимум 10MB.",
			imageFormDestructiveActionSubtitle: "Жою",
			imageFormSubtitle: "Жүктеу",
			imageFormTitle: "Профиль суреті",
			readonly: "Профиль ақпараты кәсіпорын байланысы арқылы берілген және өзгертілмейді.",
			successMessage: "Профиль жаңартылды.",
			title: "Профильді жаңарту"
		},
		start: {
			activeDevicesSection: {
				destructiveAction: "Құрылғыдан шығу",
				title: "Белсенді құрылғылар"
			},
			connectedAccountsSection: {
				actionLabel__connectionFailed: "Қайта қосу",
				actionLabel__reauthorize: "Қазір рұқсат беру",
				destructiveActionTitle: "Жою",
				primaryButton: "Есептік жазбаны қосу",
				subtitle__disconnected: "Бұл есептік жазба ажыратылды.",
				subtitle__reauthorize: "Рұқсаттар өзгерді. Функционалдық шектеулерден аулақ болу үшін қайта рұқсат беріңіз",
				title: "Қосылған есептік жазбалар"
			},
			dangerSection: {
				deleteAccountButton: "Есептік жазбаны жою",
				title: "Есептік жазбаны жою"
			},
			emailAddressesSection: {
				destructiveAction: "Электрондық поштаны жою",
				detailsAction__nonPrimary: "Негізгі ретінде орнату",
				detailsAction__primary: "Растауды аяқтау",
				detailsAction__unverified: "Растау",
				primaryButton: "Электрондық поштаны қосу",
				title: "Электрондық пошталар"
			},
			enterpriseAccountsSection: {
				primaryButton: "Есептік жазбаны қосу",
				title: "Кәсіпорын есептік жазбалары"
			},
			headerTitle__account: "Профиль мәліметтері",
			headerTitle__security: "Қауіпсіздік",
			mfaSection: {
				backupCodes: {
					actionLabel__regenerate: "Қайта жасау",
					headerTitle: "Сақтық кодтары",
					subtitle__regenerate: "Жаңа сақтық кодтарын алыңыз. Алдыңғы кодтар жойылады және қолданылмайды.",
					title__regenerate: "Сақтық кодтарын қайта жасау"
				},
				phoneCode: {
					actionLabel__setDefault: "Әдепкі ретінде орнату",
					destructiveActionLabel: "Жою"
				},
				primaryButton: "Екі қадамды растауды қосу",
				title: "Екі қадамды растау",
				totp: {
					destructiveActionTitle: "Жою",
					headerTitle: "Аутентификатор қолданбасы"
				}
			},
			passkeysSection: {
				menuAction__destructive: "Жою",
				menuAction__rename: "Атауын өзгерту",
				primaryButton: "Құпия кілтті қосу",
				title: "Құпия кілттер"
			},
			passwordSection: {
				primaryButton__setPassword: "Құпия сөзді орнату",
				primaryButton__updatePassword: "Құпия сөзді жаңарту",
				title: "Құпия сөз"
			},
			phoneNumbersSection: {
				destructiveAction: "Телефон нөмірін жою",
				detailsAction__nonPrimary: "Негізгі ретінде орнату",
				detailsAction__primary: "Растауды аяқтау",
				detailsAction__unverified: "Телефон нөмірін растау",
				primaryButton: "Телефон нөмірін қосу",
				title: "Телефон нөмірлері"
			},
			profileSection: {
				primaryButton: "Профильді жаңарту",
				title: "Профиль"
			},
			usernameSection: {
				primaryButton__setUsername: "Пайдаланушы атын орнату",
				primaryButton__updateUsername: "Пайдаланушы атын жаңарту",
				title: "Пайдаланушы аты"
			},
			web3WalletsSection: {
				destructiveAction: "Әптәпқалманы жою",
				detailsAction__nonPrimary: "Негізгі ретінде орнату",
				primaryButton: "Әптәпқалманы қосу",
				title: "Web3 әптәпқалмалары",
				web3SelectSolanaWalletScreen: {
					subtitle: "Есептік жазбаңызға қосу үшін Solana әмиянын таңдаңыз.",
					title: "Solana әмиянын қосу"
				}
			}
		},
		usernamePage: {
			successMessage: "Пайдаланушы аты жаңартылды.",
			title__set: "Пайдаланушы атын орнату",
			title__update: "Пайдаланушы атын жаңарту"
		},
		web3WalletPage: {
			removeResource: {
				messageLine1: "{{identifier}} есептік жазбадан жойылады.",
				messageLine2: "Бұл әптәпқалма арқылы кіре алмайсыз.",
				successMessage: "{{web3Wallet}} жойылды.",
				title: "Web3 әптәпқалманы жою"
			},
			subtitle__availableWallets: "Есептік жазбаға қосылатын Web3 әптәпқалманы таңдаңыз.",
			subtitle__unavailableWallets: "Қолжетімді Web3 әптәпқалмалар жоқ.",
			successMessage: "Әптәпқалма есептік жазбаға қосылды.",
			title: "Web3 әптәпқалманы қосу",
			web3WalletButtonsBlockButton: "{{provider|titleize}}"
		}
	},
	waitlist: {
		start: {
			actionLink: "Кіру",
			actionText: "Есептік жазбаңыз бар ма?",
			formButton: "Күту тізіміне қосылу",
			subtitle: "Электрондық поштаңызды енгізіңіз, біз сізбен хабарласамыз",
			title: "Күту тізіміне қосылу"
		},
		success: {
			message: "Жақында бетке қайта бағытталасыз...",
			subtitle: "Даяр болған кезде хабарласамыз",
			title: "Күту тізіміне қосылғаныңыз үшін рақмет!"
		}
	},
	web3SolanaWalletButtons: {
		connect: "{{walletName}} арқылы қосылу",
		continue: "{{walletName}} арқылы жалғастыру",
		noneAvailable: "Solana Web3 әмияндары табылмады. Web3 қолдайтын {{ solanaWalletsLink || link(\"wallet extension\") }} орнатыңыз."
	}
};

//#endregion
exports.kkKZ = kkKZ;
