// @clerk/localizations/dist/pl-PL.js
Object.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });

//#region src/pl-PL.ts
const plPL = {
	locale: "pl-PL",
	apiKeys: {
		action__add: void 0,
		action__search: void 0,
		copySecret: {
			formButtonPrimary__copyAndClose: "Kopiuj i zamknij",
			formHint: "Ze względów bezpieczeństwa nie będziemy mogli wyświetlić tego ponownie później.",
			formTitle: "Skopiuj teraz swój klucz API \"{{name}}\""
		},
		createdAndExpirationStatus__expiresOn: void 0,
		createdAndExpirationStatus__never: void 0,
		detailsTitle__emptyRow: void 0,
		formButtonPrimary__add: void 0,
		formFieldCaption__expiration__expiresOn: void 0,
		formFieldCaption__expiration__never: void 0,
		formFieldOption__expiration__180d: void 0,
		formFieldOption__expiration__1d: void 0,
		formFieldOption__expiration__1y: void 0,
		formFieldOption__expiration__30d: void 0,
		formFieldOption__expiration__60d: void 0,
		formFieldOption__expiration__7d: void 0,
		formFieldOption__expiration__90d: void 0,
		formFieldOption__expiration__never: void 0,
		formHint: void 0,
		formTitle: void 0,
		lastUsed__days: void 0,
		lastUsed__hours: void 0,
		lastUsed__minutes: void 0,
		lastUsed__months: void 0,
		lastUsed__seconds: void 0,
		lastUsed__years: void 0,
		menuAction__revoke: void 0,
		revokeConfirmation: {
			confirmationText: void 0,
			formButtonPrimary__revoke: void 0,
			formHint: void 0,
			formTitle: void 0,
			inputLabel: void 0
		},
		tableHeader__actions: void 0,
		tableHeader__lastUsed: void 0,
		tableHeader__name: void 0
	},
	backButton: "Powrót",
	badge__activePlan: void 0,
	badge__banned: void 0,
	badge__canceledEndsAt: void 0,
	badge__currentPlan: void 0,
	badge__default: "Domyślny",
	badge__endsAt: void 0,
	badge__expired: void 0,
	badge__freeTrial: void 0,
	badge__otherImpersonatorDevice: "Inne urządzenie osobiste",
	badge__pastDueAt: void 0,
	badge__pastDuePlan: void 0,
	badge__primary: "Podstawowy",
	badge__renewsAt: void 0,
	badge__requiresAction: "Wymaga działania",
	badge__startsAt: void 0,
	badge__thisDevice: "To urządzenie",
	badge__trialEndsAt: void 0,
	badge__unverified: "Niezweryfikowany",
	badge__upcomingPlan: void 0,
	badge__userDevice: "Urządzenie użytkownika",
	badge__you: "Ty",
	billing: {
		accountCredit: void 0,
		addPaymentMethod__label: void 0,
		alwaysFree: void 0,
		annually: void 0,
		availableFeatures: void 0,
		billedAnnually: void 0,
		billedAnnuallyOnly: void 0,
		billedMonthly: void 0,
		billedMonthlyOnly: void 0,
		cancelFreeTrial: void 0,
		cancelFreeTrialAccessUntil: void 0,
		cancelFreeTrialTitle: void 0,
		cancelSubscription: void 0,
		cancelSubscriptionAccessUntil: void 0,
		cancelSubscriptionNoCharge: void 0,
		cancelSubscriptionPastDue: void 0,
		cancelSubscriptionTitle: void 0,
		cannotSubscribeMonthly: void 0,
		cannotSubscribeUnrecoverable: void 0,
		checkout: {
			description__paymentSuccessful: void 0,
			description__subscriptionSuccessful: void 0,
			downgradeNotice: void 0,
			emailForm: {
				subtitle: void 0,
				title: void 0
			},
			lineItems: {
				title__freeTrialEndsAt: void 0,
				title__paymentMethod: void 0,
				title__statementId: void 0,
				title__subscriptionBegins: void 0,
				title__totalPaid: void 0
			},
			pastDueNotice: void 0,
			perMonth: void 0,
			title: void 0,
			title__paymentSuccessful: void 0,
			title__subscriptionSuccessful: void 0,
			title__trialSuccess: void 0,
			totalDueAfterTrial: void 0,
			totalDuePerPeriod: void 0
		},
		credit: void 0,
		creditRemainder: void 0,
		defaultFreePlanActive: void 0,
		free: void 0,
		getStarted: void 0,
		highlightedPlanBadge: void 0,
		keepFreeTrial: void 0,
		keepSubscription: void 0,
		manage: void 0,
		manageSubscription: void 0,
		month: void 0,
		monthAbbreviation: void 0,
		monthPerUnit: void 0,
		monthly: void 0,
		pastDue: void 0,
		pay: void 0,
		payerCreditRemainder: void 0,
		paymentMethod: {
			applePayDescription: {
				annual: void 0,
				monthly: void 0
			},
			dev: {
				anyNumbers: void 0,
				cardNumber: void 0,
				cvcZip: void 0,
				developmentMode: void 0,
				expirationDate: void 0,
				testCardInfo: void 0
			}
		},
		paymentMethods__label: void 0,
		pricingTable: {
			billingCycle: void 0,
			included: void 0,
			seatCost: {
				additionalSeats: void 0,
				freeUpToSeats: void 0,
				includedSeats: void 0,
				perSeat: void 0,
				tooltip: {
					additionalSeatsEach: void 0,
					firstSeatsIncludedInPlan: void 0,
					freeForUpToSeats: void 0
				},
				unlimitedSeats: void 0,
				upToSeats: void 0
			}
		},
		proratedDiscount: void 0,
		prorationCredit: void 0,
		reSubscribe: void 0,
		seatBreakdownIncludedPlural: void 0,
		seatBreakdownIncludedSingular: void 0,
		seatBreakdownPlural: void 0,
		seatBreakdownSingular: void 0,
		seats: void 0,
		seatsWithLimit: void 0,
		seeAllFeatures: void 0,
		startFreeTrial: void 0,
		startFreeTrial__days: void 0,
		subscribe: void 0,
		subscriptionDetails: {
			beginsOn: void 0,
			currentBillingCycle: void 0,
			endsOn: void 0,
			firstPaymentAmount: void 0,
			firstPaymentOn: void 0,
			nextPaymentAmount: void 0,
			nextPaymentOn: void 0,
			pastDueAt: void 0,
			renewsAt: void 0,
			subscribedOn: void 0,
			title: void 0,
			trialEndsOn: void 0,
			trialStartedOn: void 0
		},
		subtotal: void 0,
		subtotalRenewal: void 0,
		switchPlan: void 0,
		switchToAnnual: void 0,
		switchToAnnualWithAnnualPrice: void 0,
		switchToMonthly: void 0,
		switchToMonthlyWithPrice: void 0,
		totalDue: void 0,
		totalDuePerPeriod: void 0,
		totalDueToday: void 0,
		viewFeatures: void 0,
		viewPayment: void 0,
		year: void 0,
		yearAbbreviation: void 0,
		yearPerUnit: void 0
	},
	configureSSO: {
		activate: {
			activateButton: void 0,
			activeSubtitle: void 0,
			activeTitle: void 0,
			doneButton: void 0,
			skipButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		changeProviderDialog: {
			cancelButton: void 0,
			confirmButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		configureStep: {
			activeConnectionWarning: {
				dismiss: void 0,
				title: void 0
			},
			attributeMappingTable: { badges: {
				optional: void 0,
				required: void 0
			} },
			samlCustom: {
				assignUsersStep: {
					headerSubtitle: void 0,
					paragraph: void 0
				},
				attributeMappingStep: {
					attributeMappingTable: {
						columns: {
							attributeName: void 0,
							userAttribute: void 0
						},
						rows: {
							email: {
								attributeName: void 0,
								userAttribute: void 0
							},
							firstName: {
								attributeName: void 0,
								userAttribute: void 0
							},
							lastName: {
								attributeName: void 0,
								userAttribute: void 0
							}
						}
					},
					headerSubtitle: void 0,
					paragraph: void 0
				},
				createAppStep: {
					createAppInstructions: { paragraph: void 0 },
					headerSubtitle: void 0,
					serviceProviderFields: {
						acsUrl: { label: void 0 },
						spEntityId: { label: void 0 }
					}
				},
				identityProviderMetadataStep: {
					headerSubtitle: void 0,
					manual: {
						description: void 0,
						issuer: {
							label: void 0,
							placeholder: void 0
						},
						signOnUrl: {
							label: void 0,
							placeholder: void 0
						},
						signingCertificate: {
							fileUploaded: void 0,
							label: void 0,
							removeFile: void 0,
							replaceFile: void 0,
							uploadFile: void 0
						}
					},
					metadataUrl: {
						description: void 0,
						label: void 0,
						placeholder: void 0
					},
					modes: {
						ariaLabel: void 0,
						manual: void 0,
						metadataUrl: void 0
					}
				},
				mainHeaderTitle: void 0
			},
			samlGoogle: {
				attributeMappingStep: {
					attributeMappingTable: {
						columns: {
							appAttribute: void 0,
							googleAttribute: void 0
						},
						rows: {
							email: {
								appAttribute: void 0,
								googleAttribute: void 0
							},
							firstName: {
								appAttribute: void 0,
								googleAttribute: void 0
							},
							lastName: {
								appAttribute: void 0,
								googleAttribute: void 0
							}
						}
					},
					headerSubtitle: void 0,
					paragraph: void 0,
					step1: void 0,
					step2: void 0
				},
				configureUserAccess: {
					assignUsersInstructions: {
						paragraph1: void 0,
						paragraph2: void 0,
						step1: void 0,
						step2: void 0,
						step3: void 0
					},
					headerSubtitle: void 0
				},
				createAppStep: {
					createAppInstructions: {
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: void 0,
						title: void 0
					},
					headerSubtitle: void 0
				},
				identityProviderMetadataStep: {
					headerSubtitle: void 0,
					manual: {
						description: void 0,
						issuer: {
							label: void 0,
							placeholder: void 0
						},
						signOnUrl: {
							label: void 0,
							placeholder: void 0
						},
						signingCertificate: {
							fileUploaded: void 0,
							label: void 0,
							removeFile: void 0,
							replaceFile: void 0,
							uploadFile: void 0
						}
					},
					metadataFile: {
						description: void 0,
						fileUploaded: void 0,
						label: void 0,
						removeFile: void 0,
						replaceFile: void 0,
						uploadFile: void 0
					},
					modes: {
						ariaLabel: void 0,
						manual: void 0,
						metadataFile: void 0
					}
				},
				mainHeaderTitle: void 0,
				serviceProviderStep: {
					headerSubtitle: void 0,
					nameIdInstructions: {
						step1: void 0,
						step2: void 0
					},
					paragraph: void 0,
					serviceProviderFields: {
						acsUrl: { label: void 0 },
						spEntityId: { label: void 0 }
					},
					title: void 0
				}
			},
			samlMicrosoft: {
				attributeMappingStep: {
					attributeMappingTable: {
						columns: {
							attribute: void 0,
							claimName: void 0,
							value: void 0
						},
						copyClaimName: void 0,
						copyClaimNameCopied: void 0,
						rows: {
							email: {
								attribute: void 0,
								claimName: void 0,
								value: void 0
							},
							firstName: {
								attribute: void 0,
								claimName: void 0,
								value: void 0
							},
							lastName: {
								attribute: void 0,
								claimName: void 0,
								value: void 0
							}
						}
					},
					headerSubtitle: void 0,
					step1: void 0,
					step2: void 0,
					title: void 0
				},
				createAppStep: {
					assignUsersInstructions: {
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: void 0,
						step5: void 0,
						title: void 0
					},
					createAppInstructions: {
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: {
							label: void 0,
							subSteps: {
								appName: void 0,
								create: void 0,
								nonGallery: void 0
							}
						},
						title: void 0
					},
					headerSubtitle: void 0
				},
				identityProviderMetadataStep: {
					headerSubtitle: void 0,
					manual: {
						description: void 0,
						issuer: {
							label: void 0,
							placeholder: void 0
						},
						signOnUrl: {
							label: void 0,
							placeholder: void 0
						},
						signingCertificate: {
							fileUploaded: void 0,
							label: void 0,
							removeFile: void 0,
							replaceFile: void 0,
							uploadFile: void 0
						}
					},
					metadataUrl: {
						description: void 0,
						label: void 0,
						placeholder: void 0
					},
					modes: {
						ariaLabel: void 0,
						manual: void 0,
						metadataUrl: void 0
					}
				},
				mainHeaderTitle: void 0,
				serviceProviderStep: {
					headerSubtitle: void 0,
					serviceProviderFields: {
						acsUrl: { label: void 0 },
						spEntityId: { label: void 0 }
					},
					step1: void 0,
					step2: void 0,
					step3: void 0,
					step4: void 0,
					step5: void 0,
					step6: void 0,
					title: void 0
				}
			},
			samlOkta: {
				assignUsersStep: {
					assignUsersInstructions: {
						paragraph: void 0,
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: void 0,
						step5: void 0
					},
					headerSubtitle: void 0
				},
				attributeMappingStep: {
					attributeMappingTable: {
						columns: {
							expression: void 0,
							name: void 0
						},
						rows: {
							email: {
								expression: void 0,
								name: void 0
							},
							firstName: {
								expression: void 0,
								name: void 0
							},
							lastName: {
								expression: void 0,
								name: void 0
							}
						}
					},
					headerSubtitle: void 0,
					paragraph: void 0,
					step1: void 0,
					step2: void 0
				},
				createAppStep: {
					completeSamlIntegrationInstructions: {
						step1: void 0,
						step2: void 0,
						title: void 0
					},
					createAppInstructions: {
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: void 0,
						title: void 0
					},
					headerSubtitle: void 0,
					serviceProviderInstructions: {
						paragraph1: void 0,
						paragraph2: void 0,
						serviceProviderFields: {
							acsUrl: { label: void 0 },
							spEntityId: { label: void 0 }
						},
						title: void 0
					}
				},
				identityProviderMetadataStep: {
					headerSubtitle: void 0,
					manual: {
						description: void 0,
						issuer: {
							label: void 0,
							placeholder: void 0
						},
						signOnUrl: {
							label: void 0,
							placeholder: void 0
						},
						signingCertificate: {
							fileUploaded: void 0,
							label: void 0,
							removeFile: void 0,
							replaceFile: void 0,
							uploadFile: void 0
						}
					},
					metadataUrl: {
						description: void 0,
						label: void 0,
						placeholder: void 0
					},
					modes: {
						ariaLabel: void 0,
						manual: void 0,
						metadataUrl: void 0
					}
				},
				mainHeaderTitle: void 0
			}
		},
		missingManageEnterpriseConnectionsPermission: {
			subtitle: "Skontaktuj się z administratorem swojej organizacji, aby rozszerzyć swoje uprawnienia.",
			title: "Nie masz uprawnień do zarządzania logowaniem jednokrotnym (SSO)"
		},
		navbar: { title: "Skonfiguruj logowanie jednokrotne (SSO)" },
		organizationDomainsStep: {
			domainCard: {
				badge__expired: void 0,
				badge__unverified: "Niezweryfikowano",
				badge__verified: "Zweryfikowano",
				expiredAtLabel: void 0,
				expiredLabel: void 0,
				removeButtonTooltip__lastVerifiedDomain: void 0,
				removeButtonTooltip__lastVerifiedDomainActive: void 0,
				txtRecord: {
					hostLabel: "Host / Nazwa",
					instructions: "Dodaj ten rekord TXT u swojego dostawcy DNS. Zweryfikujemy go automatycznie, gdy rekord stanie się aktywny.",
					typeLabel: "Typ",
					valueLabel: "Wartość"
				},
				verifiedAtLabel: "Zweryfikowano {{ date | shortDate('pl-PL') }}",
				verifyAgainButton: void 0
			},
			domainSuggestion: {
				formButtonPrimary__add: "Dodaj {{domain}}",
				messageLabel: "Twój e-mail używa {{domain}}. Czy chcesz go dodać?"
			},
			formButtonPrimary__add: "Dodaj",
			formFieldInputPlaceholder__domain: "Dodaj domenę",
			formFieldLabel__domain: "Domena",
			removeDomainDialog: {
				cancelButton: void 0,
				removeButton: void 0,
				subtitle__active: void 0,
				subtitle__inactive: void 0,
				title: void 0
			},
			subtitle: "Dodaj i zweryfikuj własność domen, których Twoja organizacja używa do logowania.",
			title: "Dodaj domeny SSO"
		},
		resetConnectionDialog: {
			cancelButton: void 0,
			confirmationFieldLabel: void 0,
			confirmationFieldPlaceholder: void 0,
			resetButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		selectProviderStep: {
			saml: {
				customSaml: "Niestandardowy dostawca SAML",
				google: void 0,
				groupLabel: "SAML",
				microsoft: void 0,
				okta: "Okta Workforce"
			},
			subtitle: "Wybierz dostawcę, dla którego skonfigurujesz SSO.",
			title: "Wybierz dostawcę",
			warning: "Po wybraniu dostawcy nie można go ponownie zmienić aż do zakończenia konfiguracji"
		},
		testConfigurationStep: {
			error__noSuccessfulTestRun: void 0,
			subtitle: void 0,
			testResults: {
				actionLabel__refresh: void 0,
				empty: {
					subtitle: void 0,
					title: void 0
				},
				polling: void 0,
				status__failed: void 0,
				status__pending: void 0,
				status__success: void 0,
				title: void 0
			},
			testRunDetails: {
				howToFix: {
					actionLabel__viewDocumentation: void 0,
					oauth_access_denied: { description: void 0 },
					oauth_fetch_user_error: {
						intro: void 0,
						step1: void 0,
						step2: void 0
					},
					oauth_token_exchange_error: { description: void 0 },
					saml_email_address_domain_mismatch: { description: void 0 },
					saml_response_relaystate_missing: { description: void 0 },
					saml_user_attribute_missing: {
						intro: void 0,
						step1: void 0,
						step2: void 0,
						step3: void 0
					},
					sectionTitle: void 0
				},
				parsedUserInfo: {
					email: void 0,
					firstName: void 0,
					sectionTitle: void 0
				},
				runDetails: {
					actionLabel__copied: void 0,
					actionLabel__copy: void 0,
					errorCode: void 0,
					fullMessage: void 0,
					sectionTitle: void 0,
					status: void 0,
					timestamp: void 0
				},
				title: void 0
			},
			testUrl: { actionLabel__open: void 0 },
			title: void 0
		}
	},
	createOrganization: {
		formButtonSubmit: "Utwórz organizację",
		invitePage: { formButtonReset: "Pomiń" },
		title: "Utwórz organizację"
	},
	dates: {
		lastDay: "Wczoraj o godzinie {{ date | timeString('pl-PL') }}",
		next6Days: "{{ date | weekday('pl-PL','long') }} o godzinie {{ date | timeString('pl-PL') }}",
		nextDay: "Jutro o godzinie {{ date | timeString('pl-PL') }}",
		numeric: "{{ date | numeric('pl-PL') }}",
		previous6Days: "Ostatni(a) {{ date | weekday('pl-PL','long') }} o godzinie {{ date | timeString('pl-PL') }}",
		sameDay: "Dzisiaj o godzinie {{ date | timeString('pl-PL') }}"
	},
	dividerText: "lub",
	footerActionLink__alternativePhoneCodeProvider: void 0,
	footerActionLink__useAnotherMethod: "Użyj innej metody",
	footerPageLink__help: "Pomoc",
	footerPageLink__privacy: "Prywatność",
	footerPageLink__terms: "Warunki",
	formButtonPrimary: "Kontynuuj",
	formButtonPrimary__verify: "Zweryfikuj",
	formFieldAction__forgotPassword: "Zapomniałem/am hasła",
	formFieldError__matchingPasswords: "Hasła się zgadzają.",
	formFieldError__notMatchingPasswords: "Hasła się nie zgadzają.",
	formFieldError__verificationLinkExpired: "Link weryfikacyjny wygasł. Spróbuj ponownie.",
	formFieldHintText__optional: "Opcjonalne",
	formFieldHintText__slug: "Jest to unikalne ID, które jest czytelne dla człowieka, często używane w adresach URL.",
	formFieldInputPlaceholder__apiKeyDescription: void 0,
	formFieldInputPlaceholder__apiKeyExpirationDate: void 0,
	formFieldInputPlaceholder__apiKeyName: void 0,
	formFieldInputPlaceholder__backupCode: "Wprowadź kod zapasowy",
	formFieldInputPlaceholder__confirmDeletionUserAccount: "Usuń konto",
	formFieldInputPlaceholder__emailAddress: "Wprowadź adres email",
	formFieldInputPlaceholder__emailAddress_username: "Adres e-mail lub nazwa użytkownika",
	formFieldInputPlaceholder__emailAddresses: "Wprowadź lub wklej jeden lub więcej adresów e-mail, oddzielonych spacjami lub przecinkami",
	formFieldInputPlaceholder__firstName: "Imię",
	formFieldInputPlaceholder__lastName: "Nazwisko",
	formFieldInputPlaceholder__organizationDomain: "example.com",
	formFieldInputPlaceholder__organizationDomainEmailAddress: "jan.kowalski@example.com",
	formFieldInputPlaceholder__organizationName: "Nazwa organizacji",
	formFieldInputPlaceholder__organizationSlug: "moja-organizacja",
	formFieldInputPlaceholder__password: "Wprowadź swoje hasło",
	formFieldInputPlaceholder__phoneNumber: "Wprowadź numer telefonu",
	formFieldInputPlaceholder__signUpPassword: void 0,
	formFieldInputPlaceholder__username: void 0,
	formFieldInput__emailAddress_format: void 0,
	formFieldLabel__apiKey: "Klucz API",
	formFieldLabel__apiKeyDescription: "Opis",
	formFieldLabel__apiKeyExpiration: "Wygaśnięcie",
	formFieldLabel__apiKeyName: "Nazwa klucza tajnego",
	formFieldLabel__automaticInvitations: "Włącz automatyczne zaproszenia dla tej domeny",
	formFieldLabel__backupCode: "Kod zapasowy",
	formFieldLabel__confirmDeletion: "Potwierdzenie",
	formFieldLabel__confirmPassword: "Potwierdź hasło",
	formFieldLabel__currentPassword: "Obecne hasło",
	formFieldLabel__emailAddress: "Adres e-mail",
	formFieldLabel__emailAddress_username: "Adres e-mail lub nazwa użytkownika",
	formFieldLabel__emailAddresses: "Adresy e-mail",
	formFieldLabel__firstName: "Imię",
	formFieldLabel__lastName: "Nazwisko",
	formFieldLabel__newPassword: "Nowe hasło",
	formFieldLabel__organizationDomain: "Domena",
	formFieldLabel__organizationDomainDeletePending: "Usuń oczekujące zaproszenia i propozycje",
	formFieldLabel__organizationDomainEmailAddress: "Weryfikacyjny adres e-mail",
	formFieldLabel__organizationDomainEmailAddressDescription: "Wprowadź adres e-mail w tej domenie, aby otrzymać kod i zweryfikować tę domenę.",
	formFieldLabel__organizationName: "Nazwa organizacji",
	formFieldLabel__organizationSlug: "Slug URL",
	formFieldLabel__passkeyName: "Nazwa klucza dostępu",
	formFieldLabel__password: "Hasło",
	formFieldLabel__phoneNumber: "Numer telefonu",
	formFieldLabel__role: "Rola",
	formFieldLabel__signOutOfOtherSessions: "Wyloguj się ze wszystkich innych urządzeń",
	formFieldLabel__username: "Nazwa użytkownika",
	identityPreviewEditButton__emailAddress: void 0,
	identityPreviewEditButton__identifier: void 0,
	identityPreviewEditButton__phoneNumber: void 0,
	impersonationFab: {
		action__signOut: "Wyloguj",
		title: "Zalogowano jako {{identifier}}"
	},
	lastAuthenticationStrategy: "Ostatnio używany",
	maintenanceMode: "Aktualnie trwają prace konserwacyjne, ale nie powinno to zająć dłużej niż kilka minut.",
	membershipRole__admin: "Administrator",
	membershipRole__basicMember: "Użytkownik",
	membershipRole__guestMember: "Gość",
	oauthConsent: {
		action__allow: void 0,
		action__deny: void 0,
		offlineAccessNotice: void 0,
		redirectNotice: void 0,
		redirectUriModal: {
			subtitle: void 0,
			title: void 0
		},
		scopeList: { title: void 0 },
		subtitle: void 0,
		viewFullUrl: void 0,
		warning: void 0
	},
	organizationList: {
		action__createOrganization: "Utwórz organizację",
		action__invitationAccept: "Dołącz",
		action__suggestionsAccept: "Poproś o dołączenie",
		createOrganization: "Utwórz organizację",
		invitationAcceptedLabel: "Dołączono",
		subtitle: "aby przejść do {{applicationName}}",
		suggestionsAcceptedLabel: "Prośby o dołączenie",
		title: "Wybierz konto",
		titleWithoutPersonal: "Wybierz organizację"
	},
	organizationProfile: {
		apiKeysPage: { title: "Klucze API" },
		badge__automaticInvitation: "Automatyczne zaproszenia",
		badge__automaticSuggestion: "Automatyczne sugestie",
		badge__enterpriseSso: void 0,
		badge__manualInvitation: "Brak automatycznej rejestracji",
		badge__unverified: "Niezweryfikowany",
		billingPage: {
			accountCreditsSection: {
				title: void 0,
				viewHistory: void 0
			},
			creditHistoryPage: {
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				title: void 0
			},
			paymentHistorySection: {
				empty: void 0,
				notFound: void 0,
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				tableHeader__status: void 0
			},
			paymentMethodsSection: {
				actionLabel__default: void 0,
				actionLabel__remove: void 0,
				add: void 0,
				addSubtitle: void 0,
				cancelButton: void 0,
				formButtonPrimary__add: void 0,
				formButtonPrimary__pay: void 0,
				payWithTestCardButton: void 0,
				removeMethod: {
					messageLine1: void 0,
					messageLine2: void 0,
					successMessage: void 0,
					title: void 0
				},
				title: void 0
			},
			start: {
				headerTitle__payments: void 0,
				headerTitle__plans: void 0,
				headerTitle__statements: void 0,
				headerTitle__subscriptions: void 0
			},
			statementsSection: {
				empty: void 0,
				itemCaption__paidForPlan: void 0,
				itemCaption__payerCredit: void 0,
				itemCaption__proratedCredit: void 0,
				itemCaption__subscribedAndPaidForPlan: void 0,
				notFound: void 0,
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				title: void 0,
				totalPaid: void 0
			},
			subscriptionsListSection: {
				actionLabel__manageSubscription: void 0,
				actionLabel__newSubscription: void 0,
				actionLabel__switchPlan: void 0,
				includedSeatsUsage: void 0,
				overview: void 0,
				paidSeatsUsage: void 0,
				seatLimit: void 0,
				seatLimitAndIncludedSeats: void 0,
				tableHeader__edit: void 0,
				tableHeader__plan: void 0,
				tableHeader__startDate: void 0,
				title: void 0
			},
			subscriptionsSection: { actionLabel__default: void 0 },
			switchPlansSection: { title: void 0 },
			title: void 0
		},
		createDomainPage: {
			subtitle: "Dodaj domenę do weryfikacji. Użytkownicy z adresami e-mail w tej domenie mogą dołączyć do organizacji automatycznie lub poprosić o dołączenie.",
			title: "Dodaj domenę"
		},
		invitePage: {
			detailsTitle__inviteFailed: "Nie udało się wysłać zaproszeń. Napraw poniższe problemy i spróbuj ponownie:",
			formButtonPrimary__continue: "Wyślij zaproszenia",
			formButtonPrimary__purchaseSeats: void 0,
			selectDropdown__role: "Wybierz rolę",
			subtitle: "Zaproś nowych użytkowników do tej organizacji",
			successMessage: "Zaproszenia zostały pomyślnie wysłane",
			title: "Zaproś użytkowników"
		},
		membersPage: {
			action__invite: "Zaproś",
			action__search: "Wyszukaj",
			activeMembersTab: {
				menuAction__remove: "Usuń użytkownika",
				tableHeader__actions: "Akcje",
				tableHeader__joined: "Dołączył",
				tableHeader__role: "Rola",
				tableHeader__user: "Użytkownik"
			},
			alerts: { roleSetMigrationInProgress: {
				subtitle: "Aktualizujemy dostępne role. Po zakończeniu będziesz mógł ponownie aktualizować role.",
				title: "Role są tymczasowo zablokowane"
			} },
			detailsTitle__emptyRow: "Brak użytkowników do wyświetlenia",
			invitationsTab: {
				autoInvitations: {
					headerSubtitle: "Zaproś użytkowników, łącząc domenę e-mail ze swoją organizacją. Każdy, kto zarejestruje się za pomocą pasującej domeny e-mail, będzie mógł dołączyć do organizacji w dowolnym momencie.",
					headerTitle: "Automatyczne zaproszenia",
					primaryButton: "Zarządzanie zweryfikowanymi domenami"
				},
				table__emptyRow: "Brak zaproszeń do wyświetlenia"
			},
			invitedMembersTab: {
				menuAction__revoke: "Anuluj zaproszenie",
				tableHeader__invited: "Zaproszony"
			},
			requestsTab: {
				autoSuggestions: {
					headerSubtitle: "Użytkownicy, którzy zarejestrują się za pomocą pasującej domeny e-mail, będą mogli zobaczyć propozycję, aby poprosić o dołączenie do Twojej organizacji.",
					headerTitle: "Automatyczne propozycje",
					primaryButton: "Zarządzanie zweryfikowanymi domenami"
				},
				menuAction__approve: "Zatwierdź",
				menuAction__reject: "Odrzuć",
				tableHeader__requested: "Prośby o dostęp",
				table__emptyRow: "Brak próśb do wyświetlenia"
			},
			start: {
				headerTitle__invitations: "Zaproszenia",
				headerTitle__members: "Członkowie",
				headerTitle__requests: "Prośby"
			}
		},
		navbar: {
			apiKeys: void 0,
			billing: void 0,
			description: "Zarządzaj organizacją.",
			general: "Główne",
			members: "Członkowie",
			security: void 0,
			title: "Organizacja"
		},
		plansPage: {
			alerts: {
				noPermissionsToManageBilling: void 0,
				planMembershipLimitExceeded: void 0
			},
			title: void 0
		},
		profilePage: {
			dangerSection: {
				deleteOrganization: {
					actionDescription: "Wpisz \"{{organizationName}}\" poniżej aby kontynuować.",
					messageLine1: "Czy na pewno chcesz usunąć tę organizację?",
					messageLine2: "To działanie jest trwałe i nieodwracalne.",
					successMessage: "Organizacja została usunięta.",
					title: "Usuń organizację"
				},
				leaveOrganization: {
					actionDescription: "Wpisz \"{{organizationName}}\" poniżej aby kontynuować.",
					messageLine1: "Czy na pewno chcesz opuścić tę organizację? Stracisz dostęp do tej organizacji i jej aplikacji.",
					messageLine2: "Ta akcja jest trwała i nieodwracalna.",
					successMessage: "Opuściłeś organizację.",
					title: "Opuść organizację"
				},
				title: "Zagrożenie"
			},
			domainSection: {
				menuAction__manage: "Zarządzaj",
				menuAction__remove: "Usuń",
				menuAction__verify: "Zweryfikuj",
				primaryButton: "Dodaj domenę",
				subtitle: "Zezwalaj użytkownikom na automatyczne dołączanie do organizacji lub żądaj dołączenia na podstawie zweryfikowanej domeny e-mail.",
				title: "Zweryfikowane domeny"
			},
			successMessage: "Organizacja została zaktualizowana.",
			title: "Profil organizacji"
		},
		removeDomainPage: {
			messageLine1: "Domena e-mail {{domain}} zostanie usunięta.",
			messageLine2: "Po wykonaniu tej czynności użytkownicy nie będą mogli automatycznie dołączyć do organizacji.",
			successMessage: "Domena {{domain}} została usunięta.",
			title: "Usuń domenę"
		},
		securityPage: {
			removeDialog: {
				confirmButton: void 0,
				subtitle: void 0,
				title: void 0
			},
			ssoSection: {
				badge__active: void 0,
				badge__inProgress: void 0,
				badge__inactive: void 0,
				badge__unconfigured: void 0,
				descriptionLine1: void 0,
				domainLabel: void 0,
				menuAction__activate: void 0,
				menuAction__deactivate: void 0,
				menuAction__edit: void 0,
				menuAction__remove: void 0,
				primaryButton__continueConfiguration: void 0,
				primaryButton__startConfiguration: void 0,
				title: void 0,
				tooltip: void 0,
				tooltipLabel: void 0,
				tooltip__noRole: void 0
			},
			title: void 0
		},
		start: {
			headerTitle__general: "Ogólne",
			headerTitle__members: "Członkowie",
			membershipSeatUsageLabel: void 0,
			profileSection: {
				primaryButton: void 0,
				title: "Profil organizacji",
				uploadAction__title: "Logo"
			}
		},
		verifiedDomainPage: {
			dangerTab: {
				calloutInfoLabel: "Usunięcie tej domeny wpłynie na zaproszonych użytkowników.",
				removeDomainActionLabel__remove: "Usuń domenę",
				removeDomainSubtitle: "Usuń tą domenę ze zweryfikowanych domen",
				removeDomainTitle: "Usuń domenę"
			},
			enrollmentTab: {
				automaticInvitationOption__description: "Użytkownicy są automatycznie zapraszani do dołączenia do organizacji podczas rejestracji i mogą dołączyć w dowolnym momencie.",
				automaticInvitationOption__label: "Automatyczne zaproszenia",
				automaticSuggestionOption__description: "Użytkownicy otrzymują propozycję, aby poprosić o dołączenie, ale muszą zostać zatwierdzeni przez administratora, zanim będą mogli dołączyć do organizacji.",
				automaticSuggestionOption__label: "Automatyczne propozycje",
				calloutInfoLabel: "Zmiana trybu rejestracji będzie miała wpływ tylko na nowych użytkowników.",
				calloutInvitationCountLabel: "Oczekujące zaproszenia wysłane do użytkowników: {{count}}",
				calloutSuggestionCountLabel: "Oczekujące propozycje wysłane do użytkowników: {{count}}",
				manualInvitationOption__description: "Użytkowników można zapraszać do organizacji wyłącznie ręcznie.",
				manualInvitationOption__label: "Brak automatycznej rejestracji",
				subtitle: "Wybierz sposób, w jaki użytkownicy z tej domeny mogą dołączyć do organizacji."
			},
			start: {
				headerTitle__danger: "Zagrożenie",
				headerTitle__enrollment: "Opcje rejestracji"
			},
			subtitle: "Domena {{domain}} została zweryfikowana. Kontynuuj, wybierając opcje rejestracji",
			title: "Zaktualizuj {{domain}}"
		},
		verifyDomainPage: {
			formSubtitle: "Wprowadź kod weryfikacyjny wysłany na Twój adres e-mail",
			formTitle: "Kod weryfikacyjny",
			resendButton: "Nie otrzymałeś kodu? Wyślij ponownie",
			subtitle: "Domena {{domainName}} musi zostać zweryfikowana przez e-mail.",
			subtitleVerificationCodeScreen: "Kod weryfikacyjny został wysłany na adres {{emailAddress}}. Wprowadź kod, aby kontynuować.",
			title: "Zweryfikuj domenę"
		}
	},
	organizationSwitcher: {
		action__closeOrganizationSwitcher: "Zamknij",
		action__createOrganization: "Utwórz organizację",
		action__invitationAccept: "Dołącz",
		action__manageOrganization: "Zarządzaj organizacją",
		action__openOrganizationSwitcher: "Otwórz",
		action__suggestionsAccept: "Poproś o dołączenie",
		notSelected: "Nie wybrano organizacji",
		personalWorkspace: "Przestrzeń osobista",
		suggestionsAcceptedLabel: "Oczekiwanie na zatwierdzenie"
	},
	paginationButton__next: "Następny",
	paginationButton__previous: "Poprzedni",
	paginationRowText__displaying: "Wyświetlanie",
	paginationRowText__of: "z",
	reverification: {
		alternativeMethods: {
			actionLink: "Uzyskaj pomoc",
			actionText: "Nie używasz żadnej z tych metod?",
			blockButton__backupCode: "Użyj kodu zapasowego",
			blockButton__emailCode: "Wyślij kod e-mailem do {{identifier}}",
			blockButton__passkey: void 0,
			blockButton__password: "Zaloguj się za pomocą hasła",
			blockButton__phoneCode: "Wyślij kod SMS-em do {{identifier}}",
			blockButton__totp: "Użyj aplikacji uwierzytelniającej",
			getHelp: {
				blockButton__emailSupport: "Skontaktuj się z pomocą",
				content: "Jeśli masz problem z weryfikacją konta, wyślij do nas e-mail, a postaramy się jak najszybciej przywrócić dostęp.",
				title: "Uzyskaj wsparcie"
			},
			subtitle: "Masz problem? Możesz użyć dowolnej z tych metod weryfikacji.",
			title: "Użyj innej metody"
		},
		backupCodeMfa: {
			subtitle: "Twój kod zapasowy to ten, który otrzymałeś podczas konfigurowania uwierzytelniania dwuetapowego.",
			title: "Wprowadź kod zapasowy"
		},
		emailCode: {
			formTitle: "Kod weryfikacyjny",
			resendButton: "Nie otrzymałeś kodu? Wyślij ponownie",
			subtitle: "Wprowadź kod wysłany na Twój adres e-mail, aby kontynuować",
			title: "Wymagana weryfikacja"
		},
		noAvailableMethods: {
			message: "Nie można kontynuować weryfikacji. Brak dostępnych czynników uwierzytelniania.",
			subtitle: "Wystąpił błąd",
			title: "Nie możemy zweryfikować twojego konta"
		},
		passkey: {
			blockButton__passkey: void 0,
			subtitle: void 0,
			title: void 0
		},
		password: {
			actionLink: "Użyj innej metody",
			subtitle: "Wprowadź hasło, aby kontynuować",
			title: "Wymagana weryfikacja"
		},
		phoneCode: {
			formTitle: "Kod weryfikacyjny",
			resendButton: "Nie otrzymałeś kodu? Wyślij ponownie",
			subtitle: "Wprowadź kod wysłany na Twój telefon, aby kontynuować",
			title: "Wymagana weryfikacja"
		},
		phoneCodeMfa: {
			formTitle: "Kod weryfikacyjny",
			resendButton: "Nie otrzymałeś kodu? Wyślij ponownie",
			subtitle: "Aby kontynuować, wprowadź kod weryfikacyjny wysłany na twój telefon",
			title: "Wymagana weryfikacja"
		},
		totpMfa: {
			formTitle: "Kod weryfikacyjny",
			subtitle: "Aby kontynuować, wprowadź kod weryfikacyjny wygenerowany przez swoją aplikację uwierzytelniającą",
			title: "Wymagana weryfikacja"
		}
	},
	searchInput: { action__clear: void 0 },
	signIn: {
		accountSwitcher: {
			action__addAccount: "Dodaj konto",
			action__signOutAll: "Wyloguj się ze wszystkich kont",
			subtitle: "Wybierz konto, na którym chcesz kontynuować.",
			title: "Wybierz konto"
		},
		alternativeMethods: {
			actionLink: "Uzyskaj pomoc",
			actionText: "Nie używasz żadnej z tych metod?",
			blockButton__backupCode: "Użyj kodu zapasowego",
			blockButton__emailCode: "Wyślij kod do {{identifier}}",
			blockButton__emailLink: "Wyślij link do {{identifier}}",
			blockButton__passkey: "Zaloguj się za pomocą klucza dostępowego",
			blockButton__password: "Zaloguj się za pomocą hasła",
			blockButton__phoneCode: "Wyślij kod do {{identifier}}",
			blockButton__totp: "Użyj aplikacji uwierzytelniającej",
			getHelp: {
				blockButton__emailSupport: "Wyślij e-mail do pomocy technicznej",
				content: "Jeśli masz problem z zalogowaniem się do swojego konta, wyślij do nas e-mail, a postaramy się jak najszybciej przywrócić dostęp.",
				title: "Uzyskaj pomoc"
			},
			subtitle: "Masz problem? Możesz użyć dowolnej z tych metod weryfikacji.",
			title: "Użyj innego sposobu"
		},
		alternativePhoneCodeProvider: {
			formTitle: void 0,
			resendButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		backupCodeMfa: {
			subtitle: "aby przejść do {{applicationName}}",
			title: "Wprowadź kod zapasowy"
		},
		emailCode: {
			formTitle: "Kod weryfikacyjny",
			resendButton: "Wyślij kod ponownie",
			subtitle: "aby kontynuować w {{applicationName}}",
			title: "Sprawdź swoją pocztę e-mail"
		},
		emailCodeMfa: {
			formTitle: "Sprawdź swoją pocztę e-mail",
			resendButton: "Nie otrzymałeś kodu? Wyślij ponownie",
			subtitle: "aby kontynuować w {{applicationName}}",
			title: "Sprawdź swoją pocztę e-mail"
		},
		emailLink: {
			clientMismatch: {
				subtitle: "Aby kontynuować, otwórz link weryfikacyjny na urządzeniu i w przeglądarce, w której rozpocząłeś logowanie",
				title: "Link weryfikacyjny jest nieprawidłowy dla tego urządzenia"
			},
			expired: {
				subtitle: "Powróć do oryginalnej karty, aby kontynuować.",
				title: "Ten link weryfikacyjny wygasł"
			},
			failed: {
				subtitle: "Powróć do oryginalnej karty, aby kontynuować.",
				title: "Ten link weryfikacyjny jest nieprawidłowy"
			},
			formSubtitle: "Użyj linku weryfikacyjnego wysłanego na Twój adres e-mail",
			formTitle: "Link weryfikacyjny",
			loading: {
				subtitle: "Zostaniesz przekierowany wkrótce",
				title: "Logowanie..."
			},
			resendButton: "Wyślij link ponownie",
			subtitle: "aby kontynuować w {{applicationName}}",
			title: "Sprawdź swoją pocztę e-mail",
			unusedTab: { title: "Możesz zamknąć tę kartę" },
			verified: {
				subtitle: "Zostaniesz przekierowany wkrótce",
				title: "Pomyślnie zalogowano"
			},
			verifiedSwitchTab: {
				subtitle: "Powróć do oryginalnej karty, aby kontynuować",
				subtitleNewTab: "Powróć do nowo otwartej karty, aby kontynuować",
				titleNewTab: "Zalogowano na innej karcie"
			}
		},
		emailLinkMfa: {
			formSubtitle: "Użyj linku weryfikacyjnego wysłanego na Twój e-mail",
			resendButton: "Nie otrzymałeś linku? Wyślij ponownie",
			subtitle: "aby kontynuować do {{applicationName}}",
			title: "Sprawdź swoją pocztę"
		},
		enterpriseConnections: {
			subtitle: void 0,
			title: void 0
		},
		forgotPassword: {
			formTitle: "Kod werfikacyjny resetowania hasła",
			resendButton: "Nie otrzymałeś kodu? Wyślij ponownie",
			subtitle: "aby zresetować hasło",
			subtitle_email: "Najpierw wprowadź kod wysłany na Twój adres e-mail",
			subtitle_phone: "Najpierw wprowadź kod wysłany na Twój numer telefonu",
			title: "Zmień hasło"
		},
		forgotPasswordAlternativeMethods: {
			blockButton__resetPassword: "Zresetuj hasło",
			label__alternativeMethods: "Lub zaloguj się za pomocą innej metody",
			title: "Zapomniałeś hasła?"
		},
		newDeviceVerificationNotice: "Logujesz się z nowego urządzenia. Prosimy o weryfikację, aby zapewnić bezpieczeństwo Twojego konta.",
		noAvailableMethods: {
			message: "Nie można kontynuować logowania. Brak dostępnych czynników uwierzytelniających.",
			subtitle: "Wystąpił błąd",
			title: "Nie można się zalogować"
		},
		passkey: {
			subtitle: "Użycie klucza dostępu potwierdza, że to Ty. Urządzenie może poprosić o twój odcisk palca, twarz lub blokadę ekranu.",
			title: "Użyj swojego klucza dostępowego"
		},
		password: {
			actionLink: "Użyj innego sposobu",
			subtitle: "aby kontynuować w {{applicationName}}",
			title: "Wprowadź swoje hasło"
		},
		passwordCompromised: { title: void 0 },
		passwordPwned: { title: "Hasło skompromitowane" },
		passwordUntrusted: { title: void 0 },
		phoneCode: {
			formTitle: "Kod weryfikacyjny",
			resendButton: "Wyślij kod ponownie",
			subtitle: "aby przejść do {{applicationName}}",
			title: "Sprawdź swój telefon"
		},
		phoneCodeMfa: {
			formTitle: "Kod weryfikacyjny",
			resendButton: "Wyślij kod ponownie",
			subtitle: "Aby kontynuować, wprowadź kod weryfikacyjny wysłany na Twój telefon",
			title: "Sprawdź swój telefon"
		},
		protectCheck: {
			loading: void 0,
			retryButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		resetPassword: {
			formButtonPrimary: "Zmień hasło",
			requiredMessage: "Z powodów bezpieczeństwa konieczne jest zresetowanie hasła.",
			successMessage: "Twoje hasło zostało pomyślnie zresetowane. Logujemy Cię, proszę czekać...",
			title: "Ustaw nowe hasło"
		},
		resetPasswordMfa: { detailsLabel: "Przed zresetowaniem hasła musimy zweryfikować tożsamość użytkownika." },
		start: {
			actionLink: "Zarejestruj się",
			actionLink__join_waitlist: "Dołącz do listy oczekujących",
			actionLink__use_email: "Użyj adresu e-mail",
			actionLink__use_email_username: "Użyj adresu e-mail lub nazwy użytkownika",
			actionLink__use_passkey: "Użyj klucza dostępowego",
			actionLink__use_phone: "Użyj numeru telefonu",
			actionLink__use_username: "Użyj nazwy użytkownika",
			actionText: "Nie masz konta?",
			actionText__join_waitlist: "Chcesz otrzymać wczesny dostęp?",
			alternativePhoneCodeProvider: {
				actionLink: void 0,
				label: void 0,
				subtitle: void 0,
				title: void 0
			},
			subtitle: "by przejść do aplikacji {{applicationName}}",
			subtitleCombined: void 0,
			title: "Zaloguj się",
			titleCombined: "Kontynuuj do {{applicationName}}"
		},
		totpMfa: {
			formTitle: "Kod weryfikacyjny",
			subtitle: "Aby kontynuować, wprowadź kod weryfikacyjny wygenerowany przez aplikację uwierzytelniającą",
			title: "Weryfikacja dwustopniowa"
		},
		web3Solana: {
			subtitle: "Wybierz poniżej portfel, aby się zalogować",
			title: "Zaloguj się przez Solana"
		}
	},
	signInEnterPasswordTitle: "Wprowadź swoje hasło",
	signUp: {
		alternativePhoneCodeProvider: {
			resendButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		continue: {
			actionLink: "Zaloguj się",
			actionText: "Masz już konto?",
			subtitle: "by przejść do aplikacji {{applicationName}}",
			title: "Uzupełnij brakujące pola"
		},
		emailCode: {
			formSubtitle: "Wprowadź kod weryfikacyjny wysłany na Twój adres e-mail",
			formTitle: "Kod weryfikacyjny",
			resendButton: "Nie otrzymałeś kodu? Wyślij ponownie",
			subtitle: "by przejść do aplikacji {{applicationName}}",
			title: "Zweryfikuj swój adres e-mail"
		},
		emailLink: {
			clientMismatch: {
				subtitle: "Aby kontynuować, otwórz link weryfikacyjny na urządzeniu i w przeglądarce, w której rozpocząłeś rejestrację",
				title: "Link weryfikacyjny jest nieprawidłowy dla tego urządzenia"
			},
			formSubtitle: "Użyj linku weryfikacyjnego wysłanego na Twój adres e-mail",
			formTitle: "Link weryfikacyjny",
			loading: { title: "Rejestrowanie..." },
			resendButton: "Nie otrzymałeś linku? Wyślij ponownie",
			subtitle: "by przejść do aplikacji {{applicationName}}",
			title: "Zweryfikuj swój adres e-mail",
			verified: { title: "Pomyślnie zarejestrowano" },
			verifiedSwitchTab: {
				subtitle: "Powróć do nowo otwartej karty, aby kontynuować",
				subtitleNewTab: "Powróć do poprzedniej karty, aby kontynuować",
				title: "Adres e-mail został pomyślnie zweryfikowany"
			}
		},
		enterpriseConnections: {
			subtitle: void 0,
			title: void 0
		},
		legalConsent: {
			checkbox: {
				label__onlyPrivacyPolicy: "Akceptuję {{ privacyPolicyLink || link(\"Politykę prywatności\") }}",
				label__onlyTermsOfService: "Akceptuję {{ termsOfServiceLink || link(\"Warunki świadczenia usługi\") }}\"",
				label__termsOfServiceAndPrivacyPolicy: "Akceptuję {{ termsOfServiceLink || link(\"Warunki świadczenia usługi\") }} i {{ privacyPolicyLink || link(\"Politykę prywatności\") }}"
			},
			continue: {
				subtitle: "Przeczytaj i zaakceptuj warunki, aby kontynuować",
				title: "Zgoda prawna"
			}
		},
		phoneCode: {
			formSubtitle: "Wprowadź kod weryfikacyjny wysłany na Twój numer telefonu",
			formTitle: "Kod weryfikacyjny",
			resendButton: "Nie otrzymałeś kodu? Wyślij ponownie",
			subtitle: "by przejść do aplikacji {{applicationName}}",
			title: "Zweryfikuj swój numer telefonu"
		},
		protectCheck: {
			loading: void 0,
			retryButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		restrictedAccess: {
			actionLink: "Zaloguj się",
			actionText: "Masz już konto?",
			blockButton__emailSupport: "Skontaktuj się z pomocą",
			blockButton__joinWaitlist: "Dołącz do listy oczekujących",
			subtitle: "Rejestracja jest obecnie wyłączona. Jeśli uważasz, że powinieneś mieć dostęp, skontaktuj się z pomocą.",
			subtitleWaitlist: "Rejestracja jest obecnie wyłączona. Aby dowiedzieć się jako pierwszy o naszym starcie, dołącz do listy oczekujących.",
			title: "Dostęp ograniczony"
		},
		start: {
			actionLink: "Zaloguj się",
			actionLink__use_email: "Użyj adresu e-mail",
			actionLink__use_phone: "Użyj numeru telefonu",
			actionText: "Masz już konto?",
			alternativePhoneCodeProvider: {
				actionLink: void 0,
				label: void 0,
				subtitle: void 0,
				title: void 0
			},
			subtitle: "by przejść do aplikacji {{applicationName}}",
			subtitleCombined: "by przejść do aplikacji {{applicationName}}",
			title: "Utwórz swoje konto",
			titleCombined: "Utwórz swoje konto"
		},
		web3Solana: {
			subtitle: "Wybierz poniżej portfel, aby się zarejestrować",
			title: "Zarejestruj się przez Solana"
		}
	},
	socialButtonsBlockButton: "Kontynuuj z {{provider|titleize}}",
	socialButtonsBlockButtonManyInView: "{{provider|titleize}}",
	taskChooseOrganization: {
		alerts: { organizationAlreadyExists: "Organizacja już istnieje dla wykrytej nazwy firmy ({{organizationName}}) i {{organizationDomain}}. Dołącz przez zaproszenie." },
		chooseOrganization: {
			action__createOrganization: "Utwórz organizację",
			action__invitationAccept: "Dołącz",
			action__suggestionsAccept: "Poproś o dołączenie",
			subtitle: "Dołącz do istniejącej organizacji lub utwórz nową",
			subtitle__createOrganizationDisabled: "Dołącz do istniejącej organizacji",
			suggestionsAcceptedLabel: "Oczekiwanie na dołączenie",
			title: "Wybierz organizację"
		},
		createOrganization: {
			formButtonReset: "Anuluj",
			formButtonSubmit: "Kontynuuj",
			formFieldInputPlaceholder__name: "Moja Organizacja",
			formFieldInputPlaceholder__slug: "moja-organizacja",
			formFieldLabel__name: "Nazwa",
			formFieldLabel__slug: "Slug",
			subtitle: "Wprowadź szczegóły swojej organizacji",
			title: "Utwórz swoją organizację"
		},
		organizationCreationDisabled: {
			subtitle: "Skontaktuj się z administratorem swojej organizacji, aby uzyskać zaproszenie.",
			title: "Musisz należeć do organizacji"
		},
		signOut: {
			actionLink: "Wyloguj",
			actionText: "Zalogowano jako {{identifier}}"
		}
	},
	taskResetPassword: {
		formButtonPrimary: "Zresetuj hasło",
		signOut: {
			actionLink: "Wyloguj",
			actionText: "Zalogowano jako {{identifier}}"
		},
		subtitle: void 0,
		title: "Zresetuj hasło"
	},
	taskSetupMfa: {
		badge: void 0,
		signOut: {
			actionLink: void 0,
			actionText: void 0
		},
		smsCode: {
			addPhone: {
				formButtonPrimary: void 0,
				infoText: void 0
			},
			addPhoneNumber: void 0,
			cancel: void 0,
			subtitle: void 0,
			success: {
				finishButton: void 0,
				message1: void 0,
				message2: void 0,
				title: void 0
			},
			title: void 0,
			verifyPhone: {
				formButtonPrimary: void 0,
				formTitle: void 0,
				resendButton: void 0,
				subtitle: void 0,
				title: void 0
			}
		},
		start: {
			methodSelection: {
				phoneCode: void 0,
				totp: void 0
			},
			subtitle: void 0,
			title: void 0
		},
		totpCode: {
			addAuthenticatorApp: {
				buttonAbleToScan__nonPrimary: void 0,
				buttonUnableToScan__nonPrimary: void 0,
				formButtonPrimary: void 0,
				formButtonReset: void 0,
				infoText__ableToScan: void 0,
				infoText__unableToScan: void 0,
				inputLabel__unableToScan1: void 0
			},
			success: {
				finishButton: void 0,
				message1: void 0,
				message2: void 0,
				title: void 0
			},
			title: void 0,
			verifyTotp: {
				formButtonPrimary: void 0,
				formButtonReset: void 0,
				formTitle: void 0,
				subtitle: void 0,
				title: void 0
			}
		}
	},
	unstable__errors: {
		action_blocked: void 0,
		already_a_member_in_organization: "{{email}} jest już członkiem organizacji.",
		api_key_name_already_exists: void 0,
		api_key_usage_exceeded: void 0,
		avatar_file_size_exceeded: "Rozmiar pliku przekracza maksymalny limit 10 MB. Wybierz mniejszy plik.",
		avatar_file_type_invalid: "Nieobsługiwany typ pliku. Prześlij obraz w formacie JPG, PNG, GIF lub WEBP.",
		captcha_invalid: "Rejestracja nie powiodła się z powodu niepowodzenia weryfikacji zabezpieczeń. Odśwież stronę, aby spróbować ponownie lub skontaktuj się z pomocą, aby uzyskać wsparcie.",
		captcha_unavailable: "Rejestracja nie powiodła się z powodu niedostępności weryfikacji botów. Odśwież stronę, aby spróbować ponownie lub skontaktuj się z pomocą, aby uzyskać wsparcie.",
		form_code_incorrect: void 0,
		form_email_address_blocked: "Tymczasowe usługi e-mail nie są obsługiwane. Proszę użyć zwykłego adresu e-mail, aby utworzyć konto.",
		form_identifier_exists__email_address: "Adres e-mail jest już zajęty. Proszę spróbować innego.",
		form_identifier_exists__phone_number: "Ten numer telefonu jest zajęty. Spróbuj użyć innego.",
		form_identifier_exists__username: "Ta nazwa użytkownika jest zajęta. Spróbuj użyć innej.",
		form_identifier_not_found: "Nie znaleziono konta o tym identyfikatorze. Sprawdź i spróbuj ponownie.",
		form_new_password_matches_current: void 0,
		form_param_format_invalid: "Wprowadzona wartość ma nieprawidłowy format. Sprawdź i popraw ją.",
		form_param_format_invalid__email_address: "Adres e-mail powinien być poprawnym adresem e-mail.",
		form_param_format_invalid__phone_number: "Numer telefonu powinien mieć prawidłowy format międzynarodowy",
		form_param_max_length_exceeded__first_name: "Imię nie powinno przekraczać 256 znaków.",
		form_param_max_length_exceeded__last_name: "Nazwisko nie powinno przekraczać 256 znaków.",
		form_param_max_length_exceeded__name: "Nazwa nie powinna przekraczać 256 znaków.",
		form_param_nil: "To pole jest wymagane i nie może być puste.",
		form_param_type_invalid: void 0,
		form_param_type_invalid__email_address: void 0,
		form_param_type_invalid__phone_number: void 0,
		form_param_value_invalid: "Wprowadzona wartość jest nieprawidłowa. Popraw ją.",
		form_password_compromised__sign_in: void 0,
		form_password_incorrect: "Wprowadzone hasło jest nieprawidłowe. Spróbuj ponownie.",
		form_password_length_too_short: "Twoje hasło jest zbyt krótkie. Musi mieć co najmniej 8 znaków.",
		form_password_not_strong_enough: "Twoje hasło nie jest wystarczająco silne",
		form_password_or_identifier_incorrect: "Hasło lub adres e-mail jest nieprawidłowy. Spróbuj ponownie lub użyj innej metody.",
		form_password_pwned: "To hasło zostało znalezione w wyniku włamania i nie można go użyć. Zamiast tego spróbuj użyć innego hasła.",
		form_password_pwned__sign_in: "To hasło zostało znalezione w wyniku włamania i nie można go użyć. Zresetuj hasło.",
		form_password_size_in_bytes_exceeded: "Twoje hasło przekroczyło maksymalną dozwoloną liczbę bajtów, skróć je lub usuń niektóre znaki specjalne.",
		form_password_untrusted__sign_in: void 0,
		form_password_validation_failed: "Podane hasło jest nieprawidłowe",
		form_username_invalid_character: "Twoja nazwa użytkownika zawiera nieprawidłowe znaki. Prosimy o używanie wyłącznie liter, cyfr i podkreśleń.",
		form_username_invalid_length: "Nazwa użytkownika musi zawierać od {{min_length}} do {{max_length}} znaków.",
		form_username_needs_non_number_char: "Nazwa użytkownika musi zawierać co najmniej jeden znak niebędący cyfrą.",
		identification_deletion_failed: "Nie można usunąć ostatniego identyfikatora.",
		insufficient_seats_change_plan: void 0,
		insufficient_seats_contact_support: void 0,
		not_allowed_access: "Adres e-mail lub numer telefonu nie jest dozwolony do rejestracji. Może to być spowodowane użyciem '+', '=', '#' lub '.' w adresie e-mail, użyciem domeny skojarzonej z usługą poczty e-mail tymczasowej lub jawnego wykluczenia.",
		oauth_access_denied: void 0,
		organization_domain_blocked: "To jest zablokowana domena dostawcy poczty e-mail. Użyj innej domeny.",
		organization_domain_common: "To jest popularna domena dostawcy poczty e-mail. Użyj innej domeny.",
		organization_domain_exists_for_enterprise_connection: "Ta domena jest już używana do logowania jednokrotnego w organizacji.",
		organization_membership_quota_exceeded: "Osiągnięto limit członkostwa w organizacji, w tym zaległych zaproszeń.",
		organization_minimum_permissions_needed: "Musi istnieć co najmniej jeden członek organizacji z minimalnymi wymaganymi uprawnieniami.",
		organization_not_found_or_unauthorized: void 0,
		organization_not_found_or_unauthorized_with_create_organization_disabled: void 0,
		passkey_already_exists: "Klucz dostępu jest już zarejestrowany w tym urządzeniu.",
		passkey_not_supported: "Klucze dostępu nie są obsługiwane przez to urządzenie.",
		passkey_pa_not_supported: "Rejestracja wymaga platformy uwierzytelniającej, ale urządzenie jej nie obsługuje.",
		passkey_registration_cancelled: "Rejestracja klucza dostępu została anulowana lub upłynął jej limit czasu.",
		passkey_retrieval_cancelled: "Weryfikacja klucza dostępu została anulowana lub upłynął limit czasu.",
		passwordComplexity: {
			maximumLength: "mniej niż {{length}} znaków",
			minimumLength: "{{length}} lub więcej znaków",
			requireLowercase: "małą literę",
			requireNumbers: "cyfrę",
			requireSpecialCharacter: "znak specjalny",
			requireUppercase: "wielką literę",
			sentencePrefix: "Twoje hasło musi zawierać"
		},
		phone_number_exists: "Numer telefonu jest już zajęty. Proszę spróbować innego.",
		protect_check_aborted: void 0,
		protect_check_already_resolved: void 0,
		protect_check_execution_failed: void 0,
		protect_check_invalid_script: void 0,
		protect_check_invalid_sdk_url: void 0,
		protect_check_script_load_failed: void 0,
		protect_check_timed_out: void 0,
		protect_check_unsupported_environment: void 0,
		session_exists: "Jesteś już zalogowany.",
		web3_missing_identifier: "Nie można znaleźć rozszerzenia Web3 Wallet. Zainstaluj je, aby kontynuować.",
		web3_signature_request_rejected: "Odrzuciłeś prośbę o podpis. Spróbuj ponownie, aby kontynuować.",
		web3_solana_signature_generation_failed: "Wystąpił błąd podczas generowania podpisu. Spróbuj ponownie, aby kontynuować.",
		zxcvbn: {
			couldBeStronger: "Twoje hasło jest odpowiednie, ale mogłoby być silniejsze. Spróbuj dodać więcej znaków.",
			goodPassword: "Twoje hasło jest wystarczająco silne.",
			notEnough: "Twoje hasło jest zbyt słabe. Spróbuj dodać więcej znaków.",
			suggestions: {
				allUppercase: "Unikaj używania samych wielkich liter.",
				anotherWord: "Dodaj więcej słów, które są rzadsze.",
				associatedYears: "Unikaj lat związanych z Tobą.",
				capitalization: "Używaj wielkich liter częsciej.",
				dates: "Unikaj dat związanych z Tobą.",
				l33t: "Unikaj przewidywalnego zamieniania liter, takich jak '@' za 'a'.",
				longerKeyboardPattern: "Używaj długich wzorów na klawiaturze, zmieniając kierunek pisania wielokrotnie.",
				noNeed: "Możesz tworzyć silne hasła bez używania symboli, cyfr lub wielkich liter.",
				pwned: "Jeżeli używasz tego hasła gdzie indziej, zmień je jak najszybciej.",
				recentYears: "Unikaj ostatnich lat.",
				repeated: "Unikaj powtarzanych słów i znaków.",
				reverseWords: "Unikaj wpisywania popularnych słów od tyłu.",
				sequences: "Unikaj popularnych kombinacji znaków.",
				useWords: "Używaj wielu słów, ale unikaj popularnych fraz."
			},
			warnings: {
				common: "Jest to często używane hasło.",
				commonNames: "Imiona i nazwiska są łatwe do odgadnięcia.",
				dates: "Daty są łatwe do odgadnięcia.",
				extendedRepeat: "Powtarzające się wzorce znaków, takie jak \"abcabcabc\", są łatwe do odgadnięcia.",
				keyPattern: "Krótkie wzory klawiatury są łatwe do odgadnięcia.",
				namesByThemselves: "Pojedyncze imiona lub nazwiska są łatwe do odgadnięcia.",
				pwned: "Twoje hasło zostało ujawnione w wyniku wycieku danych w Internecie.",
				recentYears: "Ostatnie lata są łatwe do odgadnięcia.",
				sequences: "Typowe sekwencje znaków, takie jak \"abc\", są łatwe do odgadnięcia.",
				similarToCommon: "Jest to podobne do powszechnie używanego hasła.",
				simpleRepeat: "Powtarzające się znaki, takie jak \"aaa\", są łatwe do odgadnięcia.",
				straightRow: "Proste rzędy klawiszy na klawiaturze są łatwe do odgadnięcia.",
				topHundred: "To jest często używane hasło.",
				topTen: "To jest bardzo często używane hasło.",
				userInputs: "Nie powinno być żadnych danych osobowych ani danych związanych ze stroną.",
				wordByItself: "Pojedyncze słowa są łatwe do odgadnięcia."
			}
		}
	},
	userButton: {
		action__addAccount: "Dodaj konto",
		action__closeUserMenu: void 0,
		action__manageAccount: "Zarządzaj kontem",
		action__openUserMenu: void 0,
		action__signOut: "Wyloguj",
		action__signOutAll: "Wyloguj ze wszystkich kont",
		label__accountActions: "Działania na koncie",
		label__activeSessions: "Aktywne sesje",
		label__userButtonPopover: "Panel konta"
	},
	userProfile: {
		apiKeysPage: { title: void 0 },
		backupCodePage: {
			actionLabel__copied: "Skopiowane!",
			actionLabel__copy: "Skopiuj wszystkie",
			actionLabel__download: "Pobierz .txt",
			actionLabel__print: "Drukuj",
			infoText1: "Kody zapasowe zostaną włączone dla tego konta.",
			infoText2: "Przechowuj kody zapasowe w tajemnicy i bezpiecznie. Możesz wygenerować nowe kody, jeśli podejrzewasz, że zostały skompromitowane.",
			subtitle__codelist: "Przechowuj je bezpiecznie i zachowaj w tajemnicy.",
			successMessage: "Kody zapasowe są teraz włączone. Możesz użyć jednego z tych kodów do zalogowania się na swoje konto, jeśli utracisz dostęp do urządzenia uwierzytelniającego. Każdy kod można użyć tylko raz.",
			successSubtitle: "Możesz użyć jednego z tych kodów do zalogowania się na swoje konto, jeśli utracisz dostęp do urządzenia uwierzytelniającego.",
			title: "Dodaj weryfikację kodem zapasowym",
			title__codelist: "Kody zapasowe"
		},
		billingPage: {
			accountCreditsSection: {
				title: void 0,
				viewHistory: void 0
			},
			creditHistoryPage: {
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				title: void 0
			},
			paymentHistorySection: {
				empty: void 0,
				notFound: void 0,
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				tableHeader__status: void 0
			},
			paymentMethodsSection: {
				actionLabel__default: void 0,
				actionLabel__remove: void 0,
				add: void 0,
				addSubtitle: void 0,
				cancelButton: void 0,
				formButtonPrimary__add: void 0,
				formButtonPrimary__pay: void 0,
				payWithTestCardButton: void 0,
				removeMethod: {
					messageLine1: void 0,
					messageLine2: void 0,
					successMessage: void 0,
					title: void 0
				},
				title: void 0
			},
			start: {
				headerTitle__payments: void 0,
				headerTitle__plans: void 0,
				headerTitle__statements: void 0,
				headerTitle__subscriptions: void 0
			},
			statementsSection: {
				empty: void 0,
				itemCaption__paidForPlan: void 0,
				itemCaption__payerCredit: void 0,
				itemCaption__proratedCredit: void 0,
				itemCaption__subscribedAndPaidForPlan: void 0,
				notFound: void 0,
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				title: void 0,
				totalPaid: void 0
			},
			subscriptionsListSection: {
				actionLabel__manageSubscription: void 0,
				actionLabel__newSubscription: void 0,
				actionLabel__switchPlan: void 0,
				overview: void 0,
				tableHeader__edit: void 0,
				tableHeader__plan: void 0,
				tableHeader__startDate: void 0,
				title: void 0
			},
			subscriptionsSection: { actionLabel__default: void 0 },
			switchPlansSection: { title: void 0 },
			title: void 0
		},
		connectedAccountPage: {
			formHint: "Wybierz dostawcę, aby połączyć konto.",
			formHint__noAccounts: "Nie ma dostępnych zewnętrznych dostawców kont.",
			removeResource: {
				messageLine1: "{{identifier}} zostanie usunięte z tego konta.",
				messageLine2: "Nie będziesz już mógł korzystać z tego połączonego konta i wszystkie zależne funkcje przestaną działać.",
				successMessage: "{{connectedAccount}} zostało usunięte z Twojego konta.",
				title: "Usuń połączone konto"
			},
			socialButtonsBlockButton: "Połącz konto {{provider|titleize}}",
			successMessage: "Dostawca został dodany do Twojego konta.",
			title: "Dodaj połączone konto"
		},
		deletePage: {
			actionDescription: "Wpisz \"Usuń konto\" poniżej aby kontynuować.",
			confirm: "Usuń konto",
			messageLine1: "Czy na pewno chcesz usunąć swoje konto? Niektóre powiązane dane mogą zostać zachowane. Aby poprosić o całkowite usunięcie danych, skontaktuj się z pomocą techniczną.",
			messageLine2: "Ta operacja jest nieodwracalna i nie można jej cofnąć.",
			title: "Usuń konto"
		},
		emailAddressPage: {
			emailCode: {
				formHint: "E-mail zawierający kod weryfikacyjny zostanie wysłany na ten adres e-mail.",
				formSubtitle: "Wprowadź kod weryfikacyjny wysłany na adres {{identifier}}",
				formTitle: "Kod weryfikacyjny",
				resendButton: "Wyślij ponownie kod",
				successMessage: "Adres e-mail {{identifier}} został dodany do twojego konta."
			},
			emailLink: {
				formHint: "E-mail zawierający link weryfikacyjny zostanie wysłany na ten adres e-mail.",
				formSubtitle: "Kliknij w link weryfikacyjny w e-mailu wysłanym na adres {{identifier}}",
				formTitle: "Link weryfikacyjny",
				resendButton: "Wyślij ponownie link",
				successMessage: "Adres e-mail {{identifier}} został dodany do twojego konta."
			},
			enterpriseSSOLink: {
				formButton: "Kliknij, aby się zalogować",
				formSubtitle: "Ukończ logowanie za pomocą {{identifier}}"
			},
			formHint: "Przed dodaniem adresu e-mail do konta należy go zweryfikować.",
			removeResource: {
				messageLine1: "{{identifier}} zostanie usunięty z tego konta.",
				messageLine2: "Nie będzie już możliwe zalogowanie się za pomocą tego adresu e-mail.",
				successMessage: "{{emailAddress}} został usunięty z twojego konta.",
				title: "Usuń adres e-mail"
			},
			title: "Dodaj adres e-mail",
			verifyTitle: "Zweryfikuj adres e-mail"
		},
		formButtonPrimary__add: "Dodaj",
		formButtonPrimary__continue: "Kontynuuj",
		formButtonPrimary__finish: "Zakończ",
		formButtonPrimary__remove: "Usuń",
		formButtonPrimary__save: "Zapisz",
		formButtonReset: "Anuluj",
		mfaPage: {
			formHint: "Wybierz metodę dodania.",
			title: "Dodaj weryfikację dwuetapową"
		},
		mfaPhoneCodePage: {
			backButton: "Użyj istniejącego numeru",
			primaryButton__addPhoneNumber: "Dodaj numer telefonu",
			removeResource: {
				messageLine1: "{{identifier}} nie będzie już otrzymywał kodów weryfikacyjnych podczas logowania.",
				messageLine2: "Twoje konto może być mniej bezpieczne. Czy na pewno chcesz kontynuować?",
				successMessage: "Weryfikacja kodem SMS w dwustopniowym procesie uwierzytelniania została usunięta dla {{mfaPhoneCode}}",
				title: "Usuń dwustopniową weryfikację"
			},
			subtitle__availablePhoneNumbers: "Wybierz numer telefonu, aby zarejestrować weryfikację kodem SMS w dwustopniowym procesie uwierzytelniania.",
			subtitle__unavailablePhoneNumbers: "Brak dostępnych numerów telefonów do zarejestrowania weryfikacji kodem SMS w dwustopniowym procesie uwierzytelniania.",
			successMessage1: "Podczas logowania należy dodatkowo wprowadzić kod weryfikacyjny wysłany na ten numer telefonu.",
			successMessage2: "Zapisz te kody zapasowe i przechowuj je w bezpiecznym miejscu. Jeśli utracisz dostęp do urządzenia uwierzytelniającego, możesz użyć kodów zapasowych, aby się zalogować.",
			successTitle: "Weryfikacja kodem SMS włączona",
			title: "Dodaj weryfikację kodem SMS"
		},
		mfaTOTPPage: {
			authenticatorApp: {
				buttonAbleToScan__nonPrimary: "Zamiast tego zeskanuj kod QR",
				buttonUnableToScan__nonPrimary: "Nie można zeskanować kodu QR?",
				infoText__ableToScan: "Ustaw nową metodę logowania w swojej aplikacji autentykacyjnej i zeskanuj następujący kod QR, aby połączyć go z Twoim kontem.",
				infoText__unableToScan: "Ustaw nową metodę logowania w swojej aplikacji autentykacyjnej i wprowadź poniższy klucz.",
				inputLabel__unableToScan1: "Upewnij się, że włączona jest opcja jednorazowe hasła lub hasła oparte na czasie, a następnie zakończ łączenie konta.",
				inputLabel__unableToScan2: "Alternatywnie, jeśli Twoja aplikacja autentykacyjna obsługuje URI TOTP, możesz również skopiować pełny URI."
			},
			removeResource: {
				messageLine1: "Kody weryfikacyjne z tej aplikacji autentykacyjnej nie będą już wymagane podczas logowania.",
				messageLine2: "Twoje konto może być mniej bezpieczne. Czy na pewno chcesz kontynuować?",
				successMessage: "Weryfikacja dwuetapowa za pomocą aplikacji autentykacyjnej została usunięta.",
				title: "Usuń weryfikację dwuetapową"
			},
			successMessage: "Weryfikacja dwuetapowa jest teraz włączona. Przy logowaniu będziesz musiał wprowadzić kod weryfikacyjny z tej aplikacji jako dodatkowy krok.",
			title: "Dodaj aplikację autentykacyjną",
			verifySubtitle: "Wprowadź kod weryfikacyjny wygenerowany przez Twoją aplikację autentykacyjną",
			verifyTitle: "Kod weryfikacyjny"
		},
		mobileButton__menu: "Menu",
		navbar: {
			account: "Profil",
			apiKeys: void 0,
			billing: void 0,
			description: "Zarządzaj danymi konta.",
			security: "Bezpieczeństwo",
			title: "Konto"
		},
		passkeyScreen: {
			removeResource: {
				messageLine1: "{{name}} zostanie usunięty z tego konta.",
				title: "Usuń klucz dostępu"
			},
			subtitle__rename: "Możesz zmienić nazwę klucza dostępu aby go łatwiej znaleźć.",
			title__rename: "Zmień nazwę klucza dostępu"
		},
		passwordPage: {
			checkboxInfoText__signOutOfOtherSessions: "Zaleca się wylogowanie z innych urządzeń, które mogły używać starego hasła.",
			readonly: "Obecnie nie można edytować hasła, ponieważ możesz zalogować się tylko za pośrednictwem połączenia firmowego.",
			successMessage__set: "Twoje hasło zostało ustawione.",
			successMessage__signOutOfOtherSessions: "Wylogowano z wszystkich innych urządzeń.",
			successMessage__update: "Twoje hasło zostało zaktualizowane.",
			title__set: "Ustaw hasło",
			title__update: "Zmień hasło"
		},
		phoneNumberPage: {
			infoText: "Wiadomość tekstowa zawierająca link weryfikacyjny zostanie wysłana na ten numer telefonu.",
			removeResource: {
				messageLine1: "{{identifier}} zostanie usunięty z tego konta.",
				messageLine2: "Nie będzie już możliwe zalogowanie się za pomocą tego numeru telefonu.",
				successMessage: "{{phoneNumber}} został usunięty z twojego konta.",
				title: "Usuń numer telefonu"
			},
			successMessage: "{{identifier}} został dodany do twojego konta.",
			title: "Dodaj numer telefonu",
			verifySubtitle: "Wpisz kod weryfikacyjny wysłany na {{identifier}}",
			verifyTitle: "Zweryfikuj numer telefonu"
		},
		plansPage: { title: void 0 },
		profilePage: {
			fileDropAreaHint: "Prześlij zdjęcie w formacie JPG, PNG, GIF lub WEBP mniejsze niż 10 MB",
			imageFormDestructiveActionSubtitle: "Usuń zdjęcie",
			imageFormSubtitle: "Prześlij zdjęcie",
			imageFormTitle: "Zdjęcie profilowe",
			readonly: "Informacje o Twoim profilu zostały udostępnione przez połączenie firmowe i nie można ich edytować.",
			successMessage: "Twój profil został zaktualizowany.",
			title: "Edytuj profil"
		},
		start: {
			activeDevicesSection: {
				destructiveAction: "Wyloguj z urządzenia",
				title: "Aktywne urządzenia"
			},
			connectedAccountsSection: {
				actionLabel__connectionFailed: "Spróbuj ponownie",
				actionLabel__reauthorize: "Autoryzuj teraz",
				destructiveActionTitle: "Odłącz",
				primaryButton: "Połącz konto",
				subtitle__disconnected: "To konto zostało odłączone",
				subtitle__reauthorize: "Wymagane zakresy zostały zaktualizowane i funkcjonalność aplikacji może być ograniczona. Aby uniknąć problemów, należy ponownie autoryzować aplikację",
				title: "Połączone konta"
			},
			dangerSection: {
				deleteAccountButton: "Usuń konto",
				title: "Niebezpieczeństwo"
			},
			emailAddressesSection: {
				destructiveAction: "Usuń adres email",
				detailsAction__nonPrimary: "Ustaw jako główny",
				detailsAction__primary: "Zakończ weryfikację",
				detailsAction__unverified: "Zakończ weryfikację",
				primaryButton: "Dodaj adres email",
				title: "Adresy email"
			},
			enterpriseAccountsSection: {
				primaryButton: "Połącz konto",
				title: "Konta firmowe"
			},
			headerTitle__account: "Konto",
			headerTitle__security: "Bezpieczeństwo",
			mfaSection: {
				backupCodes: {
					actionLabel__regenerate: "Wygeneruj kody",
					headerTitle: "Kody zapasowe",
					subtitle__regenerate: "Otrzymaj nowy zestaw bezpiecznych kodów zapasowych. Poprzednie kody zapasowe zostaną usunięte i nie będą działać.",
					title__regenerate: "Wygeneruj nowe kody zapasowe"
				},
				phoneCode: {
					actionLabel__setDefault: "Ustaw jako domyślny",
					destructiveActionLabel: "Usuń numer telefonu"
				},
				primaryButton: "Dodaj weryfikację dwuetapową",
				title: "Weryfikacja dwuetapowa",
				totp: {
					destructiveActionTitle: "Usuń",
					headerTitle: "Aplikacja autoryzacyjna"
				}
			},
			passkeysSection: {
				menuAction__destructive: "Usuń",
				menuAction__rename: "Zmień nazwę",
				primaryButton: void 0,
				title: "Klucze dostępu"
			},
			passwordSection: {
				primaryButton__setPassword: "Ustaw hasło",
				primaryButton__updatePassword: "Zmień hasło",
				title: "Hasło"
			},
			phoneNumbersSection: {
				destructiveAction: "Usuń numer telefonu",
				detailsAction__nonPrimary: "Ustaw jako główny",
				detailsAction__primary: "Zakończ weryfikację",
				detailsAction__unverified: "Zakończ weryfikację",
				primaryButton: "Dodaj numer telefonu",
				title: "Numery telefonów"
			},
			profileSection: {
				primaryButton: "Zaaktualizuj profil",
				title: "Profil"
			},
			usernameSection: {
				primaryButton__setUsername: "Ustaw nazwę użytkownika",
				primaryButton__updateUsername: "Zmień nazwę użytkownika",
				title: "Nazwa użytkownika"
			},
			web3WalletsSection: {
				destructiveAction: "Usuń portfel",
				detailsAction__nonPrimary: void 0,
				primaryButton: "Portfele Web3",
				title: "Portfele Web3",
				web3SelectSolanaWalletScreen: {
					subtitle: "Wybierz portfel Solana, aby połączyć go z kontem.",
					title: "Dodaj portfel Solana"
				}
			}
		},
		usernamePage: {
			successMessage: "Twoja nazwa użytkownika została zaktualizowana.",
			title__set: "Zmień nazwę użytkownika",
			title__update: "Zmień nazwę użytkownika"
		},
		web3WalletPage: {
			removeResource: {
				messageLine1: "{{identifier}} zostanie usunięty z tego konta.",
				messageLine2: "Nie będziesz już mógł się zalogować za pomocą tego portfela web3.",
				successMessage: "{{web3Wallet}} został usunięty z Twojego konta.",
				title: "Usuń portfel web3"
			},
			subtitle__availableWallets: "Wybierz portfel web3 do połączenia z Twoim kontem.",
			subtitle__unavailableWallets: "Nie ma dostępnych portfeli web3.",
			successMessage: "Portfel został dodany do Twojego konta.",
			title: "Dodaj portfel web3",
			web3WalletButtonsBlockButton: void 0
		}
	},
	waitlist: {
		start: {
			actionLink: "Zaloguj się",
			actionText: "Masz już konto?",
			formButton: "Dołącz do listy oczekujących",
			subtitle: "Wpisz swój adres e-mail, a my powiadomimy Cię, gdy miejsce dla Ciebie będzie gotowe.",
			title: "Dołącz do listy oczekujących"
		},
		success: {
			message: "Wkrótce nastąpi przekierowanie...",
			subtitle: "Skontaktujemy się z Tobą, gdy miejsce dla Ciebie będzie gotowe",
			title: "Dziękujemy za dołączenie do listy oczekujących!"
		}
	},
	web3SolanaWalletButtons: {
		connect: "Połącz z {{walletName}}",
		continue: "Kontynuuj z {{walletName}}",
		noneAvailable: "Nie wykryto portfeli Solana Web3. Zainstaluj {{ solanaWalletsLink || link(\"wallet extension\") }} obsługujący Web3."
	}
};

//#endregion
exports.plPL = plPL;
