// @clerk/localizations/dist/sr-RS.js
Object.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });

//#region src/sr-RS.ts
const srRS = {
	locale: "sr-RS",
	apiKeys: {
		action__add: void 0,
		action__search: void 0,
		copySecret: {
			formButtonPrimary__copyAndClose: "Копирај и затвори",
			formHint: "Из безбедносних разлога, нећемо вам дозволити да га поново погледате касније.",
			formTitle: "Копирајте свој API кључ \"{{name}}\" сада"
		},
		createdAndExpirationStatus__expiresOn: void 0,
		createdAndExpirationStatus__never: void 0,
		detailsTitle__emptyRow: void 0,
		formButtonPrimary__add: void 0,
		formFieldCaption__expiration__expiresOn: void 0,
		formFieldCaption__expiration__never: void 0,
		formFieldOption__expiration__180d: void 0,
		formFieldOption__expiration__1d: void 0,
		formFieldOption__expiration__1y: void 0,
		formFieldOption__expiration__30d: void 0,
		formFieldOption__expiration__60d: void 0,
		formFieldOption__expiration__7d: void 0,
		formFieldOption__expiration__90d: void 0,
		formFieldOption__expiration__never: void 0,
		formHint: void 0,
		formTitle: void 0,
		lastUsed__days: void 0,
		lastUsed__hours: void 0,
		lastUsed__minutes: void 0,
		lastUsed__months: void 0,
		lastUsed__seconds: void 0,
		lastUsed__years: void 0,
		menuAction__revoke: void 0,
		revokeConfirmation: {
			confirmationText: void 0,
			formButtonPrimary__revoke: void 0,
			formHint: void 0,
			formTitle: void 0,
			inputLabel: void 0
		},
		tableHeader__actions: void 0,
		tableHeader__lastUsed: void 0,
		tableHeader__name: void 0
	},
	backButton: "Nazad",
	badge__activePlan: void 0,
	badge__banned: void 0,
	badge__canceledEndsAt: void 0,
	badge__currentPlan: void 0,
	badge__default: "Podrazumevano",
	badge__endsAt: void 0,
	badge__expired: void 0,
	badge__freeTrial: void 0,
	badge__otherImpersonatorDevice: "Drugi uređaj koji se predstavlja",
	badge__pastDueAt: void 0,
	badge__pastDuePlan: void 0,
	badge__primary: "Primarni",
	badge__renewsAt: void 0,
	badge__requiresAction: "Zahteva akciju",
	badge__startsAt: void 0,
	badge__thisDevice: "Ovaj uređaj",
	badge__trialEndsAt: void 0,
	badge__unverified: "Nepotvrđen",
	badge__upcomingPlan: void 0,
	badge__userDevice: "Korisnički uređaj",
	badge__you: "Vi",
	billing: {
		accountCredit: void 0,
		addPaymentMethod__label: void 0,
		alwaysFree: void 0,
		annually: void 0,
		availableFeatures: void 0,
		billedAnnually: void 0,
		billedAnnuallyOnly: void 0,
		billedMonthly: void 0,
		billedMonthlyOnly: void 0,
		cancelFreeTrial: void 0,
		cancelFreeTrialAccessUntil: void 0,
		cancelFreeTrialTitle: void 0,
		cancelSubscription: void 0,
		cancelSubscriptionAccessUntil: void 0,
		cancelSubscriptionNoCharge: void 0,
		cancelSubscriptionPastDue: void 0,
		cancelSubscriptionTitle: void 0,
		cannotSubscribeMonthly: void 0,
		cannotSubscribeUnrecoverable: void 0,
		checkout: {
			description__paymentSuccessful: void 0,
			description__subscriptionSuccessful: void 0,
			downgradeNotice: void 0,
			emailForm: {
				subtitle: void 0,
				title: void 0
			},
			lineItems: {
				title__freeTrialEndsAt: void 0,
				title__paymentMethod: void 0,
				title__statementId: void 0,
				title__subscriptionBegins: void 0,
				title__totalPaid: void 0
			},
			pastDueNotice: void 0,
			perMonth: void 0,
			title: void 0,
			title__paymentSuccessful: void 0,
			title__subscriptionSuccessful: void 0,
			title__trialSuccess: void 0,
			totalDueAfterTrial: void 0,
			totalDuePerPeriod: void 0
		},
		credit: void 0,
		creditRemainder: void 0,
		defaultFreePlanActive: void 0,
		free: void 0,
		getStarted: void 0,
		highlightedPlanBadge: void 0,
		keepFreeTrial: void 0,
		keepSubscription: void 0,
		manage: void 0,
		manageSubscription: void 0,
		month: void 0,
		monthAbbreviation: void 0,
		monthPerUnit: void 0,
		monthly: void 0,
		pastDue: void 0,
		pay: void 0,
		payerCreditRemainder: void 0,
		paymentMethod: {
			applePayDescription: {
				annual: void 0,
				monthly: void 0
			},
			dev: {
				anyNumbers: void 0,
				cardNumber: void 0,
				cvcZip: void 0,
				developmentMode: void 0,
				expirationDate: void 0,
				testCardInfo: void 0
			}
		},
		paymentMethods__label: void 0,
		pricingTable: {
			billingCycle: void 0,
			included: void 0,
			seatCost: {
				additionalSeats: void 0,
				freeUpToSeats: void 0,
				includedSeats: void 0,
				perSeat: void 0,
				tooltip: {
					additionalSeatsEach: void 0,
					firstSeatsIncludedInPlan: void 0,
					freeForUpToSeats: void 0
				},
				unlimitedSeats: void 0,
				upToSeats: void 0
			}
		},
		proratedDiscount: void 0,
		prorationCredit: void 0,
		reSubscribe: void 0,
		seatBreakdownIncludedPlural: void 0,
		seatBreakdownIncludedSingular: void 0,
		seatBreakdownPlural: void 0,
		seatBreakdownSingular: void 0,
		seats: void 0,
		seatsWithLimit: void 0,
		seeAllFeatures: void 0,
		startFreeTrial: void 0,
		startFreeTrial__days: void 0,
		subscribe: void 0,
		subscriptionDetails: {
			beginsOn: void 0,
			currentBillingCycle: void 0,
			endsOn: void 0,
			firstPaymentAmount: void 0,
			firstPaymentOn: void 0,
			nextPaymentAmount: void 0,
			nextPaymentOn: void 0,
			pastDueAt: void 0,
			renewsAt: void 0,
			subscribedOn: void 0,
			title: void 0,
			trialEndsOn: void 0,
			trialStartedOn: void 0
		},
		subtotal: void 0,
		subtotalRenewal: void 0,
		switchPlan: void 0,
		switchToAnnual: void 0,
		switchToAnnualWithAnnualPrice: void 0,
		switchToMonthly: void 0,
		switchToMonthlyWithPrice: void 0,
		totalDue: void 0,
		totalDuePerPeriod: void 0,
		totalDueToday: void 0,
		viewFeatures: void 0,
		viewPayment: void 0,
		year: void 0,
		yearAbbreviation: void 0,
		yearPerUnit: void 0
	},
	configureSSO: {
		activate: {
			activateButton: void 0,
			activeSubtitle: void 0,
			activeTitle: void 0,
			doneButton: void 0,
			skipButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		changeProviderDialog: {
			cancelButton: void 0,
			confirmButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		configureStep: {
			activeConnectionWarning: {
				dismiss: void 0,
				title: void 0
			},
			attributeMappingTable: { badges: {
				optional: void 0,
				required: void 0
			} },
			samlCustom: {
				assignUsersStep: {
					headerSubtitle: void 0,
					paragraph: void 0
				},
				attributeMappingStep: {
					attributeMappingTable: {
						columns: {
							attributeName: void 0,
							userAttribute: void 0
						},
						rows: {
							email: {
								attributeName: void 0,
								userAttribute: void 0
							},
							firstName: {
								attributeName: void 0,
								userAttribute: void 0
							},
							lastName: {
								attributeName: void 0,
								userAttribute: void 0
							}
						}
					},
					headerSubtitle: void 0,
					paragraph: void 0
				},
				createAppStep: {
					createAppInstructions: { paragraph: void 0 },
					headerSubtitle: void 0,
					serviceProviderFields: {
						acsUrl: { label: void 0 },
						spEntityId: { label: void 0 }
					}
				},
				identityProviderMetadataStep: {
					headerSubtitle: void 0,
					manual: {
						description: void 0,
						issuer: {
							label: void 0,
							placeholder: void 0
						},
						signOnUrl: {
							label: void 0,
							placeholder: void 0
						},
						signingCertificate: {
							fileUploaded: void 0,
							label: void 0,
							removeFile: void 0,
							replaceFile: void 0,
							uploadFile: void 0
						}
					},
					metadataUrl: {
						description: void 0,
						label: void 0,
						placeholder: void 0
					},
					modes: {
						ariaLabel: void 0,
						manual: void 0,
						metadataUrl: void 0
					}
				},
				mainHeaderTitle: void 0
			},
			samlGoogle: {
				attributeMappingStep: {
					attributeMappingTable: {
						columns: {
							appAttribute: void 0,
							googleAttribute: void 0
						},
						rows: {
							email: {
								appAttribute: void 0,
								googleAttribute: void 0
							},
							firstName: {
								appAttribute: void 0,
								googleAttribute: void 0
							},
							lastName: {
								appAttribute: void 0,
								googleAttribute: void 0
							}
						}
					},
					headerSubtitle: void 0,
					paragraph: void 0,
					step1: void 0,
					step2: void 0
				},
				configureUserAccess: {
					assignUsersInstructions: {
						paragraph1: void 0,
						paragraph2: void 0,
						step1: void 0,
						step2: void 0,
						step3: void 0
					},
					headerSubtitle: void 0
				},
				createAppStep: {
					createAppInstructions: {
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: void 0,
						title: void 0
					},
					headerSubtitle: void 0
				},
				identityProviderMetadataStep: {
					headerSubtitle: void 0,
					manual: {
						description: void 0,
						issuer: {
							label: void 0,
							placeholder: void 0
						},
						signOnUrl: {
							label: void 0,
							placeholder: void 0
						},
						signingCertificate: {
							fileUploaded: void 0,
							label: void 0,
							removeFile: void 0,
							replaceFile: void 0,
							uploadFile: void 0
						}
					},
					metadataFile: {
						description: void 0,
						fileUploaded: void 0,
						label: void 0,
						removeFile: void 0,
						replaceFile: void 0,
						uploadFile: void 0
					},
					modes: {
						ariaLabel: void 0,
						manual: void 0,
						metadataFile: void 0
					}
				},
				mainHeaderTitle: void 0,
				serviceProviderStep: {
					headerSubtitle: void 0,
					nameIdInstructions: {
						step1: void 0,
						step2: void 0
					},
					paragraph: void 0,
					serviceProviderFields: {
						acsUrl: { label: void 0 },
						spEntityId: { label: void 0 }
					},
					title: void 0
				}
			},
			samlMicrosoft: {
				attributeMappingStep: {
					attributeMappingTable: {
						columns: {
							attribute: void 0,
							claimName: void 0,
							value: void 0
						},
						copyClaimName: void 0,
						copyClaimNameCopied: void 0,
						rows: {
							email: {
								attribute: void 0,
								claimName: void 0,
								value: void 0
							},
							firstName: {
								attribute: void 0,
								claimName: void 0,
								value: void 0
							},
							lastName: {
								attribute: void 0,
								claimName: void 0,
								value: void 0
							}
						}
					},
					headerSubtitle: void 0,
					step1: void 0,
					step2: void 0,
					title: void 0
				},
				createAppStep: {
					assignUsersInstructions: {
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: void 0,
						step5: void 0,
						title: void 0
					},
					createAppInstructions: {
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: {
							label: void 0,
							subSteps: {
								appName: void 0,
								create: void 0,
								nonGallery: void 0
							}
						},
						title: void 0
					},
					headerSubtitle: void 0
				},
				identityProviderMetadataStep: {
					headerSubtitle: void 0,
					manual: {
						description: void 0,
						issuer: {
							label: void 0,
							placeholder: void 0
						},
						signOnUrl: {
							label: void 0,
							placeholder: void 0
						},
						signingCertificate: {
							fileUploaded: void 0,
							label: void 0,
							removeFile: void 0,
							replaceFile: void 0,
							uploadFile: void 0
						}
					},
					metadataUrl: {
						description: void 0,
						label: void 0,
						placeholder: void 0
					},
					modes: {
						ariaLabel: void 0,
						manual: void 0,
						metadataUrl: void 0
					}
				},
				mainHeaderTitle: void 0,
				serviceProviderStep: {
					headerSubtitle: void 0,
					serviceProviderFields: {
						acsUrl: { label: void 0 },
						spEntityId: { label: void 0 }
					},
					step1: void 0,
					step2: void 0,
					step3: void 0,
					step4: void 0,
					step5: void 0,
					step6: void 0,
					title: void 0
				}
			},
			samlOkta: {
				assignUsersStep: {
					assignUsersInstructions: {
						paragraph: void 0,
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: void 0,
						step5: void 0
					},
					headerSubtitle: void 0
				},
				attributeMappingStep: {
					attributeMappingTable: {
						columns: {
							expression: void 0,
							name: void 0
						},
						rows: {
							email: {
								expression: void 0,
								name: void 0
							},
							firstName: {
								expression: void 0,
								name: void 0
							},
							lastName: {
								expression: void 0,
								name: void 0
							}
						}
					},
					headerSubtitle: void 0,
					paragraph: void 0,
					step1: void 0,
					step2: void 0
				},
				createAppStep: {
					completeSamlIntegrationInstructions: {
						step1: void 0,
						step2: void 0,
						title: void 0
					},
					createAppInstructions: {
						step1: void 0,
						step2: void 0,
						step3: void 0,
						step4: void 0,
						title: void 0
					},
					headerSubtitle: void 0,
					serviceProviderInstructions: {
						paragraph1: void 0,
						paragraph2: void 0,
						serviceProviderFields: {
							acsUrl: { label: void 0 },
							spEntityId: { label: void 0 }
						},
						title: void 0
					}
				},
				identityProviderMetadataStep: {
					headerSubtitle: void 0,
					manual: {
						description: void 0,
						issuer: {
							label: void 0,
							placeholder: void 0
						},
						signOnUrl: {
							label: void 0,
							placeholder: void 0
						},
						signingCertificate: {
							fileUploaded: void 0,
							label: void 0,
							removeFile: void 0,
							replaceFile: void 0,
							uploadFile: void 0
						}
					},
					metadataUrl: {
						description: void 0,
						label: void 0,
						placeholder: void 0
					},
					modes: {
						ariaLabel: void 0,
						manual: void 0,
						metadataUrl: void 0
					}
				},
				mainHeaderTitle: void 0
			}
		},
		missingManageEnterpriseConnectionsPermission: {
			subtitle: "Kontaktirajte administratora vaše organizacije da biste proširili svoje dozvole.",
			title: "Nemate dozvolu za upravljanje jedinstvenom prijavom (SSO)"
		},
		navbar: { title: "Konfiguriši jedinstvenu prijavu (SSO)" },
		organizationDomainsStep: {
			domainCard: {
				badge__expired: void 0,
				badge__unverified: "Nepotvrđeno",
				badge__verified: "Potvrđeno",
				expiredAtLabel: void 0,
				expiredLabel: void 0,
				removeButtonTooltip__lastVerifiedDomain: void 0,
				removeButtonTooltip__lastVerifiedDomainActive: void 0,
				txtRecord: {
					hostLabel: "Host / Naziv",
					instructions: "Dodajte ovaj TXT zapis kod svog DNS provajdera. Automatski ćemo ga potvrditi čim zapis postane aktivan.",
					typeLabel: "Tip",
					valueLabel: "Vrednost"
				},
				verifiedAtLabel: "Potvrđeno {{ date | shortDate('sr-RS') }}",
				verifyAgainButton: void 0
			},
			domainSuggestion: {
				formButtonPrimary__add: "Dodaj {{domain}}",
				messageLabel: "Vaša e-pošta koristi {{domain}}. Želite li da ga dodate?"
			},
			formButtonPrimary__add: "Dodaj",
			formFieldInputPlaceholder__domain: "Dodaj domen",
			formFieldLabel__domain: "Domen",
			removeDomainDialog: {
				cancelButton: void 0,
				removeButton: void 0,
				subtitle__active: void 0,
				subtitle__inactive: void 0,
				title: void 0
			},
			subtitle: "Dodajte i potvrdite vlasništvo nad domenima koje vaša organizacija koristi za prijavu.",
			title: "Dodaj SSO domene"
		},
		resetConnectionDialog: {
			cancelButton: void 0,
			confirmationFieldLabel: void 0,
			confirmationFieldPlaceholder: void 0,
			resetButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		selectProviderStep: {
			saml: {
				customSaml: "Prilagođeni SAML provajder",
				google: void 0,
				groupLabel: "SAML",
				microsoft: void 0,
				okta: "Okta Workforce"
			},
			subtitle: "Izaberite provajdera za koga ćete podesiti SSO.",
			title: "Izaberite provajdera",
			warning: "Kada se provajder izabere, ne možete ga ponovo menjati dok se konfiguracija ne završi"
		},
		testConfigurationStep: {
			error__noSuccessfulTestRun: void 0,
			subtitle: void 0,
			testResults: {
				actionLabel__refresh: void 0,
				empty: {
					subtitle: void 0,
					title: void 0
				},
				polling: void 0,
				status__failed: void 0,
				status__pending: void 0,
				status__success: void 0,
				title: void 0
			},
			testRunDetails: {
				howToFix: {
					actionLabel__viewDocumentation: void 0,
					oauth_access_denied: { description: void 0 },
					oauth_fetch_user_error: {
						intro: void 0,
						step1: void 0,
						step2: void 0
					},
					oauth_token_exchange_error: { description: void 0 },
					saml_email_address_domain_mismatch: { description: void 0 },
					saml_response_relaystate_missing: { description: void 0 },
					saml_user_attribute_missing: {
						intro: void 0,
						step1: void 0,
						step2: void 0,
						step3: void 0
					},
					sectionTitle: void 0
				},
				parsedUserInfo: {
					email: void 0,
					firstName: void 0,
					sectionTitle: void 0
				},
				runDetails: {
					actionLabel__copied: void 0,
					actionLabel__copy: void 0,
					errorCode: void 0,
					fullMessage: void 0,
					sectionTitle: void 0,
					status: void 0,
					timestamp: void 0
				},
				title: void 0
			},
			testUrl: { actionLabel__open: void 0 },
			title: void 0
		}
	},
	createOrganization: {
		formButtonSubmit: "Kreiraj organizaciju",
		invitePage: { formButtonReset: "Preskoči" },
		title: "Kreiraj organizaciju"
	},
	dates: {
		lastDay: "Juče u {{ date | timeString('sr-RS') }}",
		next6Days: "{{ date | weekday('sr-RS','long') }} u {{ date | timeString('sr-RS') }}",
		nextDay: "Sutra u {{ date | timeString('sr-RS') }}",
		numeric: "{{ date | numeric('sr-RS') }}",
		previous6Days: "Prošli {{ date | weekday('sr-RS','long') }} u {{ date | timeString('sr-RS') }}",
		sameDay: "Danas u {{ date | timeString('sr-RS') }}"
	},
	dividerText: "ili",
	footerActionLink__alternativePhoneCodeProvider: void 0,
	footerActionLink__useAnotherMethod: "Koristi drugu metodu",
	footerPageLink__help: "Pomoć",
	footerPageLink__privacy: "Privatnost",
	footerPageLink__terms: "Uslovi",
	formButtonPrimary: "Nastavi",
	formButtonPrimary__verify: "Verifikuj",
	formFieldAction__forgotPassword: "Zaboravljena lozinka?",
	formFieldError__matchingPasswords: "Lozinke se poklapaju.",
	formFieldError__notMatchingPasswords: "Lozinke se ne poklapaju.",
	formFieldError__verificationLinkExpired: "Link za verifikaciju je istekao. Molimo zatražite novi link.",
	formFieldHintText__optional: "Opciono",
	formFieldHintText__slug: "Slug je lako čitljivi ID koji mora biti jedinstven. Često se koristi u URL-ovima.",
	formFieldInputPlaceholder__apiKeyDescription: void 0,
	formFieldInputPlaceholder__apiKeyExpirationDate: void 0,
	formFieldInputPlaceholder__apiKeyName: void 0,
	formFieldInputPlaceholder__backupCode: void 0,
	formFieldInputPlaceholder__confirmDeletionUserAccount: "Obriši nalog",
	formFieldInputPlaceholder__emailAddress: void 0,
	formFieldInputPlaceholder__emailAddress_username: void 0,
	formFieldInputPlaceholder__emailAddresses: "primer@email.com, primer2@email.com",
	formFieldInputPlaceholder__firstName: void 0,
	formFieldInputPlaceholder__lastName: void 0,
	formFieldInputPlaceholder__organizationDomain: void 0,
	formFieldInputPlaceholder__organizationDomainEmailAddress: void 0,
	formFieldInputPlaceholder__organizationName: void 0,
	formFieldInputPlaceholder__organizationSlug: "moja-org",
	formFieldInputPlaceholder__password: void 0,
	formFieldInputPlaceholder__phoneNumber: void 0,
	formFieldInputPlaceholder__signUpPassword: void 0,
	formFieldInputPlaceholder__username: void 0,
	formFieldInput__emailAddress_format: void 0,
	formFieldLabel__apiKey: "API кључ",
	formFieldLabel__apiKeyDescription: "Опис",
	formFieldLabel__apiKeyExpiration: "Истиче",
	formFieldLabel__apiKeyName: "Назив тајног кључа",
	formFieldLabel__automaticInvitations: "Omogući automatske pozivnice za ovaj domen",
	formFieldLabel__backupCode: "Rezervni kod",
	formFieldLabel__confirmDeletion: "Potvrda",
	formFieldLabel__confirmPassword: "Potvrdi lozinku",
	formFieldLabel__currentPassword: "Trenutna lozinka",
	formFieldLabel__emailAddress: "E-mail adresa",
	formFieldLabel__emailAddress_username: "E-mail adresa ili korisničko ime",
	formFieldLabel__emailAddresses: "E-mail adrese",
	formFieldLabel__firstName: "Ime",
	formFieldLabel__lastName: "Prezime",
	formFieldLabel__newPassword: "Nova lozinka",
	formFieldLabel__organizationDomain: "Domen",
	formFieldLabel__organizationDomainDeletePending: "Obriši čekajuće pozivnice i predloge",
	formFieldLabel__organizationDomainEmailAddress: "E-mail adresa za verifikaciju",
	formFieldLabel__organizationDomainEmailAddressDescription: "Unesite e-mail adresu pod ovim domenom da biste primili kod i verifikovali ovaj domen.",
	formFieldLabel__organizationName: "Naziv",
	formFieldLabel__organizationSlug: "Slug",
	formFieldLabel__passkeyName: "Naziv ključa za prolaz",
	formFieldLabel__password: "Lozinka",
	formFieldLabel__phoneNumber: "Telefonski broj",
	formFieldLabel__role: "Uloga",
	formFieldLabel__signOutOfOtherSessions: "Odjavi se sa svih drugih uređaja",
	formFieldLabel__username: "Korisničko ime",
	identityPreviewEditButton__emailAddress: void 0,
	identityPreviewEditButton__identifier: void 0,
	identityPreviewEditButton__phoneNumber: void 0,
	impersonationFab: {
		action__signOut: "Odjavi se",
		title: "Prijavljeni ste kao {{identifier}}"
	},
	lastAuthenticationStrategy: "Poslednje korišćeno",
	maintenanceMode: "Trenutno smo u modu održavanja, ali ne brinite, neće trajati duže od nekoliko minuta.",
	membershipRole__admin: "Administrator",
	membershipRole__basicMember: "Član",
	membershipRole__guestMember: "Gost",
	oauthConsent: {
		action__allow: void 0,
		action__deny: void 0,
		offlineAccessNotice: void 0,
		redirectNotice: void 0,
		redirectUriModal: {
			subtitle: void 0,
			title: void 0
		},
		scopeList: { title: void 0 },
		subtitle: void 0,
		viewFullUrl: void 0,
		warning: void 0
	},
	organizationList: {
		action__createOrganization: "Kreiraj organizaciju",
		action__invitationAccept: "Pridruži se",
		action__suggestionsAccept: "Zatraži pridruživanje",
		createOrganization: "Kreiraj organizaciju",
		invitationAcceptedLabel: "Pridružen",
		subtitle: "da nastavite na {{applicationName}}",
		suggestionsAcceptedLabel: "Čeka odobrenje",
		title: "Izaberi nalog",
		titleWithoutPersonal: "Izaberi organizaciju"
	},
	organizationProfile: {
		apiKeysPage: { title: void 0 },
		badge__automaticInvitation: "Automatske pozivnice",
		badge__automaticSuggestion: "Automatski predlozi",
		badge__enterpriseSso: void 0,
		badge__manualInvitation: "Bez automatskog uključivanja",
		badge__unverified: "Nepotvrđen",
		billingPage: {
			accountCreditsSection: {
				title: void 0,
				viewHistory: void 0
			},
			creditHistoryPage: {
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				title: void 0
			},
			paymentHistorySection: {
				empty: void 0,
				notFound: void 0,
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				tableHeader__status: void 0
			},
			paymentMethodsSection: {
				actionLabel__default: void 0,
				actionLabel__remove: void 0,
				add: void 0,
				addSubtitle: void 0,
				cancelButton: void 0,
				formButtonPrimary__add: void 0,
				formButtonPrimary__pay: void 0,
				payWithTestCardButton: void 0,
				removeMethod: {
					messageLine1: void 0,
					messageLine2: void 0,
					successMessage: void 0,
					title: void 0
				},
				title: void 0
			},
			start: {
				headerTitle__payments: void 0,
				headerTitle__plans: void 0,
				headerTitle__statements: void 0,
				headerTitle__subscriptions: void 0
			},
			statementsSection: {
				empty: void 0,
				itemCaption__paidForPlan: void 0,
				itemCaption__payerCredit: void 0,
				itemCaption__proratedCredit: void 0,
				itemCaption__subscribedAndPaidForPlan: void 0,
				notFound: void 0,
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				title: void 0,
				totalPaid: void 0
			},
			subscriptionsListSection: {
				actionLabel__manageSubscription: void 0,
				actionLabel__newSubscription: void 0,
				actionLabel__switchPlan: void 0,
				includedSeatsUsage: void 0,
				overview: void 0,
				paidSeatsUsage: void 0,
				seatLimit: void 0,
				seatLimitAndIncludedSeats: void 0,
				tableHeader__edit: void 0,
				tableHeader__plan: void 0,
				tableHeader__startDate: void 0,
				title: void 0
			},
			subscriptionsSection: { actionLabel__default: void 0 },
			switchPlansSection: { title: void 0 },
			title: void 0
		},
		createDomainPage: {
			subtitle: "Dodajte domen za verifikaciju. Korisnici sa e-mail adresama na ovom domenu mogu se automatski pridružiti organizaciji ili zatražiti pridruživanje.",
			title: "Dodaj domen"
		},
		invitePage: {
			detailsTitle__inviteFailed: "Pozivnice nisu poslate. Već postoje čekajuće pozivnice za sledeće e-mail adrese: {{email_addresses}}.",
			formButtonPrimary__continue: "Pošalji pozivnice",
			formButtonPrimary__purchaseSeats: void 0,
			selectDropdown__role: "Izaberi ulogu",
			subtitle: "Unesi ili nalepi jednu ili više e-mail adresa, razdvojene razmacima ili zarezima.",
			successMessage: "Pozivnice su uspešno poslate",
			title: "Pozovi nove članove"
		},
		membersPage: {
			action__invite: "Pozovi",
			action__search: void 0,
			activeMembersTab: {
				menuAction__remove: "Ukloni člana",
				tableHeader__actions: void 0,
				tableHeader__joined: "Pridružio se",
				tableHeader__role: "Uloga",
				tableHeader__user: "Korisnik"
			},
			alerts: { roleSetMigrationInProgress: {
				subtitle: "Ažuriramo dostupne uloge. Kada to bude završeno, moći ćete ponovo da ažurirate uloge.",
				title: "Uloge su privremeno zaključane"
			} },
			detailsTitle__emptyRow: "Nema članova za prikaz",
			invitationsTab: {
				autoInvitations: {
					headerSubtitle: "Pozovi korisnike povezivanjem e-mail domena sa vašom organizacijom. Svako ko se registruje sa odgovarajućim e-mail domenom može se pridružiti organizaciji u bilo koje vreme.",
					headerTitle: "Automatske pozivnice",
					primaryButton: "Upravljaj potvrđenim domenima"
				},
				table__emptyRow: "Nema pozivnica za prikaz"
			},
			invitedMembersTab: {
				menuAction__revoke: "Poništi pozivnicu",
				tableHeader__invited: "Pozvan"
			},
			requestsTab: {
				autoSuggestions: {
					headerSubtitle: "Korisnici koji se registruju sa odgovarajućim e-mail domenom, moći će da vide predlog da zatraže pridruživanje vašoj organizaciji.",
					headerTitle: "Automatski predlozi",
					primaryButton: "Upravljaj potvrđenim domenima"
				},
				menuAction__approve: "Odobri",
				menuAction__reject: "Odbaci",
				tableHeader__requested: "Zatražen pristup",
				table__emptyRow: "Nema zahteva za prikaz"
			},
			start: {
				headerTitle__invitations: "Pozivnice",
				headerTitle__members: "Članovi",
				headerTitle__requests: "Zahtevi"
			}
		},
		navbar: {
			apiKeys: void 0,
			billing: void 0,
			description: "Upravljaj svojom organizacijom.",
			general: "Opšte",
			members: "Članovi",
			security: void 0,
			title: "Organizacija"
		},
		plansPage: {
			alerts: {
				noPermissionsToManageBilling: void 0,
				planMembershipLimitExceeded: void 0
			},
			title: void 0
		},
		profilePage: {
			dangerSection: {
				deleteOrganization: {
					actionDescription: "Upiši \"{{organizationName}}\" ispod da nastaviš.",
					messageLine1: "Da li ste sigurni da želite da obrišete ovu organizaciju?",
					messageLine2: "Ova akcija je trajna i nepovratna.",
					successMessage: "Organizacija je obrisana.",
					title: "Obriši organizaciju"
				},
				leaveOrganization: {
					actionDescription: "Upiši \"{{organizationName}}\" ispod da nastaviš.",
					messageLine1: "Da li ste sigurni da želite da napustite ovu organizaciju? Izgubićete pristup ovoj organizaciji i njenim aplikacijama.",
					messageLine2: "Ova akcija je trajna i nepovratna.",
					successMessage: "Napustili ste organizaciju.",
					title: "Napusti organizaciju"
				},
				title: "Opasnost"
			},
			domainSection: {
				menuAction__manage: "Upravljaj",
				menuAction__remove: "Obriši",
				menuAction__verify: "Verifikuj",
				primaryButton: "Dodaj domen",
				subtitle: "Dozvoli korisnicima da se automatski pridruže organizaciji ili zatraže pridruživanje na osnovu potvrđenog e-mail domena.",
				title: "Potvrđeni domeni"
			},
			successMessage: "Organizacija je ažurirana.",
			title: "Ažuriraj profil"
		},
		removeDomainPage: {
			messageLine1: "E-mail domen {{domain}} će biti uklonjen.",
			messageLine2: "Korisnici više neće moći automatski da se pridruže organizaciji nakon ovoga.",
			successMessage: "{{domain}} je uklonjen.",
			title: "Ukloni domen"
		},
		securityPage: {
			removeDialog: {
				confirmButton: void 0,
				subtitle: void 0,
				title: void 0
			},
			ssoSection: {
				badge__active: void 0,
				badge__inProgress: void 0,
				badge__inactive: void 0,
				badge__unconfigured: void 0,
				descriptionLine1: void 0,
				domainLabel: void 0,
				menuAction__activate: void 0,
				menuAction__deactivate: void 0,
				menuAction__edit: void 0,
				menuAction__remove: void 0,
				primaryButton__continueConfiguration: void 0,
				primaryButton__startConfiguration: void 0,
				title: void 0,
				tooltip: void 0,
				tooltipLabel: void 0,
				tooltip__noRole: void 0
			},
			title: void 0
		},
		start: {
			headerTitle__general: "Opšte",
			headerTitle__members: "Članovi",
			membershipSeatUsageLabel: void 0,
			profileSection: {
				primaryButton: "Ažuriraj profil",
				title: "Profil organizacije",
				uploadAction__title: "Logo"
			}
		},
		verifiedDomainPage: {
			dangerTab: {
				calloutInfoLabel: "Uklanjanje ovog domena će uticati na pozvane korisnike.",
				removeDomainActionLabel__remove: "Ukloni domen",
				removeDomainSubtitle: "Ukloni ovaj domen iz tvojih potvrđenih domena",
				removeDomainTitle: "Ukloni domen"
			},
			enrollmentTab: {
				automaticInvitationOption__description: "Korisnici se automatski pozivaju da se pridruže organizaciji kada se registruju i mogu se pridružiti u bilo koje vreme.",
				automaticInvitationOption__label: "Automatske pozivnice",
				automaticSuggestionOption__description: "Korisnici dobijaju predlog da zatraže pridruživanje, ali moraju biti odobreni od strane administratora pre nego što mogu da se pridruže organizaciji.",
				automaticSuggestionOption__label: "Automatski predlozi",
				calloutInfoLabel: "Promena načina upisa će uticati samo na nove korisnike.",
				calloutInvitationCountLabel: "Čekajuće pozivnice poslate korisnicima: {{count}}",
				calloutSuggestionCountLabel: "Čekajući predlozi poslati korisnicima: {{count}}",
				manualInvitationOption__description: "Korisnici mogu biti pozvani samo ručno u organizaciju.",
				manualInvitationOption__label: "Bez automatskog uključivanja",
				subtitle: "Izaberi kako korisnici iz ovog domena mogu da se pridruže organizaciji."
			},
			start: {
				headerTitle__danger: "Opasnost",
				headerTitle__enrollment: "Opcije upisa"
			},
			subtitle: "Domen {{domain}} je sada verifikovan. Nastavi biranjem načina upisa.",
			title: "Ažuriraj {{domain}}"
		},
		verifyDomainPage: {
			formSubtitle: "Unesi verifikacioni kod poslat na tvoju e-mail adresu",
			formTitle: "Verifikacioni kod",
			resendButton: "Nisi primio kod? Pošalji ponovo",
			subtitle: "Domen {{domainName}} mora biti verifikovan putem e-maila.",
			subtitleVerificationCodeScreen: "Verifikacioni kod je poslat na {{emailAddress}}. Unesi kod da nastaviš.",
			title: "Verifikuj domen"
		}
	},
	organizationSwitcher: {
		action__closeOrganizationSwitcher: void 0,
		action__createOrganization: "Kreiraj organizaciju",
		action__invitationAccept: "Pridruži se",
		action__manageOrganization: "Upravljaj",
		action__openOrganizationSwitcher: void 0,
		action__suggestionsAccept: "Zatraži pridruživanje",
		notSelected: "Organizacija nije izabrana",
		personalWorkspace: "Lični nalog",
		suggestionsAcceptedLabel: "Čeka odobrenje"
	},
	paginationButton__next: "Sledeći",
	paginationButton__previous: "Prethodni",
	paginationRowText__displaying: "Prikazujem",
	paginationRowText__of: "od",
	reverification: {
		alternativeMethods: {
			actionLink: void 0,
			actionText: void 0,
			blockButton__backupCode: void 0,
			blockButton__emailCode: void 0,
			blockButton__passkey: void 0,
			blockButton__password: void 0,
			blockButton__phoneCode: void 0,
			blockButton__totp: void 0,
			getHelp: {
				blockButton__emailSupport: void 0,
				content: void 0,
				title: void 0
			},
			subtitle: void 0,
			title: void 0
		},
		backupCodeMfa: {
			subtitle: void 0,
			title: void 0
		},
		emailCode: {
			formTitle: void 0,
			resendButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		noAvailableMethods: {
			message: void 0,
			subtitle: void 0,
			title: void 0
		},
		passkey: {
			blockButton__passkey: void 0,
			subtitle: void 0,
			title: void 0
		},
		password: {
			actionLink: void 0,
			subtitle: void 0,
			title: void 0
		},
		phoneCode: {
			formTitle: void 0,
			resendButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		phoneCodeMfa: {
			formTitle: void 0,
			resendButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		totpMfa: {
			formTitle: void 0,
			subtitle: void 0,
			title: void 0
		}
	},
	searchInput: { action__clear: void 0 },
	signIn: {
		accountSwitcher: {
			action__addAccount: "Dodaj nalog",
			action__signOutAll: "Odjavi se sa svih naloga",
			subtitle: "Izaberi nalog s kojim želiš da nastaviš.",
			title: "Izaberi nalog"
		},
		alternativeMethods: {
			actionLink: "Zatraži pomoć",
			actionText: "Nemaš ni jednu od ovih opcija?",
			blockButton__backupCode: "Koristi rezervni kod",
			blockButton__emailCode: "Pošalji kod na e-mail {{identifier}}",
			blockButton__emailLink: "Pošalji link na e-mail {{identifier}}",
			blockButton__passkey: "Prijavi se sa svojim ključem za prolaz",
			blockButton__password: "Prijavi se sa svojom lozinkom",
			blockButton__phoneCode: "Pošalji SMS kod na {{identifier}}",
			blockButton__totp: "Koristi svoju aplikaciju za autentifikaciju",
			getHelp: {
				blockButton__emailSupport: "Pošalji e-mail podršci",
				content: "Ako imaš problema sa prijavljivanjem na svoj nalog, pošalji nam e-mail i pomoći ćemo ti da što pre povratiš pristup.",
				title: "Zatraži pomoć"
			},
			subtitle: "Imaš problema? Možeš koristiti bilo koju od ovih metoda za prijavljivanje.",
			title: "Koristi drugu metodu"
		},
		alternativePhoneCodeProvider: {
			formTitle: void 0,
			resendButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		backupCodeMfa: {
			subtitle: "Tvoj rezervni kod je onaj koji si dobio kada si postavio dvostepenu autentifikaciju.",
			title: "Unesi rezervni kod"
		},
		emailCode: {
			formTitle: "Verifikacioni kod",
			resendButton: "Nisi primio kod? Pošalji ponovo",
			subtitle: "da nastaviš na {{applicationName}}",
			title: "Proveri svoj e-mail"
		},
		emailCodeMfa: {
			formTitle: "Proveri svoj e-mail",
			resendButton: "Nisi primio kod? Pošalji ponovo",
			subtitle: "da nastaviš na {{applicationName}}",
			title: "Proveri svoj e-mail"
		},
		emailLink: {
			clientMismatch: {
				subtitle: void 0,
				title: void 0
			},
			expired: {
				subtitle: "Vrati se na originalni tab da nastaviš.",
				title: "Ovaj verifikacioni link je istekao"
			},
			failed: {
				subtitle: "Vrati se na originalni tab da nastaviš.",
				title: "Ovaj verifikacioni link je nevažeći"
			},
			formSubtitle: "Koristi verifikacioni link poslat na tvoj e-mail",
			formTitle: "Verifikacioni link",
			loading: {
				subtitle: "Uskoro ćeš biti preusmeren",
				title: "Prijavljujem se..."
			},
			resendButton: "Nisi primio link? Pošalji ponovo",
			subtitle: "da nastaviš na {{applicationName}}",
			title: "Proveri svoj e-mail",
			unusedTab: { title: "Možeš zatvoriti ovaj tab" },
			verified: {
				subtitle: "Uskoro ćeš biti preusmeren",
				title: "Uspešno si prijavljen"
			},
			verifiedSwitchTab: {
				subtitle: "Vrati se na originalni tab da nastaviš",
				subtitleNewTab: "Vrati se na novootvoreni tab da nastaviš",
				titleNewTab: "Prijavljen na drugom tabu"
			}
		},
		emailLinkMfa: {
			formSubtitle: "Користите везу за верификацију послату на вашу е-пошту",
			resendButton: "Нисте примили везу? Пошаљи поново",
			subtitle: "да наставите на {{applicationName}}",
			title: "Проверите своју е-пошту"
		},
		enterpriseConnections: {
			subtitle: void 0,
			title: void 0
		},
		forgotPassword: {
			formTitle: "Kod za resetovanje lozinke",
			resendButton: "Nisi primio kod? Pošalji ponovo",
			subtitle: "da resetuješ svoju lozinku",
			subtitle_email: "Prvo, unesi kod poslat na tvoju e-mail adresu",
			subtitle_phone: "Prvo, unesi kod poslat na tvoj telefon",
			title: "Resetuj lozinku"
		},
		forgotPasswordAlternativeMethods: {
			blockButton__resetPassword: "Resetuj svoju lozinku",
			label__alternativeMethods: "Ili, prijavi se drugom metodom",
			title: "Zaboravljena lozinka?"
		},
		newDeviceVerificationNotice: "Prijavljuješ se sa novog uređaja. Tražimo verifikaciju kako bismo zadržali tvoj nalog sigurnim.",
		noAvailableMethods: {
			message: "Nije moguće nastaviti sa prijavom. Nema dostupnih metoda autentifikacije.",
			subtitle: "Došlo je do greške",
			title: "Nije moguće prijaviti se"
		},
		passkey: {
			subtitle: "Korišćenje tvojeg ključa za prolaz potvrđuje da si to ti. Tvoj uređaj može zatražiti otisak prsta, lice ili ekran zaključavanja.",
			title: "Koristi svoj ključ za prolaz"
		},
		password: {
			actionLink: "Koristi drugu metodu",
			subtitle: "Unesi lozinku koja je povezana sa tvojim nalogom",
			title: "Unesi svoju lozinku"
		},
		passwordCompromised: { title: void 0 },
		passwordPwned: { title: "Lozinka kompromitovana" },
		passwordUntrusted: { title: void 0 },
		phoneCode: {
			formTitle: "Verifikacioni kod",
			resendButton: "Nisi primio kod? Pošalji ponovo",
			subtitle: "da nastaviš na {{applicationName}}",
			title: "Proveri svoj telefon"
		},
		phoneCodeMfa: {
			formTitle: "Verifikacioni kod",
			resendButton: "Nisi primio kod? Pošalji ponovo",
			subtitle: "Da nastaviš, molimo unesi verifikacioni kod poslat na tvoj telefon",
			title: "Proveri svoj telefon"
		},
		protectCheck: {
			loading: void 0,
			retryButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		resetPassword: {
			formButtonPrimary: "Resetuj lozinku",
			requiredMessage: "Iz sigurnosnih razloga, potrebno je da resetuješ svoju lozinku.",
			successMessage: "Tvoja lozinka je uspešno promenjena. Prijavljujem te, sačekaj trenutak.",
			title: "Postavi novu lozinku"
		},
		resetPasswordMfa: { detailsLabel: "Potrebno je da potvrdimo tvoj identitet pre resetovanja lozinke." },
		start: {
			actionLink: "Registruj se",
			actionLink__join_waitlist: void 0,
			actionLink__use_email: "Koristi e-mail",
			actionLink__use_email_username: "Koristi e-mail ili korisničko ime",
			actionLink__use_passkey: "Koristi ključ za prolaz umesto toga",
			actionLink__use_phone: "Koristi telefon",
			actionLink__use_username: "Koristi korisničko ime",
			actionText: "Nemaš nalog?",
			actionText__join_waitlist: void 0,
			alternativePhoneCodeProvider: {
				actionLink: void 0,
				label: void 0,
				subtitle: void 0,
				title: void 0
			},
			subtitle: "Dobro došao nazad! Molimo prijavi se da nastaviš",
			subtitleCombined: void 0,
			title: "Prijavi se na {{applicationName}}",
			titleCombined: void 0
		},
		totpMfa: {
			formTitle: "Verifikacioni kod",
			subtitle: "Da nastaviš, molimo unesi verifikacioni kod generisan tvojom aplikacijom za autentifikaciju",
			title: "Dvostepena verifikacija"
		},
		web3Solana: {
			subtitle: "Izaberi novčanik ispod da se prijaviš",
			title: "Prijavi se sa Solana"
		}
	},
	signInEnterPasswordTitle: "Unesi svoju lozinku",
	signUp: {
		alternativePhoneCodeProvider: {
			resendButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		continue: {
			actionLink: "Prijavi se",
			actionText: "Već imaš nalog?",
			subtitle: "Molimo popuni preostale detalje da nastaviš.",
			title: "Popuni nedostajuća polja"
		},
		emailCode: {
			formSubtitle: "Unesi verifikacioni kod poslat na tvoju e-mail adresu",
			formTitle: "Verifikacioni kod",
			resendButton: "Nisi primio kod? Pošalji ponovo",
			subtitle: "Unesi verifikacioni kod poslat na tvoj e-mail",
			title: "Verifikuj svoj e-mail"
		},
		emailLink: {
			clientMismatch: {
				subtitle: void 0,
				title: void 0
			},
			formSubtitle: "Koristi verifikacioni link poslat na tvoju e-mail adresu",
			formTitle: "Verifikacioni link",
			loading: { title: "Registrujem se..." },
			resendButton: "Nisi primio link? Pošalji ponovo",
			subtitle: "da nastaviš na {{applicationName}}",
			title: "Verifikuj svoj e-mail",
			verified: { title: "Uspešno registrovan" },
			verifiedSwitchTab: {
				subtitle: "Vrati se na novootvoreni tab da nastaviš",
				subtitleNewTab: "Vrati se na prethodni tab da nastaviš",
				title: "Uspešno verifikovan e-mail"
			}
		},
		enterpriseConnections: {
			subtitle: void 0,
			title: void 0
		},
		legalConsent: {
			checkbox: {
				label__onlyPrivacyPolicy: void 0,
				label__onlyTermsOfService: void 0,
				label__termsOfServiceAndPrivacyPolicy: void 0
			},
			continue: {
				subtitle: void 0,
				title: void 0
			}
		},
		phoneCode: {
			formSubtitle: "Unesi verifikacioni kod poslat na tvoj telefonski broj",
			formTitle: "Verifikacioni kod",
			resendButton: "Nisi primio kod? Pošalji ponovo",
			subtitle: "Unesi verifikacioni kod poslat na tvoj telefon",
			title: "Verifikuj svoj telefon"
		},
		protectCheck: {
			loading: void 0,
			retryButton: void 0,
			subtitle: void 0,
			title: void 0
		},
		restrictedAccess: {
			actionLink: void 0,
			actionText: void 0,
			blockButton__emailSupport: void 0,
			blockButton__joinWaitlist: void 0,
			subtitle: void 0,
			subtitleWaitlist: void 0,
			title: void 0
		},
		start: {
			actionLink: "Prijavi se",
			actionLink__use_email: void 0,
			actionLink__use_phone: void 0,
			actionText: "Već imaš nalog?",
			alternativePhoneCodeProvider: {
				actionLink: void 0,
				label: void 0,
				subtitle: void 0,
				title: void 0
			},
			subtitle: "Dobrodošao! Molimo popuni detalje da započneš.",
			subtitleCombined: "Dobrodošao! Molimo popuni detalje da započneš.",
			title: "Kreiraj svoj nalog",
			titleCombined: "Kreiraj svoj nalog"
		},
		web3Solana: {
			subtitle: "Izaberi novčanik ispod da se registruješ",
			title: "Registruj se sa Solana"
		}
	},
	socialButtonsBlockButton: "Nastavi sa {{provider|titleize}}",
	socialButtonsBlockButtonManyInView: "{{provider|titleize}}",
	taskChooseOrganization: {
		alerts: { organizationAlreadyExists: "Organizacija već postoji za otkriveno ime kompanije ({{organizationName}}) i {{organizationDomain}}. Pridružite se putem pozivnice." },
		chooseOrganization: {
			action__createOrganization: "Napravi novu organizaciju",
			action__invitationAccept: "Pridruži se",
			action__suggestionsAccept: "Zatraži pridruživanje",
			subtitle: "Pridružite se postojećoj organizaciji ili napravite novu",
			subtitle__createOrganizationDisabled: "Pridružite se postojećoj organizaciji",
			suggestionsAcceptedLabel: "Čeka odobrenje",
			title: "Izaberite organizaciju"
		},
		createOrganization: {
			formButtonReset: "Otkaži",
			formButtonSubmit: "Nastavi",
			formFieldInputPlaceholder__name: "Moja organizacija",
			formFieldInputPlaceholder__slug: "moja-organizacija",
			formFieldLabel__name: "Naziv",
			formFieldLabel__slug: "Slug",
			subtitle: "Unesite podatke o svojoj organizaciji za nastavak",
			title: "Podesite svoju organizaciju"
		},
		organizationCreationDisabled: {
			subtitle: "Kontaktirajte administratora svoje organizacije za pozivnicu.",
			title: "Morate pripadati organizaciji"
		},
		signOut: {
			actionLink: "Odjavi se",
			actionText: "Prijavljen kao {{identifier}}"
		}
	},
	taskResetPassword: {
		formButtonPrimary: void 0,
		signOut: {
			actionLink: void 0,
			actionText: void 0
		},
		subtitle: void 0,
		title: void 0
	},
	taskSetupMfa: {
		badge: void 0,
		signOut: {
			actionLink: void 0,
			actionText: void 0
		},
		smsCode: {
			addPhone: {
				formButtonPrimary: void 0,
				infoText: void 0
			},
			addPhoneNumber: void 0,
			cancel: void 0,
			subtitle: void 0,
			success: {
				finishButton: void 0,
				message1: void 0,
				message2: void 0,
				title: void 0
			},
			title: void 0,
			verifyPhone: {
				formButtonPrimary: void 0,
				formTitle: void 0,
				resendButton: void 0,
				subtitle: void 0,
				title: void 0
			}
		},
		start: {
			methodSelection: {
				phoneCode: void 0,
				totp: void 0
			},
			subtitle: void 0,
			title: void 0
		},
		totpCode: {
			addAuthenticatorApp: {
				buttonAbleToScan__nonPrimary: void 0,
				buttonUnableToScan__nonPrimary: void 0,
				formButtonPrimary: void 0,
				formButtonReset: void 0,
				infoText__ableToScan: void 0,
				infoText__unableToScan: void 0,
				inputLabel__unableToScan1: void 0
			},
			success: {
				finishButton: void 0,
				message1: void 0,
				message2: void 0,
				title: void 0
			},
			title: void 0,
			verifyTotp: {
				formButtonPrimary: void 0,
				formButtonReset: void 0,
				formTitle: void 0,
				subtitle: void 0,
				title: void 0
			}
		}
	},
	unstable__errors: {
		action_blocked: void 0,
		already_a_member_in_organization: void 0,
		api_key_name_already_exists: void 0,
		api_key_usage_exceeded: void 0,
		avatar_file_size_exceeded: "Veličina fajla premašuje maksimalno ograničenje od 10 MB. Molimo izaberite manji fajl.",
		avatar_file_type_invalid: "Tip fajla nije podržan. Molimo otpremite sliku u formatu JPG, PNG, GIF ili WEBP.",
		captcha_invalid: "Registracija neuspešna zbog neuspelog sigurnosnog proveravanja. Osveži stranicu da pokušaš ponovo ili se obrati podršci za više pomoći.",
		captcha_unavailable: "Registracija neuspešna zbog neuspelog proveravanja bota. Osveži stranicu da pokušaš ponovo ili se obrati podršci za više pomoći.",
		form_code_incorrect: "Uneti kod je netačan.",
		form_email_address_blocked: "Привремене е-поште услуге нису подржане. Молимо користите своју редовну адресу е-поште за креирање налога.",
		form_identifier_exists__email_address: "Ova e-mail adresa je zauzeta. Molimo pokušaj sa drugom.",
		form_identifier_exists__phone_number: "Ovaj telefonski broj je zauzet. Molimo pokušaj sa drugim.",
		form_identifier_exists__username: "Ovo korisničko ime je zauzeto. Molimo pokušaj sa drugim.",
		form_identifier_not_found: "Nismo mogli pronaći nalog sa ovim podacima.",
		form_new_password_matches_current: void 0,
		form_param_format_invalid: "Format parametra je nevažeći.",
		form_param_format_invalid__email_address: "E-mail adresa mora biti važeća e-mail adresa.",
		form_param_format_invalid__phone_number: "Telefonski broj mora biti u važećem međunarodnom formatu",
		form_param_max_length_exceeded__first_name: "Ime ne sme premašiti 256 karaktera.",
		form_param_max_length_exceeded__last_name: "Prezime ne sme premašiti 256 karaktera.",
		form_param_max_length_exceeded__name: "Naziv ne sme premašiti 256 karaktera.",
		form_param_nil: "Parametar ne može biti prazan.",
		form_param_type_invalid: void 0,
		form_param_type_invalid__email_address: void 0,
		form_param_type_invalid__phone_number: void 0,
		form_param_value_invalid: void 0,
		form_password_compromised__sign_in: void 0,
		form_password_incorrect: "Lozinka je netačna.",
		form_password_length_too_short: "Lozinka je prekratka.",
		form_password_not_strong_enough: "Tvoja lozinka nije dovoljno jaka.",
		form_password_or_identifier_incorrect: "Лозинка или адреса е-поште је нетачна. Покушај поново или користи други метод.",
		form_password_pwned: "Ova lozinka je pronađena kao deo kompromitovanih podataka i ne može se koristiti, molimo pokušaj sa drugom lozinkom.",
		form_password_pwned__sign_in: "Ova lozinka je pronađena kao deo kompromitovanih podataka i ne može se koristiti, molimo resetuj svoju lozinku.",
		form_password_size_in_bytes_exceeded: "Tvoja lozinka je premašila maksimalni dozvoljeni broj bajtova, molimo skrati je ili ukloni neke specijalne znakove.",
		form_password_untrusted__sign_in: void 0,
		form_password_validation_failed: "Neispravna lozinka",
		form_username_invalid_character: "Korisničko ime sadrži nevažeće karaktere.",
		form_username_invalid_length: "Dužina korisničkog imena nije validna.",
		form_username_needs_non_number_char: "Ваше корисничко име мора садржати најмање један небројни знак.",
		identification_deletion_failed: "Ne možeš obrisati svoju poslednju identifikaciju.",
		insufficient_seats_change_plan: void 0,
		insufficient_seats_contact_support: void 0,
		not_allowed_access: "Adresa e-maila ili broja telefona nije dozvoljena za registraciju. Ovo može biti zbog korišćenja '+', '=', '#' ili '.' u adresi e-maila, korišćenja domena koji je povezan sa vremenskom e-mail uslugom ili eksplicitnom isključenju.",
		oauth_access_denied: void 0,
		organization_domain_blocked: void 0,
		organization_domain_common: void 0,
		organization_domain_exists_for_enterprise_connection: void 0,
		organization_membership_quota_exceeded: void 0,
		organization_minimum_permissions_needed: void 0,
		organization_not_found_or_unauthorized: void 0,
		organization_not_found_or_unauthorized_with_create_organization_disabled: void 0,
		passkey_already_exists: "Ključ za prolaz je već registrovan sa ovim uređajem.",
		passkey_not_supported: "Ključevi za prolaz nisu podržani na ovom uređaju.",
		passkey_pa_not_supported: "Registracija zahteva platformski autentifikator, ali uređaj to ne podržava.",
		passkey_registration_cancelled: "Registracija ključa za prolaz je otkazana ili je isteklo vreme.",
		passkey_retrieval_cancelled: "Verifikacija ključa za prolaz je otkazana ili je isteklo vreme.",
		passwordComplexity: {
			maximumLength: "manje od {{length}} karaktera",
			minimumLength: "{{length}} ili više karaktera",
			requireLowercase: "malo slovo",
			requireNumbers: "broj",
			requireSpecialCharacter: "specijalni znak",
			requireUppercase: "veliko slovo",
			sentencePrefix: "Tvoja lozinka mora sadržati"
		},
		phone_number_exists: "Ovaj telefonski broj je zauzet. Molimo pokušaj sa drugim.",
		protect_check_aborted: void 0,
		protect_check_already_resolved: void 0,
		protect_check_execution_failed: void 0,
		protect_check_invalid_script: void 0,
		protect_check_invalid_sdk_url: void 0,
		protect_check_script_load_failed: void 0,
		protect_check_timed_out: void 0,
		protect_check_unsupported_environment: void 0,
		session_exists: "Već ste prijavljeni.",
		web3_missing_identifier: void 0,
		web3_signature_request_rejected: "Odbio/la si zahtev za potpis. Pokušaj ponovo da nastaviš.",
		web3_solana_signature_generation_failed: "Došlo je do greške pri generisanju potpisa. Pokušaj ponovo da nastaviš.",
		zxcvbn: {
			couldBeStronger: "Tvoja lozinka funkcioniše, ali može biti jača. Pokušaj dodati više karaktera.",
			goodPassword: "Tvoja lozinka ispunjava sve potrebne zahteve.",
			notEnough: "Tvoja lozinka nije dovoljno jaka.",
			suggestions: {
				allUppercase: "Kapitalizuj neka, ali ne sva slova.",
				anotherWord: "Dodaj više reči koje su manje uobičajene.",
				associatedYears: "Izbegavaj godine koje su povezane sa tobom.",
				capitalization: "Kapitalizuj više od prvog slova.",
				dates: "Izbegavaj datume i godine koje su povezane sa tobom.",
				l33t: "Izbegavaj predvidljive zamene slova kao što su '@' za 'a'.",
				longerKeyboardPattern: "Koristi duže šablone na tastaturi i promeni smer kucanja više puta.",
				noNeed: "Možeš kreirati jake lozinke bez korišćenja simbola, brojeva ili velikih slova.",
				pwned: "Ako koristiš ovu lozinku negde drugde, trebalo bi da je promeniš.",
				recentYears: "Izbegavaj skorašnje godine.",
				repeated: "Izbegavaj ponavljane reči i karaktere.",
				reverseWords: "Izbegavaj obrnuto napisane uobičajene reči.",
				sequences: "Izbegavaj uobičajene sekvence karaktera.",
				useWords: "Koristi više reči, ali izbegavaj uobičajene fraze."
			},
			warnings: {
				common: "Ovo je često korišćena lozinka.",
				commonNames: "Uobičajena imena i prezimena su lako za pogoditi.",
				dates: "Datumi su lako za pogoditi.",
				extendedRepeat: "Ponavljani obrasci karaktera kao što su \"abcabcabc\" su lako za pogoditi.",
				keyPattern: "Kratki šabloni na tastaturi su lako za pogoditi.",
				namesByThemselves: "Pojedinačna imena ili prezimena su lako za pogoditi.",
				pwned: "Tvoja lozinka je otkrivena u kršenju podataka na internetu.",
				recentYears: "Skorašnje godine su lako za pogoditi.",
				sequences: "Uobičajene sekvence karaktera kao što su \"abc\" su lako za pogoditi.",
				similarToCommon: "Ovo je slično često korišćenoj lozinki.",
				simpleRepeat: "Ponavljani karakteri kao što su \"aaa\" su lako za pogoditi.",
				straightRow: "Direktne linije tastera na tvojoj tastaturi su lako za pogoditi.",
				topHundred: "Ovo je često korišćena lozinka.",
				topTen: "Ovo je veoma često korišćena lozinka.",
				userInputs: "Ne sme biti ličnih podataka ili podataka vezanih za stranicu.",
				wordByItself: "Pojedinačne reči su lako za pogoditi."
			}
		}
	},
	userButton: {
		action__addAccount: "Dodaj nalog",
		action__closeUserMenu: void 0,
		action__manageAccount: "Upravljaj nalogom",
		action__openUserMenu: void 0,
		action__signOut: "Odjavi se",
		action__signOutAll: "Odjavi se sa svih naloga",
		label__accountActions: "Акције налога",
		label__activeSessions: "Активне сесије",
		label__userButtonPopover: "Панел налога"
	},
	userProfile: {
		apiKeysPage: { title: void 0 },
		backupCodePage: {
			actionLabel__copied: "Kopirano!",
			actionLabel__copy: "Kopiraj sve",
			actionLabel__download: "Preuzmi .txt",
			actionLabel__print: "Štampaj",
			infoText1: "Rezervni kodovi će biti omogućeni za ovaj nalog.",
			infoText2: "Čuvaj rezervne kodove u tajnosti i čuvaj ih na sigurnom mestu. Možeš regenerisati rezervne kodove ako sumnjaš da su kompromitovani.",
			subtitle__codelist: "Čuvaj ih na sigurnom i drži ih u tajnosti.",
			successMessage: "Rezervni kodovi su sada omogućeni. Možeš koristiti jedan od ovih kodova za prijavu na svoj nalog, ako izgubiš pristup svom uređaju za autentifikaciju. Svaki kod može biti korišćen samo jednom.",
			successSubtitle: "Možeš koristiti jedan od ovih kodova za prijavu na svoj nalog, ako izgubiš pristup svom uređaju za autentifikaciju.",
			title: "Dodaj verifikaciju rezervnim kodom",
			title__codelist: "Rezervni kodovi"
		},
		billingPage: {
			accountCreditsSection: {
				title: void 0,
				viewHistory: void 0
			},
			creditHistoryPage: {
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				title: void 0
			},
			paymentHistorySection: {
				empty: void 0,
				notFound: void 0,
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				tableHeader__status: void 0
			},
			paymentMethodsSection: {
				actionLabel__default: void 0,
				actionLabel__remove: void 0,
				add: void 0,
				addSubtitle: void 0,
				cancelButton: void 0,
				formButtonPrimary__add: void 0,
				formButtonPrimary__pay: void 0,
				payWithTestCardButton: void 0,
				removeMethod: {
					messageLine1: void 0,
					messageLine2: void 0,
					successMessage: void 0,
					title: void 0
				},
				title: void 0
			},
			start: {
				headerTitle__payments: void 0,
				headerTitle__plans: void 0,
				headerTitle__statements: void 0,
				headerTitle__subscriptions: void 0
			},
			statementsSection: {
				empty: void 0,
				itemCaption__paidForPlan: void 0,
				itemCaption__payerCredit: void 0,
				itemCaption__proratedCredit: void 0,
				itemCaption__subscribedAndPaidForPlan: void 0,
				notFound: void 0,
				tableHeader__amount: void 0,
				tableHeader__date: void 0,
				title: void 0,
				totalPaid: void 0
			},
			subscriptionsListSection: {
				actionLabel__manageSubscription: void 0,
				actionLabel__newSubscription: void 0,
				actionLabel__switchPlan: void 0,
				overview: void 0,
				tableHeader__edit: void 0,
				tableHeader__plan: void 0,
				tableHeader__startDate: void 0,
				title: void 0
			},
			subscriptionsSection: { actionLabel__default: void 0 },
			switchPlansSection: { title: void 0 },
			title: void 0
		},
		connectedAccountPage: {
			formHint: "Izaberi provajdera da povežeš svoj nalog.",
			formHint__noAccounts: "Nema dostupnih spoljnih provajdera naloga.",
			removeResource: {
				messageLine1: "{{identifier}} će biti uklonjen iz ovog naloga.",
				messageLine2: "Više nećeš moći da koristiš ovaj povezani nalog i bilo koje zavisne funkcije više neće raditi.",
				successMessage: "{{connectedAccount}} je uklonjen iz tvog naloga.",
				title: "Ukloni povezani nalog"
			},
			socialButtonsBlockButton: "{{provider|titleize}}",
			successMessage: "Provajder je dodat na tvoj nalog",
			title: "Dodaj povezani nalog"
		},
		deletePage: {
			actionDescription: "Upiši \"Delete account\" ispod da nastaviš.",
			confirm: "Obriši nalog",
			messageLine1: "Да ли сте сигурни да желите да избришете свој налог? Неки повезани подаци могу бити задржани. За захтев за потпуно брисање података, молимо контактирајте подршку.",
			messageLine2: "Ova akcija je trajna i nepovratna.",
			title: "Obriši nalog"
		},
		emailAddressPage: {
			emailCode: {
				formHint: "E-mail sadrži verifikacioni kod koji će biti poslat na ovu e-mail adresu.",
				formSubtitle: "Unesi verifikacioni kod poslat na {{identifier}}",
				formTitle: "Verifikacioni kod",
				resendButton: "Nisi primio kod? Pošalji ponovo",
				successMessage: "E-mail {{identifier}} je dodat na tvoj nalog."
			},
			emailLink: {
				formHint: "E-mail sadrži verifikacioni link koji će biti poslat na ovu e-mail adresu.",
				formSubtitle: "Klikni na verifikacioni link u e-mailu poslatom na {{identifier}}",
				formTitle: "Verifikacioni link",
				resendButton: "Nisi primio link? Pošalji ponovo",
				successMessage: "E-mail {{identifier}} je dodat na tvoj nalog."
			},
			enterpriseSSOLink: {
				formButton: void 0,
				formSubtitle: void 0
			},
			formHint: void 0,
			removeResource: {
				messageLine1: "{{identifier}} će biti uklonjen iz ovog naloga.",
				messageLine2: "Više nećeš moći da se prijaviš koristeći ovu e-mail adresu.",
				successMessage: "{{emailAddress}} je uklonjen iz tvog naloga.",
				title: "Ukloni e-mail adresu"
			},
			title: "Dodaj e-mail adresu",
			verifyTitle: "Verifikuj e-mail adresu"
		},
		formButtonPrimary__add: "Dodaj",
		formButtonPrimary__continue: "Nastavi",
		formButtonPrimary__finish: "Završi",
		formButtonPrimary__remove: "Ukloni",
		formButtonPrimary__save: "Sačuvaj",
		formButtonReset: "Otkaži",
		mfaPage: {
			formHint: "Izaberi metodu za dodavanje.",
			title: "Dodaj dvostepenu verifikaciju"
		},
		mfaPhoneCodePage: {
			backButton: "Koristi postojeći broj",
			primaryButton__addPhoneNumber: "Dodaj telefonski broj",
			removeResource: {
				messageLine1: "{{identifier}} više neće primati verifikacione kodove prilikom prijavljivanja.",
				messageLine2: "Tvoj nalog možda neće biti toliko siguran. Da li si siguran da želiš da nastaviš?",
				successMessage: "SMS kod dvostepene verifikacije je uklonjen za {{mfaPhoneCode}}",
				title: "Ukloni dvostepenu verifikaciju"
			},
			subtitle__availablePhoneNumbers: "Izaberi postojeći telefonski broj za registraciju SMS kod dvostepene verifikacije ili dodaj novi.",
			subtitle__unavailablePhoneNumbers: "Nema dostupnih telefonskih brojeva za registraciju SMS kod dvostepene verifikacije, molimo dodaj novi.",
			successMessage1: "Kada se prijaviš, moraćeš uneti verifikacioni kod poslat na ovaj telefonski broj kao dodatni korak.",
			successMessage2: "Sačuvaj ove rezervne kodove i čuvaj ih na sigurnom mestu. Ako izgubiš pristup svom uređaju za autentifikaciju, možeš koristiti rezervne kodove za prijavu.",
			successTitle: "SMS kod verifikacija je omogućena",
			title: "Dodaj SMS kod verifikaciju"
		},
		mfaTOTPPage: {
			authenticatorApp: {
				buttonAbleToScan__nonPrimary: "Umesto toga, skeniraj QR kod",
				buttonUnableToScan__nonPrimary: "Ne možeš skenirati QR kod?",
				infoText__ableToScan: "Podesi novi način prijave u svojoj aplikaciji za autentifikaciju i skeniraj sledeći QR kod da ga povežeš sa svojim nalogom.",
				infoText__unableToScan: "Podesi novi način prijave u svojoj autentifikaciji i unesi ključ naveden ispod.",
				inputLabel__unableToScan1: "Uveri se da su vremenski bazirane ili jednokratne lozinke omogućene, zatim završi povezivanje svog naloga.",
				inputLabel__unableToScan2: "Alternativno, ako tvoja autentifikacija podržava TOTP URI, možeš takođe kopirati celu URI adresu."
			},
			removeResource: {
				messageLine1: "Verifikacioni kodovi iz ove autentifikacije više neće biti potrebni prilikom prijavljivanja.",
				messageLine2: "Tvoj nalog možda neće biti toliko siguran. Da li si siguran da želiš da nastaviš?",
				successMessage: "Dvostepena verifikacija preko autentifikacione aplikacije je uklonjena.",
				title: "Ukloni dvostepenu verifikaciju"
			},
			successMessage: "Dvostepena verifikacija je sada omogućena. Kada se prijaviš, moraćeš uneti verifikacioni kod iz ove autentifikacije kao dodatni korak.",
			title: "Dodaj autentifikacionu aplikaciju",
			verifySubtitle: "Unesi verifikacioni kod generisan tvojom autentifikacijom",
			verifyTitle: "Verifikacioni kod"
		},
		mobileButton__menu: "Meni",
		navbar: {
			account: "Profil",
			apiKeys: void 0,
			billing: void 0,
			description: "Upravljaj informacijama svog naloga.",
			security: "Sigurnost",
			title: "Nalog"
		},
		passkeyScreen: {
			removeResource: {
				messageLine1: "{{name}} će biti uklonjen iz ovog naloga.",
				title: "Ukloni ključ za prolaz"
			},
			subtitle__rename: "Možeš promeniti ime ključa za prolaz kako bi ga lakše pronašao.",
			title__rename: "Preimenuj ključ za prolaz"
		},
		passwordPage: {
			checkboxInfoText__signOutOfOtherSessions: "Preporučuje se da se odjaviš sa svih drugih uređaja koji su možda koristili tvoju staru lozinku.",
			readonly: "Tvoja lozinka trenutno ne može biti uređivana jer se možeš prijaviti samo preko korporativne veze.",
			successMessage__set: "Tvoja lozinka je postavljena.",
			successMessage__signOutOfOtherSessions: "Svi drugi uređaji su odjavljeni.",
			successMessage__update: "Tvoja lozinka je ažurirana.",
			title__set: "Postavi lozinku",
			title__update: "Ažuriraj lozinku"
		},
		phoneNumberPage: {
			infoText: "Tekstualna poruka sa verifikacionim kodom će biti poslata na ovaj telefonski broj. Moguće su naknade za poruke i podatke.",
			removeResource: {
				messageLine1: "{{identifier}} će biti uklonjen iz ovog naloga.",
				messageLine2: "Više nećeš moći da se prijaviš koristeći ovaj telefonski broj.",
				successMessage: "{{phoneNumber}} je uklonjen iz tvog naloga.",
				title: "Ukloni telefonski broj"
			},
			successMessage: "{{identifier}} je dodat na tvoj nalog.",
			title: "Dodaj telefonski broj",
			verifySubtitle: "Unesi verifikacioni kod poslat na {{identifier}}",
			verifyTitle: "Verifikuj telefonski broj"
		},
		plansPage: { title: void 0 },
		profilePage: {
			fileDropAreaHint: "Preporučena veličina 1:1, do 10MB.",
			imageFormDestructiveActionSubtitle: "Ukloni",
			imageFormSubtitle: "Otpremi",
			imageFormTitle: "Profilna slika",
			readonly: "Tvoje profilne informacije su obezbeđene preko korporativne veze i ne mogu biti uređivane.",
			successMessage: "Tvoj profil je ažuriran.",
			title: "Ažuriraj profil"
		},
		start: {
			activeDevicesSection: {
				destructiveAction: "Odjavi uređaj",
				title: "Aktivni uređaji"
			},
			connectedAccountsSection: {
				actionLabel__connectionFailed: "Pokušaj ponovo",
				actionLabel__reauthorize: "Autorizuj sada",
				destructiveActionTitle: "Ukloni",
				primaryButton: "Poveži nalog",
				subtitle__disconnected: void 0,
				subtitle__reauthorize: "Potrebna ovlašćenja su ažurirana, i možda doživljavaš ograničenu funkcionalnost. Molimo re-autorizuj ovu aplikaciju da izbegneš bilo kakve probleme",
				title: "Povezani nalozi"
			},
			dangerSection: {
				deleteAccountButton: "Obriši nalog",
				title: "Obriši nalog"
			},
			emailAddressesSection: {
				destructiveAction: "Ukloni e-mail",
				detailsAction__nonPrimary: "Postavi kao primarni",
				detailsAction__primary: "Završi verifikaciju",
				detailsAction__unverified: "Verifikuj",
				primaryButton: "Dodaj e-mail adresu",
				title: "E-mail adrese"
			},
			enterpriseAccountsSection: {
				primaryButton: "Poveži nalog",
				title: "Korporativni nalozi"
			},
			headerTitle__account: "Detalji profila",
			headerTitle__security: "Sigurnost",
			mfaSection: {
				backupCodes: {
					actionLabel__regenerate: "Regeneriši",
					headerTitle: "Rezervni kodovi",
					subtitle__regenerate: "Dobij novi set sigurnih rezervnih kodova. Prethodni rezervni kodovi će biti obrisani i neće moći biti korišćeni.",
					title__regenerate: "Regeneriši rezervne kodove"
				},
				phoneCode: {
					actionLabel__setDefault: "Postavi kao podrazumevani",
					destructiveActionLabel: "Ukloni"
				},
				primaryButton: "Dodaj dvostepenu verifikaciju",
				title: "Dvostepena verifikacija",
				totp: {
					destructiveActionTitle: "Ukloni",
					headerTitle: "Autentifikaciona aplikacija"
				}
			},
			passkeysSection: {
				menuAction__destructive: "Ukloni",
				menuAction__rename: "Preimenuj",
				primaryButton: void 0,
				title: "Ključevi za prolaz"
			},
			passwordSection: {
				primaryButton__setPassword: "Postavi lozinku",
				primaryButton__updatePassword: "Ažuriraj lozinku",
				title: "Lozinka"
			},
			phoneNumbersSection: {
				destructiveAction: "Ukloni telefonski broj",
				detailsAction__nonPrimary: "Postavi kao podrazumevani",
				detailsAction__primary: "Završi verifikaciju",
				detailsAction__unverified: "Verifikuj telefonski broj",
				primaryButton: "Dodaj telefonski broj",
				title: "Telefonski brojevi"
			},
			profileSection: {
				primaryButton: "Ažuriraj profil",
				title: "Profil"
			},
			usernameSection: {
				primaryButton__setUsername: "Postavi korisničko ime",
				primaryButton__updateUsername: "Ažuriraj korisničko ime",
				title: "Korisničko ime"
			},
			web3WalletsSection: {
				destructiveAction: "Ukloni novčanik",
				detailsAction__nonPrimary: void 0,
				primaryButton: "Web3 novčanici",
				title: "Web3 novčanici",
				web3SelectSolanaWalletScreen: {
					subtitle: "Izaberi Solana novčanik da ga povežeš sa svojim nalogom.",
					title: "Dodaj Solana novčanik"
				}
			}
		},
		usernamePage: {
			successMessage: "Tvoje korisničko ime je ažurirano.",
			title__set: "Postavi korisničko ime",
			title__update: "Ažuriraj korisničko ime"
		},
		web3WalletPage: {
			removeResource: {
				messageLine1: "{{identifier}} će biti uklonjen iz ovog naloga.",
				messageLine2: "Više nećeš moći da se prijaviš koristeći ovaj web3 novčanik.",
				successMessage: "{{web3Wallet}} je uklonjen iz tvog naloga.",
				title: "Ukloni web3 novčanik"
			},
			subtitle__availableWallets: "Izaberi web3 novčanik da ga povežeš sa svojim nalogom.",
			subtitle__unavailableWallets: "Nema dostupnih web3 novčanika.",
			successMessage: "Novčanik je dodat na tvoj nalog.",
			title: "Dodaj web3 novčanik",
			web3WalletButtonsBlockButton: void 0
		}
	},
	waitlist: {
		start: {
			actionLink: "Пријави се",
			actionText: "Већ имаш приступ?",
			formButton: "Придружи се листи чекања",
			subtitle: "Унеси своју е-пошту и обавестићемо те када твоје место буде спремно",
			title: "Придружи се листи чекања"
		},
		success: {
			message: "Ускоро ћеш бити преусмерен...",
			subtitle: "Јавићемо ти се када твоје место буде спремно",
			title: "Хвала што си се придружио/ла листи чекања!"
		}
	},
	web3SolanaWalletButtons: {
		connect: "Poveži se sa {{walletName}}",
		continue: "Nastavi sa {{walletName}}",
		noneAvailable: "Nisu detektovani Solana Web3 novčanici. Instaliraj {{ solanaWalletsLink || link(\"wallet extension\") }} koji podržava Web3."
	}
};

//#endregion
exports.srRS = srRS;
