// @clerk/ui/dist/no-rhc/components/PaymentAttempts/PaymentAttemptPage.js
import { useRouter } from "../../router/RouteContext.js";
import { useClipboard } from "../../hooks/useClipboard.js";
import { localizationKeys } from "../../localization/localizationKeys.js";
import SvgCheckmark from "../../icons/checkmark.js";
import SvgCopy from "../../icons/copy.js";
import { getPlanSeatLimit, getSeatsPerUnitTotal, summarizeSeatCharges } from "../../utils/billingPlanSeats.js";
import { useSubscriberTypeContext, useSubscriberTypeLocalizationRoot } from "../../contexts/components/SubscriberType.js";
import { useLocalizations } from "../../localization/makeLocalizable.js";
import { descriptors } from "../../customizables/elementDescriptors.js";
import { Badge, Box, Button, Heading, Icon, Span, Spinner, Text } from "../../customizables/index.js";
import { Alert } from "../../elements/Alert.js";
import { Header } from "../../elements/Header.js";
import { ProfileCard } from "../../elements/ProfileCard/index.js";
import { truncateWithEndVisible } from "../../utils/truncateTextWithEndVisible.js";
import { LineItems } from "../../elements/LineItems.js";
import { formatDate } from "../../utils/formatDate.js";
import { toNegativeAmount } from "../../utils/billing.js";
import { jsx, jsxs } from "@emotion/react/jsx-runtime";
import { __internal_usePaymentAttemptQuery } from "@clerk/shared/react/index";

//#region src/components/PaymentAttempts/PaymentAttemptPage.tsx
const PaymentAttemptPage = () => {
	const { params, navigate } = useRouter();
	const subscriberType = useSubscriberTypeContext();
	const localizationRoot = useSubscriberTypeLocalizationRoot();
	const { t, translateError, $ } = useLocalizations();
	const requesterType = subscriberType === "organization" ? "organization" : "user";
	const { data: paymentAttempt, isLoading, error } = __internal_usePaymentAttemptQuery({
		paymentAttemptId: params.paymentAttemptId,
		for: requesterType,
		enabled: Boolean(params.paymentAttemptId)
	});
	if (isLoading) return /* @__PURE__ */ jsx(ProfileCard.Page, { children: /* @__PURE__ */ jsx(Box, {
		sx: {
			display: "flex",
			justifyContent: "center",
			alignItems: "center",
			height: "100%"
		},
		children: /* @__PURE__ */ jsx(Spinner, {
			colorScheme: "primary",
			sx: {
				margin: "auto",
				display: "block"
			},
			elementDescriptor: descriptors.spinner
		})
	}) });
	return /* @__PURE__ */ jsxs(ProfileCard.Page, { children: [/* @__PURE__ */ jsx(Header.Root, {
		sx: (t) => ({
			borderBlockEndWidth: t.borderWidths.$normal,
			borderBlockEndStyle: t.borderStyles.$solid,
			borderBlockEndColor: t.colors.$borderAlpha100,
			marginBlockEnd: t.space.$4,
			paddingBlockEnd: t.space.$4
		}),
		children: /* @__PURE__ */ jsx(Header.BackLink, {
			onClick: () => void navigate("../../", { searchParams: new URLSearchParams("tab=payments") }),
			children: /* @__PURE__ */ jsx(Header.Title, {
				localizationKey: localizationKeys(`${localizationRoot}.billingPage.start.headerTitle__payments`),
				textVariant: "h2"
			})
		})
	}), !paymentAttempt ? /* @__PURE__ */ jsx(Box, {
		sx: {
			display: "flex",
			justifyContent: "center",
			alignItems: "center",
			height: "100%"
		},
		children: /* @__PURE__ */ jsx(Alert, {
			variant: "danger",
			colorScheme: "danger",
			children: error ? translateError(error.errors[0]) : t(localizationKeys(`${localizationRoot}.billingPage.paymentHistorySection.notFound`))
		})
	}) : /* @__PURE__ */ jsxs(Box, {
		elementDescriptor: descriptors.paymentAttemptRoot,
		as: "article",
		sx: (t) => ({
			borderWidth: t.borderWidths.$normal,
			borderStyle: t.borderStyles.$solid,
			borderColor: t.colors.$borderAlpha100,
			borderRadius: t.radii.$lg,
			overflow: "clip"
		}),
		children: [
			/* @__PURE__ */ jsxs(Box, {
				elementDescriptor: descriptors.paymentAttemptHeader,
				as: "header",
				sx: (t) => ({
					padding: t.space.$4,
					background: t.colors.$neutralAlpha25,
					display: "flex",
					justifyContent: "space-between",
					alignItems: "flex-start"
				}),
				children: [/* @__PURE__ */ jsxs(Span, {
					elementDescriptor: descriptors.paymentAttemptHeaderTitleContainer,
					children: [/* @__PURE__ */ jsx(Heading, {
						elementDescriptor: descriptors.paymentAttemptHeaderTitle,
						textVariant: "h2",
						localizationKey: formatDate(paymentAttempt.paidAt || paymentAttempt.failedAt || paymentAttempt.updatedAt, "long")
					}), /* @__PURE__ */ jsxs(Span, {
						sx: (t) => ({
							display: "flex",
							alignItems: "center",
							gap: t.space.$0x25,
							color: t.colors.$colorMutedForeground
						}),
						children: [/* @__PURE__ */ jsx(CopyButton, {
							copyLabel: "Copy payment attempt ID",
							text: paymentAttempt.id
						}), /* @__PURE__ */ jsx(Text, {
							colorScheme: "secondary",
							variant: "subtitle",
							children: truncateWithEndVisible(paymentAttempt.id)
						})]
					})]
				}), /* @__PURE__ */ jsx(Badge, {
					elementDescriptor: descriptors.paymentAttemptHeaderBadge,
					colorScheme: paymentAttempt.status === "paid" ? "success" : paymentAttempt.status === "failed" ? "danger" : "primary",
					sx: { textTransform: "capitalize" },
					children: paymentAttempt.status
				})]
			}),
			/* @__PURE__ */ jsx(PaymentAttemptBody, { paymentAttempt }),
			/* @__PURE__ */ jsxs(Box, {
				elementDescriptor: descriptors.paymentAttemptFooter,
				as: "footer",
				sx: (t) => ({
					paddingInline: t.space.$4,
					paddingBlock: t.space.$3,
					background: t.colors.$neutralAlpha25,
					borderBlockStartWidth: t.borderWidths.$normal,
					borderBlockStartStyle: t.borderStyles.$solid,
					borderBlockStartColor: t.colors.$borderAlpha100,
					display: "flex",
					justifyContent: "space-between"
				}),
				children: [/* @__PURE__ */ jsx(Text, {
					variant: "h3",
					localizationKey: localizationKeys("billing.totalDue"),
					elementDescriptor: descriptors.paymentAttemptFooterLabel
				}), /* @__PURE__ */ jsxs(Span, {
					elementDescriptor: descriptors.paymentAttemptFooterValueContainer,
					sx: (t) => ({
						display: "flex",
						alignItems: "center",
						gap: t.space.$2x5
					}),
					children: [/* @__PURE__ */ jsx(Text, {
						variant: "caption",
						colorScheme: "secondary",
						elementDescriptor: descriptors.paymentAttemptFooterCurrency,
						sx: { textTransform: "uppercase" },
						children: paymentAttempt.amount.currency
					}), /* @__PURE__ */ jsx(Text, {
						variant: "h3",
						elementDescriptor: descriptors.paymentAttemptFooterValue,
						children: $(paymentAttempt.amount)
					})]
				})]
			})
		]
	})] });
};
function PaymentAttemptBody({ paymentAttempt }) {
	const { $ } = useLocalizations();
	if (!paymentAttempt) return null;
	const { subscriptionItem } = paymentAttempt;
	const fee = subscriptionItem.planPeriod === "month" ? subscriptionItem.plan.fee : subscriptionItem.plan.annualMonthlyFee;
	const seatSummary = summarizeSeatCharges(subscriptionItem.seats != null ? getSeatsPerUnitTotal(paymentAttempt.totals) : void 0);
	const seatsChargeable = seatSummary ? seatSummary.totalSeats - seatSummary.included : 0;
	const planSeatLimit = getPlanSeatLimit(subscriptionItem.plan);
	return /* @__PURE__ */ jsx(Box, {
		elementDescriptor: descriptors.paymentAttemptBody,
		sx: (t) => ({ padding: t.space.$4 }),
		children: /* @__PURE__ */ jsxs(LineItems.Root, { children: [
			/* @__PURE__ */ jsxs(LineItems.Group, { children: [/* @__PURE__ */ jsx(LineItems.Title, { title: subscriptionItem.plan.name }), /* @__PURE__ */ jsx(LineItems.Description, {
				prefix: subscriptionItem.planPeriod === "annual" ? "x12" : void 0,
				text: $(fee)
			})] }),
			seatSummary && /* @__PURE__ */ jsxs(LineItems.Group, { children: [/* @__PURE__ */ jsx(LineItems.Title, {
				title: planSeatLimit != null ? localizationKeys("billing.seatsWithLimit", { limit: planSeatLimit }) : localizationKeys("billing.seats"),
				description: (() => {
					const rate = $(seatSummary.paidTier.feePerBlock);
					const isSingular = seatsChargeable === 1;
					if (seatSummary.included > 0) return isSingular ? localizationKeys("billing.seatBreakdownIncludedSingular", {
						totalSeats: seatSummary.totalSeats,
						included: seatSummary.included,
						rate
					}) : localizationKeys("billing.seatBreakdownIncludedPlural", {
						totalSeats: seatSummary.totalSeats,
						included: seatSummary.included,
						chargeable: seatsChargeable,
						rate
					});
					return isSingular ? localizationKeys("billing.seatBreakdownSingular", { rate }) : localizationKeys("billing.seatBreakdownPlural", {
						chargeable: seatsChargeable,
						rate
					});
				})()
			}), /* @__PURE__ */ jsx(LineItems.Description, {
				prefix: subscriptionItem.planPeriod === "annual" ? "x12" : void 0,
				text: $(seatSummary.paidTier.total)
			})] }),
			/* @__PURE__ */ jsxs(LineItems.Group, {
				borderTop: true,
				variant: "tertiary",
				children: [/* @__PURE__ */ jsx(LineItems.Title, { title: localizationKeys("billing.subtotal") }), /* @__PURE__ */ jsx(LineItems.Description, { text: paymentAttempt.totals?.subtotal ? $(paymentAttempt.totals.subtotal) : "" })]
			}),
			paymentAttempt.totals?.discounts?.proration && paymentAttempt.totals.discounts.proration.amount.amount > 0 && /* @__PURE__ */ jsxs(LineItems.Group, {
				variant: "tertiary",
				children: [/* @__PURE__ */ jsx(LineItems.Title, { title: localizationKeys("billing.proratedDiscount") }), /* @__PURE__ */ jsx(LineItems.Description, { text: $(toNegativeAmount(paymentAttempt.totals.discounts.proration.amount)) })]
			}),
			subscriptionItem.credits && subscriptionItem.credits.proration && subscriptionItem.credits.proration.amount.amount > 0 && /* @__PURE__ */ jsxs(LineItems.Group, {
				variant: "tertiary",
				children: [/* @__PURE__ */ jsx(LineItems.Title, { title: localizationKeys("billing.prorationCredit") }), /* @__PURE__ */ jsx(LineItems.Description, { text: $(toNegativeAmount(subscriptionItem.credits.proration.amount)) })]
			}),
			subscriptionItem.credits && subscriptionItem.credits.payer && subscriptionItem.credits.payer.appliedAmount.amount > 0 && /* @__PURE__ */ jsxs(LineItems.Group, {
				variant: "tertiary",
				children: [/* @__PURE__ */ jsx(LineItems.Title, { title: localizationKeys("billing.accountCredit") }), /* @__PURE__ */ jsx(LineItems.Description, { text: $(toNegativeAmount(subscriptionItem.credits.payer.appliedAmount)) })]
			})
		] })
	});
}
function CopyButton({ text, copyLabel = "Copy" }) {
	const { onCopy, hasCopied } = useClipboard(text);
	return /* @__PURE__ */ jsx(Button, {
		elementDescriptor: descriptors.paymentAttemptCopyButton,
		variant: "unstyled",
		onClick: () => onCopy(),
		sx: (t) => ({
			color: "inherit",
			width: t.sizes.$4,
			height: t.sizes.$4,
			padding: 0,
			borderRadius: t.radii.$sm,
			"&:focus-visible": {
				outline: "2px solid",
				outlineColor: t.colors.$colorRing
			}
		}),
		focusRing: false,
		"aria-label": hasCopied ? "Copied" : copyLabel,
		children: /* @__PURE__ */ jsx(Icon, {
			size: "sm",
			icon: hasCopied ? SvgCheckmark : SvgCopy,
			"aria-hidden": true
		})
	});
}

//#endregion
export { PaymentAttemptPage };
