// @clerk/ui/dist/no-rhc/components/ConfigureSSO/steps/ConfigureStep/saml/SamlMicrosoftConfigureSteps.js
import { useClipboard } from "../../../../../hooks/useClipboard.js";
import { localizationKeys } from "../../../../../localization/localizationKeys.js";
import SvgCheckmark from "../../../../../icons/checkmark.js";
import SvgClipboard from "../../../../../icons/clipboard.js";
import { useLocalizations } from "../../../../../localization/makeLocalizable.js";
import { useFormControl } from "../../../../../utils/useFormControl.js";
import { descriptors } from "../../../../../customizables/elementDescriptors.js";
import { useCardState } from "../../../../../elements/contexts/index.js";
import { Badge, Box, Button, Col, Flex, Heading, Icon, Table, Tbody, Td, Text, Th, Thead, Tr } from "../../../../../customizables/index.js";
import { Form } from "../../../../../elements/Form.js";
import { ClipboardInput } from "../../../../../elements/ClipboardInput.js";
import { truncateWithEndVisible } from "../../../../../utils/truncateTextWithEndVisible.js";
import { Tooltip } from "../../../../../elements/Tooltip.js";
import { useConfigureSSO } from "../../../ConfigureSSOContext.js";
import { Step } from "../../../elements/Step.js";
import { useWizard } from "../../../elements/Wizard/WizardContext.js";
import { Wizard } from "../../../elements/Wizard/Wizard.js";
import { InnerStepCounter } from "../../../elements/Wizard/InnerStepCounter.js";
import { IdentityProviderConfigurationForm, applySamlSubmitError, buildSamlConfigurationPayload } from "./shared/IdentityProviderConfigurationForm.js";
import { IdentityProviderConfigurationModes } from "./shared/IdentityProviderConfigurationModes.js";
import React from "react";
import { Fragment as Fragment$1, jsx, jsxs } from "@emotion/react/jsx-runtime";

//#region src/components/ConfigureSSO/steps/ConfigureStep/saml/SamlMicrosoftConfigureSteps.tsx
const MICROSOFT_STEPS = [
	{ id: "create-app" },
	{ id: "service-provider" },
	{ id: "attribute-mapping" },
	{ id: "identity-provider-metadata" }
];
const SamlMicrosoftConfigureSteps = () => {
	return /* @__PURE__ */ jsxs(Wizard, {
		steps: MICROSOFT_STEPS,
		initialStepId: MICROSOFT_STEPS[0].id,
		children: [
			/* @__PURE__ */ jsx(Wizard.Match, {
				id: "create-app",
				children: /* @__PURE__ */ jsx(SamlMicrosoftCreateAppStep, {})
			}),
			/* @__PURE__ */ jsx(Wizard.Match, {
				id: "service-provider",
				children: /* @__PURE__ */ jsx(SamlMicrosoftServiceProviderStep, {})
			}),
			/* @__PURE__ */ jsx(Wizard.Match, {
				id: "attribute-mapping",
				children: /* @__PURE__ */ jsx(SamlMicrosoftAttributeMappingStep, {})
			}),
			/* @__PURE__ */ jsx(Wizard.Match, {
				id: "identity-provider-metadata",
				children: /* @__PURE__ */ jsx(SamlMicrosoftIdentityProviderMetadataStep, {})
			})
		]
	});
};
const SamlMicrosoftCreateAppStep = () => {
	const { goNext, goPrev, isFirstStep, isLastStep } = useWizard();
	return /* @__PURE__ */ jsxs(Fragment$1, { children: [
		/* @__PURE__ */ jsx(Step.Header, {
			title: localizationKeys("configureSSO.configureStep.samlMicrosoft.mainHeaderTitle"),
			description: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.headerSubtitle"),
			children: /* @__PURE__ */ jsx(InnerStepCounter, {})
		}),
		/* @__PURE__ */ jsx(Step.Body, { children: /* @__PURE__ */ jsxs(Step.Section, {
			sx: (theme) => ({ gap: theme.space.$5 }),
			children: [/* @__PURE__ */ jsxs(Col, {
				sx: (theme) => ({ gap: theme.space.$1x5 }),
				children: [/* @__PURE__ */ jsx(Heading, {
					elementDescriptor: descriptors.configureSSOInstructionsHeading,
					as: "h3",
					textVariant: "subtitle",
					localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.createAppInstructions.title")
				}), /* @__PURE__ */ jsxs(Col, {
					elementDescriptor: descriptors.configureSSOInstructionsList,
					as: "ul",
					sx: (theme) => ({
						gap: theme.space.$1x5,
						margin: 0,
						paddingInlineStart: theme.space.$5,
						listStyleType: "disc"
					}),
					children: [
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.createAppInstructions.step1")
						}),
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.createAppInstructions.step2")
						}),
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.createAppInstructions.step3")
						}),
						/* @__PURE__ */ jsxs(Box, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							sx: (theme) => ({
								fontSize: theme.fontSizes.$md,
								lineHeight: theme.lineHeights.$small,
								color: theme.colors.$colorMutedForeground
							}),
							children: [/* @__PURE__ */ jsx(Text, {
								as: "span",
								colorScheme: "secondary",
								localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.createAppInstructions.step4.label")
							}), /* @__PURE__ */ jsxs(Col, {
								elementDescriptor: descriptors.configureSSOInstructionsList,
								as: "ul",
								sx: (theme) => ({
									gap: theme.space.$1x5,
									marginBlockStart: theme.space.$1x5,
									marginBlockEnd: 0,
									paddingInlineStart: theme.space.$5,
									listStyleType: "circle"
								}),
								children: [
									/* @__PURE__ */ jsx(Text, {
										elementDescriptor: descriptors.configureSSOInstructionsListItem,
										as: "li",
										colorScheme: "secondary",
										localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.createAppInstructions.step4.subSteps.appName")
									}),
									/* @__PURE__ */ jsx(Text, {
										elementDescriptor: descriptors.configureSSOInstructionsListItem,
										as: "li",
										colorScheme: "secondary",
										localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.createAppInstructions.step4.subSteps.nonGallery")
									}),
									/* @__PURE__ */ jsx(Text, {
										elementDescriptor: descriptors.configureSSOInstructionsListItem,
										as: "li",
										colorScheme: "secondary",
										localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.createAppInstructions.step4.subSteps.create")
									})
								]
							})]
						})
					]
				})]
			}), /* @__PURE__ */ jsxs(Col, {
				sx: (theme) => ({ gap: theme.space.$1x5 }),
				children: [/* @__PURE__ */ jsx(Heading, {
					elementDescriptor: descriptors.configureSSOInstructionsHeading,
					as: "h3",
					textVariant: "subtitle",
					localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.assignUsersInstructions.title")
				}), /* @__PURE__ */ jsxs(Col, {
					elementDescriptor: descriptors.configureSSOInstructionsList,
					as: "ul",
					sx: (theme) => ({
						gap: theme.space.$1x5,
						margin: 0,
						paddingInlineStart: theme.space.$5,
						listStyleType: "disc"
					}),
					children: [
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.assignUsersInstructions.step1")
						}),
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.assignUsersInstructions.step2")
						}),
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.assignUsersInstructions.step3")
						}),
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.assignUsersInstructions.step4")
						}),
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.createAppStep.assignUsersInstructions.step5")
						})
					]
				})]
			})]
		}) }),
		/* @__PURE__ */ jsxs(Step.Footer, { children: [
			/* @__PURE__ */ jsx(Step.Footer.Reset, {}),
			/* @__PURE__ */ jsx(Step.Footer.Previous, {
				onClick: () => goPrev(),
				isDisabled: isFirstStep
			}),
			/* @__PURE__ */ jsx(Step.Footer.Continue, {
				onClick: () => goNext(),
				isDisabled: isLastStep
			})
		] })
	] });
};
const SamlMicrosoftServiceProviderStep = () => {
	const { goNext, goPrev, isFirstStep, isLastStep } = useWizard();
	const { enterpriseConnection } = useConfigureSSO();
	const acsUrl = enterpriseConnection?.samlConnection?.acsUrl ?? "";
	const spEntityId = enterpriseConnection?.samlConnection?.spEntityId ?? "";
	const spEntityIdField = useFormControl("spEntityId", spEntityId, {
		type: "text",
		label: localizationKeys("configureSSO.configureStep.samlMicrosoft.serviceProviderStep.serviceProviderFields.spEntityId.label"),
		isRequired: false
	});
	const acsUrlField = useFormControl("acsUrl", acsUrl, {
		type: "text",
		label: localizationKeys("configureSSO.configureStep.samlMicrosoft.serviceProviderStep.serviceProviderFields.acsUrl.label"),
		isRequired: false
	});
	return /* @__PURE__ */ jsxs(Fragment$1, { children: [
		/* @__PURE__ */ jsx(Step.Header, {
			title: localizationKeys("configureSSO.configureStep.samlMicrosoft.mainHeaderTitle"),
			description: localizationKeys("configureSSO.configureStep.samlMicrosoft.serviceProviderStep.headerSubtitle"),
			children: /* @__PURE__ */ jsx(InnerStepCounter, {})
		}),
		/* @__PURE__ */ jsx(Step.Body, { children: /* @__PURE__ */ jsx(Step.Section, {
			sx: (theme) => ({ gap: theme.space.$5 }),
			children: /* @__PURE__ */ jsxs(Col, {
				sx: (theme) => ({ gap: theme.space.$1x5 }),
				children: [/* @__PURE__ */ jsx(Heading, {
					elementDescriptor: descriptors.configureSSOInstructionsHeading,
					as: "h3",
					textVariant: "subtitle",
					localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.serviceProviderStep.title")
				}), /* @__PURE__ */ jsxs(Col, {
					elementDescriptor: descriptors.configureSSOInstructionsList,
					as: "ul",
					sx: (theme) => ({
						gap: theme.space.$1x5,
						margin: 0,
						paddingInlineStart: theme.space.$5,
						listStyleType: "disc"
					}),
					children: [
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.serviceProviderStep.step1")
						}),
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.serviceProviderStep.step2")
						}),
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.serviceProviderStep.step3")
						}),
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.serviceProviderStep.step4")
						}),
						/* @__PURE__ */ jsxs(Box, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							sx: (theme) => ({
								fontSize: theme.fontSizes.$md,
								lineHeight: theme.lineHeights.$small,
								color: theme.colors.$colorMutedForeground
							}),
							children: [/* @__PURE__ */ jsx(Text, {
								as: "span",
								colorScheme: "secondary",
								localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.serviceProviderStep.step5")
							}), /* @__PURE__ */ jsxs(Col, {
								sx: (theme) => ({
									gap: theme.space.$4,
									marginBlock: theme.space.$3
								}),
								children: [/* @__PURE__ */ jsx(Form.ControlRow, {
									elementId: spEntityIdField.id,
									children: /* @__PURE__ */ jsx(Form.CommonInputWrapper, {
										...spEntityIdField.props,
										children: /* @__PURE__ */ jsx(ClipboardInput, {
											value: spEntityId,
											readOnly: true,
											copyIcon: SvgClipboard,
											copiedIcon: SvgCheckmark
										})
									})
								}), /* @__PURE__ */ jsx(Form.ControlRow, {
									elementId: acsUrlField.id,
									children: /* @__PURE__ */ jsx(Form.CommonInputWrapper, {
										...acsUrlField.props,
										children: /* @__PURE__ */ jsx(ClipboardInput, {
											value: acsUrl,
											readOnly: true,
											copyIcon: SvgClipboard,
											copiedIcon: SvgCheckmark
										})
									})
								})]
							})]
						}),
						/* @__PURE__ */ jsx(Text, {
							elementDescriptor: descriptors.configureSSOInstructionsListItem,
							as: "li",
							colorScheme: "secondary",
							localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.serviceProviderStep.step6")
						})
					]
				})]
			})
		}) }),
		/* @__PURE__ */ jsxs(Step.Footer, { children: [
			/* @__PURE__ */ jsx(Step.Footer.Reset, {}),
			/* @__PURE__ */ jsx(Step.Footer.Previous, {
				onClick: () => goPrev(),
				isDisabled: isFirstStep
			}),
			/* @__PURE__ */ jsx(Step.Footer.Continue, {
				onClick: () => goNext(),
				isDisabled: isLastStep
			})
		] })
	] });
};
const MICROSOFT_ATTRIBUTE_ROWS = [
	{
		id: "email",
		isRequired: true
	},
	{
		id: "firstName",
		isRequired: false
	},
	{
		id: "lastName",
		isRequired: false
	}
];
const MicrosoftClaimNameCell = ({ claimNameKey }) => {
	const { t } = useLocalizations();
	const claimName = t(claimNameKey);
	const { onCopy, hasCopied } = useClipboard(claimName);
	return /* @__PURE__ */ jsx(Td, {
		sx: {
			maxWidth: 0,
			width: "100%"
		},
		children: /* @__PURE__ */ jsxs(Flex, {
			as: "span",
			align: "center",
			sx: (theme) => ({ gap: theme.space.$1 }),
			children: [/* @__PURE__ */ jsxs(Tooltip.Root, { children: [/* @__PURE__ */ jsx(Tooltip.Trigger, { children: /* @__PURE__ */ jsx(Text, {
				as: "span",
				sx: {
					fontFamily: "monospace",
					display: "block",
					whiteSpace: "nowrap"
				},
				children: truncateWithEndVisible(claimName, 32, 12)
			}) }), /* @__PURE__ */ jsx(Tooltip.Content, {
				text: claimNameKey,
				textSx: { overflowWrap: "anywhere" }
			})] }), /* @__PURE__ */ jsx(Button, {
				variant: "ghost",
				onClick: () => onCopy(),
				"aria-label": t(localizationKeys(hasCopied ? "configureSSO.configureStep.samlMicrosoft.attributeMappingStep.attributeMappingTable.copyClaimNameCopied" : "configureSSO.configureStep.samlMicrosoft.attributeMappingStep.attributeMappingTable.copyClaimName")),
				sx: (theme) => ({
					padding: 0,
					height: theme.sizes.$5,
					aspectRatio: 1,
					borderRadius: theme.radii.$sm,
					color: theme.colors.$colorMutedForeground
				}),
				children: /* @__PURE__ */ jsx(Icon, {
					size: "sm",
					icon: hasCopied ? SvgCheckmark : SvgClipboard,
					"aria-hidden": true
				})
			})]
		})
	});
};
const MicrosoftAttributeMappingTable = () => {
	return /* @__PURE__ */ jsxs(Table, {
		elementDescriptor: descriptors.configureSSOAttributeMappingTable,
		sx: (theme) => ({ "tr > th:first-of-type": { paddingInlineStart: theme.space.$4 } }),
		children: [/* @__PURE__ */ jsx(Thead, { children: /* @__PURE__ */ jsxs(Tr, { children: [
			/* @__PURE__ */ jsx(Th, { children: /* @__PURE__ */ jsx(Text, {
				sx: (theme) => ({ fontSize: theme.fontSizes.$xs }),
				localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.attributeMappingStep.attributeMappingTable.columns.attribute")
			}) }),
			/* @__PURE__ */ jsx(Th, { children: /* @__PURE__ */ jsx(Text, {
				sx: (theme) => ({ fontSize: theme.fontSizes.$xs }),
				localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.attributeMappingStep.attributeMappingTable.columns.claimName")
			}) }),
			/* @__PURE__ */ jsx(Th, { children: /* @__PURE__ */ jsx(Text, {
				sx: (theme) => ({ fontSize: theme.fontSizes.$xs }),
				localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.attributeMappingStep.attributeMappingTable.columns.value")
			}) })
		] }) }), /* @__PURE__ */ jsx(Tbody, { children: MICROSOFT_ATTRIBUTE_ROWS.map((row) => {
			const claimNameKey = localizationKeys(`configureSSO.configureStep.samlMicrosoft.attributeMappingStep.attributeMappingTable.rows.${row.id}.claimName`);
			return /* @__PURE__ */ jsxs(Tr, { children: [
				/* @__PURE__ */ jsx(Td, {
					sx: { whiteSpace: "nowrap" },
					children: /* @__PURE__ */ jsxs(Flex, {
						as: "span",
						align: "center",
						sx: (theme) => ({ gap: theme.space.$2 }),
						children: [/* @__PURE__ */ jsx(Text, {
							as: "span",
							colorScheme: "secondary",
							localizationKey: localizationKeys(`configureSSO.configureStep.samlMicrosoft.attributeMappingStep.attributeMappingTable.rows.${row.id}.attribute`)
						}), /* @__PURE__ */ jsx(Badge, {
							elementDescriptor: descriptors.configureSSOAttributeMappingBadge,
							elementId: descriptors.configureSSOAttributeMappingBadge.setId(row.isRequired ? "required" : "optional"),
							colorScheme: row.isRequired ? "warning" : "primary",
							localizationKey: localizationKeys(row.isRequired ? "configureSSO.configureStep.attributeMappingTable.badges.required" : "configureSSO.configureStep.attributeMappingTable.badges.optional")
						})]
					})
				}),
				/* @__PURE__ */ jsx(MicrosoftClaimNameCell, { claimNameKey }),
				/* @__PURE__ */ jsx(Td, { children: /* @__PURE__ */ jsx(Text, {
					as: "span",
					sx: { fontFamily: "monospace" },
					localizationKey: localizationKeys(`configureSSO.configureStep.samlMicrosoft.attributeMappingStep.attributeMappingTable.rows.${row.id}.value`)
				}) })
			] }, row.id);
		}) })]
	});
};
const SamlMicrosoftAttributeMappingStep = () => {
	const { goNext, goPrev, isFirstStep, isLastStep } = useWizard();
	return /* @__PURE__ */ jsxs(Fragment$1, { children: [
		/* @__PURE__ */ jsx(Step.Header, {
			title: localizationKeys("configureSSO.configureStep.samlMicrosoft.mainHeaderTitle"),
			description: localizationKeys("configureSSO.configureStep.samlMicrosoft.attributeMappingStep.headerSubtitle"),
			children: /* @__PURE__ */ jsx(InnerStepCounter, {})
		}),
		/* @__PURE__ */ jsx(Step.Body, { children: /* @__PURE__ */ jsxs(Step.Section, {
			sx: (theme) => ({ gap: theme.space.$3 }),
			children: [
				/* @__PURE__ */ jsx(Text, {
					as: "p",
					colorScheme: "secondary",
					elementDescriptor: descriptors.configureSSOInstructionsHeading,
					localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.attributeMappingStep.title")
				}),
				/* @__PURE__ */ jsxs(Col, {
					elementDescriptor: descriptors.configureSSOInstructionsList,
					as: "ol",
					sx: (theme) => ({
						gap: theme.space.$1x5,
						margin: 0,
						paddingInlineStart: theme.space.$5,
						listStyleType: "decimal"
					}),
					children: [/* @__PURE__ */ jsx(Text, {
						elementDescriptor: descriptors.configureSSOInstructionsListItem,
						as: "li",
						colorScheme: "secondary",
						localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.attributeMappingStep.step1")
					}), /* @__PURE__ */ jsx(Text, {
						elementDescriptor: descriptors.configureSSOInstructionsListItem,
						as: "li",
						colorScheme: "secondary",
						localizationKey: localizationKeys("configureSSO.configureStep.samlMicrosoft.attributeMappingStep.step2")
					})]
				}),
				/* @__PURE__ */ jsx(MicrosoftAttributeMappingTable, {})
			]
		}) }),
		/* @__PURE__ */ jsxs(Step.Footer, { children: [
			/* @__PURE__ */ jsx(Step.Footer.Reset, {}),
			/* @__PURE__ */ jsx(Step.Footer.Previous, {
				onClick: () => goPrev(),
				isDisabled: isFirstStep
			}),
			/* @__PURE__ */ jsx(Step.Footer.Continue, {
				onClick: () => goNext(),
				isDisabled: isLastStep
			})
		] })
	] });
};
const MICROSOFT_SAML_IDP_MODES = ["metadataUrl", "manual"];
const SamlMicrosoftIdentityProviderMetadataStep = () => {
	const card = useCardState();
	const { goNext, goPrev, isFirstStep } = useWizard();
	const { enterpriseConnection, enterpriseConnectionMutations: { updateConnection } } = useConfigureSSO();
	const samlConnection = enterpriseConnection?.samlConnection;
	const hasExistingConfig = Boolean(samlConnection?.idpSsoUrl || samlConnection?.idpEntityId || samlConnection?.idpCertificate || samlConnection?.idpMetadataUrl);
	const existingCertPresent = Boolean(samlConnection?.idpCertificate);
	const [mode, setMode] = React.useState(hasExistingConfig ? "manual" : "metadataUrl");
	const [certFile, setCertFile] = React.useState(null);
	const [isSubmitting, setIsSubmitting] = React.useState(false);
	const metadataUrlField = useFormControl("idpMetadataUrl", samlConnection?.idpMetadataUrl ?? "", {
		type: "text",
		label: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.metadataUrl.label"),
		placeholder: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.metadataUrl.placeholder"),
		isRequired: true
	});
	const signOnUrlField = useFormControl("idpSsoUrl", samlConnection?.idpSsoUrl ?? "", {
		type: "text",
		label: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.manual.signOnUrl.label"),
		placeholder: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.manual.signOnUrl.placeholder"),
		isRequired: true
	});
	const issuerField = useFormControl("idpEntityId", samlConnection?.idpEntityId ?? "", {
		type: "text",
		label: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.manual.issuer.label"),
		placeholder: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.manual.issuer.placeholder"),
		isRequired: true
	});
	const certificateField = useFormControl("idpCertificate", "", {
		type: "text",
		label: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.manual.signingCertificate.label"),
		isRequired: true
	});
	const trimmedMetadataUrl = metadataUrlField.value.trim();
	const trimmedSignOnUrl = signOnUrlField.value.trim();
	const trimmedIssuer = issuerField.value.trim();
	const hasCert = certFile !== null || existingCertPresent;
	const canSubmit = (mode === "metadataUrl" ? trimmedMetadataUrl.length > 0 : trimmedSignOnUrl.length > 0 && trimmedIssuer.length > 0 && hasCert) && !isSubmitting;
	const formProps = mode === "metadataUrl" ? {
		mode: "metadataUrl",
		form: { field: metadataUrlField },
		labels: { description: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.metadataUrl.description") }
	} : {
		mode: "manual",
		form: {
			signOnUrlField,
			issuerField,
			certificateField,
			certFile,
			onCertFileChange: setCertFile,
			existingCertPresent
		},
		labels: {
			description: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.manual.description"),
			uploadFile: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.manual.signingCertificate.uploadFile"),
			replaceFile: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.manual.signingCertificate.replaceFile"),
			removeFile: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.manual.signingCertificate.removeFile"),
			fileUploaded: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.manual.signingCertificate.fileUploaded")
		}
	};
	const handleContinue = async () => {
		if (!enterpriseConnection || !canSubmit) return;
		card.setError(void 0);
		setIsSubmitting(true);
		try {
			const saml = await buildSamlConfigurationPayload({
				mode,
				metadataUrl: { value: metadataUrlField.value },
				manual: {
					signOnUrl: signOnUrlField.value,
					issuer: issuerField.value,
					certFile
				}
			});
			await updateConnection(enterpriseConnection.id, { saml });
			goNext();
		} catch (err) {
			if (mode === "metadataUrl") applySamlSubmitError(err, card, metadataUrlField);
			else applySamlSubmitError(err, card, signOnUrlField, [issuerField, certificateField]);
			setIsSubmitting(false);
		}
	};
	return /* @__PURE__ */ jsxs(Fragment$1, { children: [
		/* @__PURE__ */ jsx(Step.Header, {
			title: localizationKeys("configureSSO.configureStep.samlMicrosoft.mainHeaderTitle"),
			description: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.headerSubtitle"),
			children: /* @__PURE__ */ jsx(InnerStepCounter, {})
		}),
		/* @__PURE__ */ jsx(Step.Body, { children: /* @__PURE__ */ jsxs(Step.Section, {
			fill: true,
			gap: 5,
			children: [/* @__PURE__ */ jsx(IdentityProviderConfigurationModes, {
				modes: MICROSOFT_SAML_IDP_MODES,
				value: mode,
				onChange: (next) => {
					card.setError(void 0);
					setMode(next);
				},
				labels: {
					ariaLabel: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.modes.ariaLabel"),
					metadataUrl: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.modes.metadataUrl"),
					manual: localizationKeys("configureSSO.configureStep.samlMicrosoft.identityProviderMetadataStep.modes.manual")
				}
			}), /* @__PURE__ */ jsx(IdentityProviderConfigurationForm, { ...formProps })]
		}) }),
		/* @__PURE__ */ jsxs(Step.Footer, { children: [
			/* @__PURE__ */ jsx(Step.Footer.Reset, {}),
			/* @__PURE__ */ jsx(Step.Footer.Previous, {
				onClick: () => goPrev(),
				isDisabled: isFirstStep || isSubmitting
			}),
			/* @__PURE__ */ jsx(Step.Footer.Continue, {
				onClick: handleContinue,
				isLoading: isSubmitting,
				isDisabled: !canSubmit
			})
		] })
	] });
};

//#endregion
export { SamlMicrosoftConfigureSteps };
